# lassl

### Puget Systems - Long Scientific Software List

---

# Scientific Applications

This is a list of important commonly used scientific applications with descriptions and links for more information. It is certainly not an exhaustive list but it is extensive, or at lease will be when finished. There are approximately 800 applications in the current list.

The initial material was sourced from the the maintained application list on the [NIH HPC Biowulf cluster](http://hpc.nih.gov).

<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=2 orderedList=false} -->

<!-- code_chunk_output -->

- [lassl](#lassl)
- [Scientific Applications](#scientific-applications)
  - [More catagories TO-DO](#more-catagories-to-do)
  - [Computational Chemistry](#computational-chemistry)
  - [Deep Learning](#deep-learning)
  - [High-Throughput Sequencing](#high-throughput-sequencing)
  - [Image Analysis](#image-analysis)
  - [Linkage/Phylogenetics](#linkagephylogenetics)
  - [Mass Spectrometry](#mass-spectrometry)
  - [Mathematical/Statistics](#mathematicalstatistics)
  - [Molecular Modeling/Graphics](#molecular-modelinggraphics)
  - [Sequence Analysis](#sequence-analysis)
  - [Structural Biology](#structural-biology)
  - [Systems Biology](#systems-biology)
  - [Utilities](#utilities)
  - [Workflow Managers](#workflow-managers)
  - [Appendex](#appendex)

<!-- /code_chunk_output -->

## More catagories TO-DO

- Physics
- Astronomy
- Earth Science
- Weather Simulation
- Environmental Biology
- Geology
- Machine Learning/AI
- Data Science
- Computer and Information Science
- Engineering
- Materials Science
- Social Behavioral Cognitive Science
- Economics Econometrics

## Computational Chemistry

---

### Acemd

- **Description:** ACEMD is a high performance molecular dynamics code for biomolecular systems designed specifically for NVIDIA GPUs. Simple and fast, ACEMD uses very similar commands and input files of NAMD and output files as NAMD or Gromacs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/acemd.html](https://hpc.nih.gov/apps/acemd.html)

### AMBER

- **Description:** AMBER (Assisted Model Building with Energy Refinement) is a package of molecular simulation programs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AMBER.html](https://hpc.nih.gov/apps/AMBER.html)

### AMPL

- **Description:** The Accelerating Therapeutics for Opportunites in Medicine (ATOM) Consortium Modeling PipeLine for Drug Discovery. AMPL is an open-source, modular, extensible software pipeline for building and sharing models to advance in silico drug discovery.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ampl.html](https://hpc.nih.gov/apps/ampl.html)

### APBS

- **Description:** APBS (Adaptive Poisson-Boltzmann Solver) is a software package for the numerical solution of the Poisson-Boltzmann equation (PBE), one of the most popular continuum models for describing electrostatic interactions between molecular solutes in salty, aqueous media.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/APBS.html](https://hpc.nih.gov/apps/APBS.html)

### Autodock

- **Description:** Autodock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure.

### AutodockVina

- **Description:** AutoDock Vina is a program for drug discovery, molecular docking and virtual screening, offering multi-core capability, high performance and enhanced accuracy and ease of use. It is closely tied to Autodock.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AutodockVina.html](https://hpc.nih.gov/apps/AutodockVina.html)

### campari

- **Description:** Campari is a molecular simulations application that supports Monte Carlo and molecular dynamics simulations of biomolecules. It has robust support for a number of different methods, including in silico docking.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/campari.html](https://hpc.nih.gov/apps/campari.html)

### CHARMM

- **Description:** CHARMM is a general and flexible software application for modeling the structure and behavior of molecular systems.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/charmm/index.html](https://hpc.nih.gov/apps/charmm/index.html)

### cryoDRGN

- **Description:** CryoDRGN is an algorithm that leverages the representation power of deep neural networks
  to directly reconstruct continuous distributions of 3D density maps
  and map per-particle heterogeneity of single-particle cryo-EM datasets.
  It contains interactive tools to visualize a dataset’s distribution of per-particle
  variability, generate density maps for exploratory analysis, extract particle
  subsets for use with other tools and generate trajectories to visualize molecular motions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cryoDRGN.html](https://hpc.nih.gov/apps/cryoDRGN.html)

### GAMESS

- **Description:** GAMESS is a general ab initio quantum chemistry package.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GAMESS.html](https://hpc.nih.gov/apps/GAMESS.html)

### Gaussian

- **Description:** Gaussian is a connected system of programs for performing semiempirical and ab initio molecular orbital (MO) calculations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Gaussian.html](https://hpc.nih.gov/apps/Gaussian.html)

### Gromacs

- **Description:** Gromacs is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins and lipids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Gromacs.html](https://hpc.nih.gov/apps/Gromacs.html)

### NAMD

- **Description:** NAMD is a parallel molecular dynamics program for UNIX platforms designed for high-performance simulations in structural biology. VMD, the associated molecular visualization program, is also available.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/NAMD.html](https://hpc.nih.gov/apps/NAMD.html)

### ORCA

- **Description:** ORCA is an ab initio, DFT, and semi-empirical SCF-MO package.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ORCA.html](https://hpc.nih.gov/apps/ORCA.html)

### Psi4

- **Description:** Psi4 is an ab-initio electronic structure code that supports various methods for calculating energies and gradients of molecular systems.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/psi4.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/psi4.html)

### Q-Chem

- **Description:** Q-Chem is a comprehensive ab initio quantum chemistry package for accurate predictions of molecular structures, reactivities, and vibrational, electronic and NMR spectra.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/qchem.html](https://hpc.nih.gov/apps/qchem.html)

### Schrodinger

- **Description:** A limited number of Schrödinger applications are available on the Biowulf cluster through the Molecular Modeling Interest Group. Most are available through the Maestro GUI.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/schrodinger/index.html](https://hpc.nih.gov/apps/schrodinger/index.html)

### VMD

- **Description:** VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. To use, type vmd at the prompt.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VMD.html](https://hpc.nih.gov/apps/VMD.html)

## Deep Learning

---

### ARDISS

- **Description:** ARDISS is a method to impute missing summary statistics in mixed-ethnicity cohorts
  through Gaussian Process Regression and automatic relevance determination.
  ARDISS is trained on an external reference panel and does not require information
  about allele frequencies of genotypes from the original study.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ARDISS.html](https://hpc.nih.gov/apps/ARDISS.html)

### B-SOiD

- **Description:** B-SOiD (Behavioral Segmentation in Deeplabcut) is an
  unsupervised learning algorithm that serves to discover and classify behaviors that are not pre-defined by users.
  It segregates statistically different, sub-second rodent behaviors with a single bottom-up perspective video cameraR by performing a novel expectation maximization
  fitting of Gaussian mixture models on t-Distributed Stochastic Neighbor Embedding (t-SNE).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/b-soid.html](https://hpc.nih.gov/apps/b-soid.html)

### BioBERT

- **Description:** BioBERT is a biomedical language representation model
  designed for biomedical text mining tasks
  such as biomedical named entity recognition, relation extraction, question answering, etc.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BioBERT.html](https://hpc.nih.gov/apps/BioBERT.html)

### BioGANs

- **Description:** BioGANs is a novel application of Generative Adversarial Networks (GAN) to the synthesis of cells
  imaged by fluorescence microscopy. It allows to infer the correlation between the spatial pattern
  of different fluorescent proteins that reflects important biological functions.
  The synthesized images capture these relationships, which are relevant for biological applications.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/biogans.html](https://hpc.nih.gov/apps/biogans.html)

### CANDLE

- **Description:** Open-source software platform providing highly scalable deep learning methodologies, including intelligent hyperparameter optimization.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/candle/index.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/candle/index.html)

### Clairvoyante

- **Description:** The accurate identification of DNA sequence variants is particularly difficult
  for single molecule sequencing, which has a high per-nucleotide
  error rate (~5%-15%). Clairvoyante implements a multitask
  five-layer convolutional neural network model for predicting variant type (SNP or indel),
  zygosity, alternative allele and indel length from aligned reads.
  Using well-characterized tesing data, Clairvoyante achieved 99.73%, 97.68% and 95.36% precision on
  known variants, and 98.65%, 92.57%, 77.89% F1-score for whole-genome analysis, using
  Illumina, PacBio, and Oxford Nanopore data, respectively.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Clairvoyante.html](https://hpc.nih.gov/apps/Clairvoyante.html)

### cryoDRGN

- **Description:** CryoDRGN is an algorithm that leverages the representation power of deep neural networks
  to directly reconstruct continuous distributions of 3D density maps
  and map per-particle heterogeneity of single-particle cryo-EM datasets.
  It contains interactive tools to visualize a dataset’s distribution of per-particle
  variability, generate density maps for exploratory analysis, extract particle
  subsets for use with other tools and generate trajectories to visualize molecular motions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cryoDRGN.html](https://hpc.nih.gov/apps/cryoDRGN.html)

### cuDNN

- **Description:** The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. cuDNN provides highly tuned implementations for standard routines such as forward and backward convolution, pooling, normalization, and activation layers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/development/cuDNN.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/development/cuDNN.html)

### DanQ

- **Description:** DanQ is a hybrid convolutional and recurrent deep
  neural network for quantifying the function of DNA sequences
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DanQ.html](https://hpc.nih.gov/apps/DanQ.html)

### DeepEmhancer

- **Description:** A deep learning solution for cryo-EM map.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepEMhancer.html](https://hpc.nih.gov/apps/DeepEMhancer.html)

### DeepHiC

- **Description:** DeepHiC ("deep Hi-C") is a generative adversarial network aimed at predicting the high-resolution Hi-C contact maps from ithe low-coverage sequencing data.
  Hi-C is commonly used to study three-dimensional genome organization.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepHiC.html](https://hpc.nih.gov/apps/DeepHiC.html)

### DeepHit

- **Description:** The DeepHit application uses a deep neural
  network to learn the distribution of survival times directly.
  DeepHit makes no assumptions about the underlying stochastic
  process and allows for the possibility that the relationship
  between covariates and risk(s) changes over time. Most importantly,
  DeepHit smoothly handles competing risks; i.e. settings
  in which there is more than one possible event of interest.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepHit.html](https://hpc.nih.gov/apps/DeepHit.html)

### DeepLab

- **Description:** DeepLab is a Semantic Image Segmentation tool. It makes use of the Deep Convolutional Networks, Dilated (a.k.a. Atrous) Convolution, and Fully Connected Conditional Random Fields.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepLab.html](https://hpc.nih.gov/apps/DeepLab.html)

### DeepLabCut

- **Description:** DeepLabCut is an open source toolbox
  that builds on a state-of-the-art human pose estimation algorithm.
  It allows training of a deep neural network by using limited training data
  to precisely track user-defined features, so that the human labeling accuracy will be matched.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/DeepLabCut.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/DeepLabCut.html)

### DeepM6A

- **Description:** DeepM6A application implements a deep-learning-based algorithm
  for predicting potential DNA 6mA sites de novo from sequence at single-nucleotide resolution.
  Application of this tool to three representative model organisms, Arabidopsis thaliana, Drosophila melanogaster and Escherichia coli,
  demonstrated that it supercedes the conventional k-mer-based approaches
  in both accuracy and performance.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepM6A.html](https://hpc.nih.gov/apps/DeepM6A.html)

### DeepMM

- **Description:** DeepMM implements fully automated de novo structure modeling method, MAINMAST, which builds three-dimensional models of a protein from a near-atomic resolution EM map. The method directly traces the protein’s main-chain and identifies Cα positions as tree-graph structures in the EM map.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepMM.html](https://hpc.nih.gov/apps/DeepMM.html)

### digits

- **Description:** DIGITS simplifies common deep learning tasks such as managing data, designing and training neural networks on multi-GPU systems, monitoring performance in real time with advanced visualizations, and selecting the best performing model from the results browser for deployment. DIGITS is completely interactive so that data scientists can focus on designing and training networks rather than programming and debugging.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/digits.html](https://hpc.nih.gov/apps/digits.html)

### meka

- **Description:** A Multi-label Extension to WEKA
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/meka.html](https://hpc.nih.gov/apps/meka.html)

### ReLeaSE

- **Description:** ReLeaSE (Reinforcement Learning for Structural Evolution) is an application for de-novo Drug Design
  based on Reinforcement Learning. It integrates two deep neural networks: generative and
  predictive, that are trained separately but are used jointly to generate novel targeted chemical libraries.
  ReLeaSE uses simple representation of molecules by their simplified molecular input line entry specification (SMILES) strings only.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ReLeaSE.html](https://hpc.nih.gov/apps/ReLeaSE.html)

### RoseTTAFold

- **Description:** Accurate prediction of protein structures and interactions using a 3-track network, , in which information at the 1D sequence level, the 2D distance map level, and the 3D coordinate level is successively transformed and integrated.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rosettafold.html](https://hpc.nih.gov/apps/rosettafold.html)

### SpliceAI

- **Description:** SpliceAI is a deep neural network that accurately predicts
  splice junctions from an arbitrary pre-mRNA
  transcript sequence, enabling precise prediction of
  noncoding genetic variants that cause cryptic
  splicing.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SpliceAI.html](https://hpc.nih.gov/apps/SpliceAI.html)

### tensorrt

- **Description:** NVIDIA TensorRT™ is a platform for high-performance deep learning inference. It includes a deep learning inference optimizer and runtime that delivers low latency and high-throughput for deep learning inference applications. TensorRT-based applications perform up to 40x faster than CPU-only platforms during inference. With TensorRT, you can optimize neural network models trained in all major frameworks, calibrate for lower precision with high accuracy, and finally deploy to hyperscale data centers, embedded, or automotive product platforms.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tensorrt.html](https://hpc.nih.gov/apps/tensorrt.html)

### Tybalt

- **Description:** Tybalt implements a Variational EutoEncoder (VAE), a deep neural network approach capable of generating meaningful latent spaces for image and text data. Tybalt has been trained on The Cancer Genome Atlas (TCGA) pan-cancer RNA-seq data and used to identify specific patterns in the VAE encoded features.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Tybalt.html](https://hpc.nih.gov/apps/Tybalt.html)

### UNet

- **Description:** U-Net is an image segmentation tool. It relies on the strong use of data augmentation to use the available
  annotated samples more efficiently. The architecture consists of a contracting
  path to capture context and a symmetric expanding path that enables precise localization.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/UNet.html](https://hpc.nih.gov/apps/UNet.html)

### YOLO

- **Description:** YOLO is a new approach to object detection.
  Prior work on object detection repurposed classifiers to perform detection.
  Instead, YOLO frames object detection as a regression problem
  to spatially separated bounding boxes and associated class probabilities.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/YOLO.html](https://hpc.nih.gov/apps/YOLO.html)

## High-Throughput Sequencing

---

### ABC

- **Description:** The Activity-by-Contact (ABC) model predicts which enhancers regulate which genes on a cell type specific basis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ABC.html](https://hpc.nih.gov/apps/ABC.html)

### abruijn

- **Description:** ABruijn is a assembler for long reads from, for example, PacBio and Oxford Nanopore Technologies sequencers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/abruijn.html](https://hpc.nih.gov/apps/abruijn.html)

### abyss

- **Description:** Abyss represents Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler. The parallel version is implemented using MPI and is capable of assembling larger genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/abyss.html](https://hpc.nih.gov/apps/abyss.html)

### adam

- **Description:** ADAM is a library and command line tool that enables the use of Apache Spark to parallelize genomic data analysis across cluster/cloud computing environments. ADAM uses a set of schemas to describe genomic sequences, reads, variants/genotypes, and features, and can be used with data in legacy genomic file formats such as SAM/BAM/CRAM, BED/GFF3/GTF, and VCF, as well as data stored in the columnar Apache Parquet format. On a single node, ADAM provides competitive performance to optimized multi-threaded tools, while enabling scale out to clusters with more than a thousand cores. ADAM's APIs can be used from Scala, Java, Python, R, and SQL.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/adam.html](https://hpc.nih.gov/apps/adam.html)

### AfterQC

- **Description:** Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AfterQC.html](https://hpc.nih.gov/apps/AfterQC.html)

### albacore

- **Description:** ONT basecaller
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/angsd.html](https://hpc.nih.gov/apps/angsd.html)

### angsd

- **Description:** ANGSD is a software for analyzing next generation sequencing data. The software can handle a number of different input types from mapped reads to imputed genotype probabilities. Most methods take genotype uncertainty into account instead of basing the analysis on called genotypes. This is especially useful for low and medium depth data. The software is written in C++ and has been used on large sample sizes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/angsd.html](https://hpc.nih.gov/apps/angsd.html)

### AnnotSV

- **Description:** AnnotSV is a program designed for annotating Structural Variations (SV). This tool compiles functionally, regulatory and clinically relevant information and aims at providing annotations useful to i) interpret SV potential pathogenicity and ii) filter out SV potential false positives.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AnnotSV.html](https://hpc.nih.gov/apps/AnnotSV.html)

### ANNOVAR

- **Description:** ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ANNOVAR.html](https://hpc.nih.gov/apps/ANNOVAR.html)

### Arioc

- **Description:** A GPU-accelerated short read aligner

### ascatNgs

- **Description:** AscatNGS contains the Cancer Genome Projects workflow implementation of the ASCAT copy number algorithm for paired end sequencing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ascatNgs.html](https://hpc.nih.gov/apps/ascatNgs.html)

### atac_dnase_pipelines

- **Description:** This pipeline is designed for automated end-to-end quality control and processing of ATAC-seq or DNase-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/atac_dnase_pipelines.html](https://hpc.nih.gov/apps/atac_dnase_pipelines.html)

### atropos

- **Description:** An NGS read trimming tool that is specific, sensitive, and speedy.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/atropos.html](https://hpc.nih.gov/apps/atropos.html)

### augustus

- **Description:** AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/augustus.html](https://hpc.nih.gov/apps/augustus.html)

### bam2fastq

- **Description:** This tool is used to extract raw sequences (with qualities) from bam files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bam2fastq.html](https://hpc.nih.gov/apps/bam2fastq.html)

### bam2mpg

- **Description:** The program “bam2mpg” calls genotypes from sequence reads of haploid or diploid DNA
  aligned to a closely-related reference sequence. The program reads alignments in BAM
  format (http://samtools.sourceforge.net). The MPG (Most Probable Genotype) algorithm
  is based on a Bayesian model which simulates sampling from one or two alleles with
  sequencing error, and then calculates the likelihood of each possible genotype given
  the observed sequence data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bam2mpg.html](https://hpc.nih.gov/apps/bam2mpg.html)

### bamliquidator

- **Description:** bamliquidator is a set of tools for analyzing the density of short DNA sequence read alignments in the BAM file format.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamliquidator.html](https://hpc.nih.gov/apps/bamliquidator.html)

### bamreadcount

- **Description:** Bam-readcount generates metrics at single nucleotide positions.
  There are number of metrics generated which can be useful for filtering out false positive calls.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamreadcount.html](https://hpc.nih.gov/apps/bamreadcount.html)

### BAMSurgeon

- **Description:** Tools for adding mutations to existing .bam files; used for testing mutation callers
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BAMSurgeon.html](https://hpc.nih.gov/apps/BAMSurgeon.html)

### bamtools

- **Description:** BamTools provides a fast, flexible C++ API & toolkit for reading, writing, and manipulating BAM files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamtools.html](https://hpc.nih.gov/apps/bamtools.html)

### bamUtil

- **Description:** bamUtil is a repository that contains several programs that perform operations on SAM/BAM files. All of these programs are built into a single executable, bam.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamUtil.html](https://hpc.nih.gov/apps/bamUtil.html)

### Bartender

- **Description:** Bartender is an accurate clustering algorithm to detect barcodes
  and their abundances from raw next-generation sequencing data.
  In contrast with existing methods that cluster based on sequence similarity alone,
  Bartender uses a modified two-sample proportion test that also considers cluster size. This modification
  results in higher accuracy and lower rates of under- and over-clustering artifacts.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bartender.html](https://hpc.nih.gov/apps/bartender.html)

### basespace_cli

- **Description:** Command line interface for Illumina's BaseSpace
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/basespace_cli.html](https://hpc.nih.gov/apps/basespace_cli.html)

### bazam

- **Description:** A tool to extract paired reads in FASTQ format from coordinate sorted BAM files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bazam.html](https://hpc.nih.gov/apps/bazam.html)

### bbtools

- **Description:** An extensive set of bioinformatics tools including bbmap (short read aligner), bbnorm (kmer based normalization), dedupe (deduplication and clustering of unaligned reads), reformat (formatting and trimming reads) and many more.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bbtools.html](https://hpc.nih.gov/apps/bbtools.html)

### bcbio-nextgen

- **Description:** Validated, scalable, community developed variant calling, RNA-seq and small RNA analysis
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://bcbio-nextgen.readthedocs.io/en/latest/index.html](https://hpc.nih.gov/apps/https://bcbio-nextgen.readthedocs.io/en/latest/index.html)

### bcl-convert

- **Description:** The Illumina BCL Convert is a standalone local software app that converts the Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files. BCL Convert also provides adapter handling (through masking and trimming) and UMI trimming and produces metric outputs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bcl-convert.html](https://hpc.nih.gov/apps/bcl-convert.html)

### bcl2fastq

- **Description:** a tool to handle bcl conversion and demultiplexing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bcl2fastq.html](https://hpc.nih.gov/apps/bcl2fastq.html)

### bedops

- **Description:** Bedops is a suite of tools to address common questions raised in genomic studies - mostly with regard to overlap and proximity relationships between data sets - BEDOPS aims to be scalable and flexible, facilitating the efficient and accurate analysis and management of large-scale genomic data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bedops.html](https://hpc.nih.gov/apps/bedops.html)

### bedtools

- **Description:** The BEDTools utilities allow one to address common genomics tasks such finding feature overlaps and computing coverage. In addition, one can develop sophisticated pipelines that answer complicated research questions by "streaming" several BEDTools together.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bedtools.html](https://hpc.nih.gov/apps/bedtools.html)

### BETA

- **Description:** Binding and expression target analysis (BETA) is a software package that integrates ChIP-seq of TFs or chromatin regulators with differential gene expression data to infer direct target genes. The combination of ChIP-seq and transcriptome analysis is a compelling approach to unravel the regulation of gene expression.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/beta.html](https://hpc.nih.gov/apps/beta.html)

### BFC

- **Description:** BFC is a standalone tool for correcting sequencing errors from Illumina sequencing data. It is specifically designed for high-coverage whole-genome human data, though also performs well for small genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bfc.html](https://hpc.nih.gov/apps/bfc.html)

### bigSCale2

- **Description:** bigSCale is a complete framework for the analysis and visualization of single cell data. It allows to cluster, phenotype, perform pseudotime analysis, infer gene regulatory networks and reduce large datasets in smaller datasets with higher quality.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/bigSCale2.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/bigSCale2.html)

### binlorry

- **Description:** BinLorry is a tool for binning and filtering sequencing reads into distinct files. Reads can be binned and filtered by any attributes encoded in their headers, documented in a CSV file or by length.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/binlorry.html](https://hpc.nih.gov/apps/binlorry.html)

### bioawk

- **Description:** Regular awk with support for several common biological data formats, including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bioawk.html](https://hpc.nih.gov/apps/bioawk.html)

### biobambam2

- **Description:** Tools for early stage alignment file processing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/biobambam2.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/biobambam2.html)

### biom-format

- **Description:** tool (and library) to manipulate Biological Observation Matrix (BIOM) Format files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/biom-format.html](https://hpc.nih.gov/apps/biom-format.html)

### bismark

- **Description:** Bismark is a program to map bisulfite treated sequencing reads to a genome of interest and perform methylation calls in a single step. The output can be easily imported into a genome viewer, such as SeqMonk, and enables a researcher to analyse the methylation levels of their samples straight away.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bismark.html](https://hpc.nih.gov/apps/bismark.html)

### bison

- **Description:** BISON is a bisulfite-converted short-read aligner that can natively utilize high-performance computing clusters to increase speed.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bison.html](https://hpc.nih.gov/apps/bison.html)

### bonito

- **Description:** A PyTorch Basecaller for Oxford Nanopore Reads
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bonito.html](https://hpc.nih.gov/apps/bonito.html)

### bowtie

- **Description:** bowtie is an ultrafast, memory-efficient short read aligner geared toward quickly aligning large sets of short DNA sequences (reads) to large genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bowtie.html](https://hpc.nih.gov/apps/bowtie.html)

### bowtie2

- **Description:** A version of bowtie that's particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bowtie2.html](https://hpc.nih.gov/apps/bowtie2.html)

### bpipe

- **Description:** Bpipe provides a platform for running big bioinformatics jobs

### BRASS

- **Description:** BRASS analyses one or more related BAM files of paired-end sequencing to determine potential rearrangement breakpoints.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BRASS.html](https://hpc.nih.gov/apps/BRASS.html)

### breakdancer

- **Description:** provides genome-wide detection of structural variants from next generation paired-end sequencing reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/breakdancer.html](https://hpc.nih.gov/apps/breakdancer.html)

### breseq

- **Description:** breseq is a computational pipeline for finding mutations relative to a reference sequence in short-read DNA re-sequencing data. It is intended for haploid microbial genomes (<20 Mb).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/breseq.html](https://hpc.nih.gov/apps/breseq.html)

### bsmap

- **Description:** BSMAP is a short reads mapping software for bisulfite sequencing reads. Bisulfite treatment converts unmethylated Cytosines into Uracils (sequenced as Thymine) and leave methylated Cytosines unchanged, hence provides a way to study DNA cytosine methylation at single nucleotide resolution. BSMAP aligns the Ts in the reads to both Cs and Ts in the reference.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bsmap.html](https://hpc.nih.gov/apps/bsmap.html)

### busco

- **Description:** BUSCO completeness assessments employ sets of Benchmarking Universal Single-Copy Orthologs from OrthoDB (www.orthodb.org) to provide quantitative measures of the completeness of genome assemblies, annotated gene sets, and transcriptomes in terms of expected gene content.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/busco.html](https://hpc.nih.gov/apps/busco.html)

### bwa

- **Description:** BWA is a fast light-weighted tool that aligns short sequences to a sequence database, such as the human reference genome.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bwa.html](https://hpc.nih.gov/apps/bwa.html)

### bwa-mem2

- **Description:** The next version of the bwa-mem algorithm in bwa.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bwa-mem2.html](https://hpc.nih.gov/apps/bwa-mem2.html)

### CADD

- **Description:** CADD (Combined Annotation Dependent Depletion) is a tool for scoring the deleteriousness of single nucleotide variants as well as insertion/deletions variants in the human genome. Currently, it supports the builds: GRCh37/hg19 and GRCh38/hg38.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CADD.html](https://hpc.nih.gov/apps/CADD.html)

### Canu

- **Description:** Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing (such as the PacBio RSII or Oxford Nanopore MinION). Canu will correct the reads, then trim suspicious regions (such as remaining SMRTbell adapter), then assemble the corrected and cleaned reads into unitigs.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/canu.html](https://hpc.nih.gov/apps/canu.html)

### Canvas

- **Description:** Canvas is a tool for calling copy number variants (CNVs) from human DNA sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Canvas.html](https://hpc.nih.gov/apps/Canvas.html)

### casper

- **Description:** CASPER (Context-Aware Scheme for Paired-End Read) is state-of-the art merging tool in terms of accuracy and robustness. Using this sophisticated merging method, we could get elongated reads from the forward and reverse reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/casper.html](https://hpc.nih.gov/apps/casper.html)

### caveman

- **Description:** SNV expectation maximisation based mutation calling algorithm aimed at detecting somatic mutations in paired (tumour/normal) cancer samples
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/caveman.html](https://hpc.nih.gov/apps/caveman.html)

### ccbrpipeliner

- **Description:** CCBR Pipeliner provides access to a set of best-practices NGS pipelines developed, tested, and benchmarked by experts at CCBR and NCBR for Biowulf. Contact CCBR_Pipeliner@mail.nih.gov with questions
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/CCBR/Pipeliner/wiki](https://hpc.nih.gov/apps/https://github.com/CCBR/Pipeliner/wiki)

### ceas

- **Description:** Cis-regulatory Element Annotation System is a tool designed to characterize genome-wide protein-DNA interaction patterns from ChIP-chip and ChIP-Seq of both sharp and broad binding factors.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ceas.html](https://hpc.nih.gov/apps/ceas.html)

### cellprofiler

- **Description:** An open-source application for biological image analysis
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellprofiler.html](https://hpc.nih.gov/apps/cellprofiler.html)

### cellranger

- **Description:** Cell Ranger is a set of analysis pipelines that processes Chromium single cell 3’ RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellranger.html](https://hpc.nih.gov/apps/cellranger.html)

### cellranger-arc

- **Description:** Cell Ranger ARC is a set of analysis pipelines that process Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a variety of analyses pertaining to gene expression, chromatin accessibility and their linkage. Furthermore, since the ATAC and gene expression measurements are on the very same cell, we are able to perform analyses that link chromatin accessibility and gene expression.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellranger-arc.html](https://hpc.nih.gov/apps/cellranger-arc.html)

### cellranger-atac

- **Description:** Cell Ranger ATAC is a set of analysis pipelines that process Chromium Single Cell ATAC data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellranger-atac.html](https://hpc.nih.gov/apps/cellranger-atac.html)

### cellranger-dna

- **Description:** Cell Ranger DNA is a set of analysis pipelines that process Chromium single cell DNA sequencing output to align reads, identify copy number variation (CNV), and compare heterogeneity among cells.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellranger-dna.html](https://hpc.nih.gov/apps/cellranger-dna.html)

### cellsnp

- **Description:** Pileup biallelic SNPs from single-cell and bulk RNA-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellsnp.html](https://hpc.nih.gov/apps/cellsnp.html)

### cellsnp-lite

- **Description:** Efficient genotyping bi-allelic SNPs on single cells
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellsnp-lite.html](https://hpc.nih.gov/apps/cellsnp-lite.html)

### cgpBattenberg

- **Description:** Detect subclonality and copy number in matched NGS data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cgpBattenberg.html](https://hpc.nih.gov/apps/cgpBattenberg.html)

### CHESS

- **Description:** The CHESS (Comparison of Hi-C Experiments using Structural Similarity) application
  implements an algorithm for the comparison of chromatin contact maps and automatic
  differential feature extraction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/chess.html](https://hpc.nih.gov/apps/chess.html)

### ChIPseeqer

- **Description:** ChIPseeqer is an integrative, comprehensive, fast and user-friendly computational framework for in-depth analysis of ChIP-seq datasets. It combinse several computational tools
  in order to create easily customized workflows that can be adapted to the user’s needs and objectives.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ChIPseeqer.html](https://hpc.nih.gov/apps/ChIPseeqer.html)

### ChromHMM

- **Description:** ChromHMM is software for learning and characterizing chromatin states.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ChromHMM.html](https://hpc.nih.gov/apps/ChromHMM.html)

### cicero

- **Description:** an assembly-based algorithm to detect diverse classes of driver gene fusions from RNA-seq.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cicero.html](https://hpc.nih.gov/apps/cicero.html)

### circaidme

- **Description:** circaidme is a tool designed to analyze data generated with CircAID-p-seq for Oxford Nanopore Technologies. It trimmed known adapter sequences used by CircAID-p-seq kit for nanopore reads.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circaidme.html](https://hpc.nih.gov/apps/circaidme.html)

### circexplorer2

- **Description:** A combined strategy to identify circular RNAs (circRNAs and ciRNAs)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circexplorer2.html](https://hpc.nih.gov/apps/circexplorer2.html)

### circlator

- **Description:** A tool to circularize genome assemblies
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circlator.html](https://hpc.nih.gov/apps/circlator.html)

### circos

- **Description:** Circos is a program for the generation of publication-quality, circularly composited renditions of genomic data and related annotations. Circos is particularly suited for visualizing alignments, conservation and intra and inter-chromosomal relationships. Also, Circos is useful to visualize any type of information that benefits from a circular layout. Thus, although it has been designed for the field of genomics, it is sufficiently flexible to be used in other data domains.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circos.html](https://hpc.nih.gov/apps/circos.html)

### circtools

- **Description:** Circtools is a modular, Python3-based framework for circRNA-related tools that unifies several functionalities in single command line driven software. The command line follows the circtools subcommand standard that is employed in samtools or bedtools. Currently, circtools includes modules for detecting and reconstructing circRNAs, a quick check of circRNA mapping results, RBP enrichment screenings, circRNA primer design, statistical testing, and an exon usage module.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circtools.html](https://hpc.nih.gov/apps/circtools.html)

### Clair3

- **Description:** Clair3 is a small variant caller for Illumina, PacBio and ONT long reads. Compare to PEPPER (r0.4), Clair3 (v0.1) shows a better SNP F1-score with ≤30-fold of ONT data (precisionFDA Truth Challenge V2), and a better Indel F1-score, while runs generally four times faster.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Clair3.html](https://hpc.nih.gov/apps/Clair3.html)

### clark

- **Description:** A method based on a supervised sequence classification using discriminative k-mers
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/clark.html](https://hpc.nih.gov/apps/clark.html)

### clinEff

- **Description:** ClinEff is a professional version of the SnpEff and SnpSift packages, suitable for production in clincal labs. ClinEff combines the flexibility of multiple SnpEff/SnpSift commands with simplicity of running one program to perform all the annotations at once (i.e. in a single pass). It is highly customizable and can be taylored to specific pipeline needs in Clinical production environments.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/snpEff.html](https://hpc.nih.gov/apps/snpEff.html)

### cnvkit

- **Description:** Copy number variant detection from targeted DNA sequencing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cnvkit.html](https://hpc.nih.gov/apps/cnvkit.html)

### cnvnator

- **Description:** CNVnator is a tool for CNV discovery and genotyping from depth of read mapping.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cnvnator.html](https://hpc.nih.gov/apps/cnvnator.html)

### cogentap

- **Description:** Cogent NGS Analysis Pipeline (CogentAP) is bioinformatic software for analyzing RNA-seq NGS data generated using various takara kits.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cogentap.html](https://hpc.nih.gov/apps/cogentap.html)

### combp

- **Description:** A library to combine, analyze, group and correct p-values in BED files. Unique tools involve correction for spatial autocorrelation. This is useful for ChIP-Seq probes and Tiling arrays, or any data with spatial correlation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/combp.html](https://hpc.nih.gov/apps/combp.html)

### conpair

- **Description:** Concordance and contamination estimator for tumor–normal pairs

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/conpair.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/conpair.html)

### CREST

- **Description:** CREST (Clipping Reveals Structure) is an algorithm for detecting genomic structural variations at base-pair resolution using next-generation sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CREST.html](https://hpc.nih.gov/apps/CREST.html)

### crispresso

- **Description:** Software pipeline for the analysis of CRISPR-Cas9 genome editing outcomes from sequencing data

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/crispresso.html](https://hpc.nih.gov/apps/crispresso.html)

### crossmap

- **Description:** CrossMap is a program for convenient conversion of genome coordinates between different assemblies (e.g. mm9->mm10). It can convert SAM, BAM, bed, GTF, GFF, wig/bigWig, and VCF files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/crossmap.html](https://hpc.nih.gov/apps/crossmap.html)

### csvkit

- **Description:** csvkit is a suite of command-line tools for converting to and working with CSV, the king of tabular file formats.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/csvkit.html](https://hpc.nih.gov/apps/csvkit.html)

### ctk

- **Description:** The CLIP Tool Kit (CTK) is a software package that provides a set of tools for analysis of CLIP data starting from the raw reads generated by the sequencer.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ctk.html](https://hpc.nih.gov/apps/ctk.html)

### cufflinks

- **Description:** Cufflinks assembles transcripts and estimates their abundances in RNA-Seq samples. It accepts aligned RNA-Seq reads and assembles the alignments into a parsimonious set of transcripts. Cufflinks then estimates the relative abundances of these transcripts based on how many reads support each one.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cufflinks.html](https://hpc.nih.gov/apps/cufflinks.html)

### cutadapt

- **Description:** cutadapt removes adapter sequences from DNA high-throughput
  sequencing data. This is usually necessary when the read length of the
  machine is longer than the molecule that is sequenced, such as in
  microRNA data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cutadapt.html](https://hpc.nih.gov/apps/cutadapt.html)

### CutRunTools2

- **Description:** CutRunTools2 is a major update of CutRunTools2, including a set of new features specially designed for CUT&RUN and CUT&Tag experiments. Both of the bulk and single-cell data can be processed, analyzed and interpreted.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cutruntools2.html](https://hpc.nih.gov/apps/cutruntools2.html)

### DANPOS

- **Description:** A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DANPOS.html](https://hpc.nih.gov/apps/DANPOS.html)

### deepconsensus

- **Description:** DeepConsensus uses gap-aware sequence transformers to correct errors in Pacific Biosciences (PacBio) Circular Consensus Sequencing (CCS) data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/deepconsensus.html](https://hpc.nih.gov/apps/deepconsensus.html)

### DeepM6A

- **Description:** DeepM6A application implements a deep-learning-based algorithm
  for predicting potential DNA 6mA sites de novo from sequence at single-nucleotide resolution.
  Application of this tool to three representative model organisms, Arabidopsis thaliana, Drosophila melanogaster and Escherichia coli,
  demonstrated that it supercedes the conventional k-mer-based approaches
  in both accuracy and performance.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepM6A.html](https://hpc.nih.gov/apps/DeepM6A.html)

### deepsignal

- **Description:** A deep-learning method for detecting DNA methylation state from Oxford Nanopore sequencing reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/deepsignal.html](https://hpc.nih.gov/apps/deepsignal.html)

### deeptools

- **Description:** deepTools is a suite of user-friendly tools for the visualization, quality control and normalization of data from deep-sequencing DNA sequencing experiments.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/deeptools.html](https://hpc.nih.gov/apps/deeptools.html)

### deepvariant

- **Description:** DeepVariant is an analysis pipeline that uses a deep neural network to call genetic variants from next-generation DNA sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/deepvariant.html](https://hpc.nih.gov/apps/deepvariant.html)

### defuse

- **Description:** deFuse is a software package for gene fusion discovery using RNA-Seq data. The software uses clusters of discordant paired end alignments to inform a split read alignment analysis for finding fusion boundaries. The software also employs a number of heuristic filters in an attempt to reduce the number of false positives and produces a fully annotated output for each predicted fusion.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/defuse.html](https://hpc.nih.gov/apps/defuse.html)

### delly

- **Description:** DELLY is an integrated structural variant prediction method that can detect deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends and split-reads to sensitively and accurately delineate genomic rearrangements throughout the genome.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/delly.html](https://hpc.nih.gov/apps/delly.html)

### DETONATE

- **Description:** DETONATE is a tool for evaluation of de novo transcriptome
  assemblies from RNA-Seq data. It consists of two component packages, RSEM-EVAL and REF-EVAL. RSEM-EVAL is a reference-free evaluation method based on a novel probabilistic model that depends only on an assembly and the RNA-Seq reads used for its construction. REF-EVAL is a toolkit of reference-based measures.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DETONATE.html](https://hpc.nih.gov/apps/DETONATE.html)

### distiller-nf

- **Description:** A modular Hi-C mapping pipeline for reproducible data analysis, it was used for Micro-C analysis too.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/distiller-nf.html](https://hpc.nih.gov/apps/distiller-nf.html)

### dosageconvertor

- **Description:** DosageConvertor is a C++ tool to convert dosage files (in VCF format) from Minimac3 to ther formats such as MaCH or PLINK.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/dosageconvertor.html](https://hpc.nih.gov/apps/dosageconvertor.html)

### drop

- **Description:** A pipeline to find aberrant events in RNA-Seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/drop.html](https://hpc.nih.gov/apps/drop.html)

### dropseq

- **Description:** Drop-seq is a technology that allows biologists to analyze genome-wide gene expression in thousands of individual cells in a single experiment.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/dropseq.html](https://hpc.nih.gov/apps/dropseq.html)

### drseq

- **Description:** Dr.seq is a QC pipeline for Drop-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/drseq.html](https://hpc.nih.gov/apps/drseq.html)

### ea-utils

- **Description:** Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc.

### encode-atac-seq-pipeline

- **Description:** This pipeline is designed for automated end-to-end quality control and processing of ATAC-seq or DNase-seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/encode-atac-seq-pipeline.html](https://hpc.nih.gov/apps/encode-atac-seq-pipeline.html)

### EPACTS

- **Description:** EPACTS (Efficient and Parallelizable Association Container Toolbox) is a versatile software pipeline to perform various statistical tests for identifying genome-wide association from sequence data through a user-friendly interface, both to scientific analysts and to method developers.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/EPACTS.html](https://hpc.nih.gov/apps/EPACTS.html)

### epic2

- **Description:** Chip-Seq broad peak/domain finder based on SICER
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/epic2.html](https://hpc.nih.gov/apps/epic2.html)

### EricScript

- **Description:** EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/EricScript.html](https://hpc.nih.gov/apps/EricScript.html)

### eukdetect

- **Description:** EukDetect: Detect eukaryotes from shotgun metagenomic data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/eukdetect.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/eukdetect.html)

### exomiser

- **Description:** The Exomiser is a Java program that functionally annotates variants from whole-exome sequencing data starting from a VCF file.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/exomiser.html](https://hpc.nih.gov/apps/exomiser.html)

### express

- **Description:** eXpress is a streaming tool for quantifying the abundances of a set of target sequences from sampled subsequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/express.html](https://hpc.nih.gov/apps/express.html)

### fade

- **Description:** Fragmentase Artifact Detection and Elimination
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fade.html](https://hpc.nih.gov/apps/fade.html)

### fanc

- **Description:** FAN-C is a toolkit for the analysis and visualization of Hi-C data. Beyond objects generated within FAN-C, the toolkit is largely compatible with Hi-C files from Cooler and Juicer.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fanc.html](https://hpc.nih.gov/apps/fanc.html)

### fastqc

- **Description:** It provide quality control functions to next gen sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fastqc.html](https://hpc.nih.gov/apps/fastqc.html)

### fastqtools

- **Description:** fastq-tools a collection of small and efficient programs for performing some common and uncommon tasks with FASTQ files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fastqtools.html](https://hpc.nih.gov/apps/fastqtools.html)

### fastq_screen

- **Description:** FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fastq_screen.html](https://hpc.nih.gov/apps/fastq_screen.html)

### fastxtoolkit

- **Description:** The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fastxtoolkit.html](https://hpc.nih.gov/apps/fastxtoolkit.html)

### flappie

- **Description:** Basecall Fast5 reads using flip-flop basecalling.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/flappie.html](https://hpc.nih.gov/apps/flappie.html)

### flashpca

- **Description:** FlashPCA performs fast principal component analysis (PCA) of single nucleotide polymorphism (SNP) data, similar to smartpca from EIGENSOFT (http://www.hsph.harvard.edu/alkes-price/software/) and shellfish (https://github.com/dandavison/shellfish). FlashPCA is based on the https://github.com/yixuan/spectra/ library.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/flashpca.html](https://hpc.nih.gov/apps/flashpca.html)

### Flexbar

- **Description:** Flexbar preprocesses high-throughput sequencing data efficiently. It demultiplexes barcoded runs and removes adapter sequences. Moreover, trimming and filtering features are provided. Flexbar increases read mapping rates and improves genome and transcriptome assemblies. It supports next-generation sequencing data in fasta and fastq format, e.g. from Illumina and the Roche 454 platform
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/flexbar.html](https://hpc.nih.gov/apps/flexbar.html)

### flye

- **Description:** Fast and accurate de novo assembler for single molecule sequencing reads
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/flye.html](https://hpc.nih.gov/apps/flye.html)

### fqtools

- **Description:** Tools for manipulating fastq files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fqtools.html](https://hpc.nih.gov/apps/fqtools.html)

### freebayes

- **Description:** Bayesian haplotype-based polymorphism discovery and genotyping
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/freebayes.html](https://hpc.nih.gov/apps/freebayes.html)

### freec

- **Description:** Control-FREEC is a tool for detection of copy-number changes and allelic imbalances (including LOH) using deep-sequencing data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/freec.html](https://hpc.nih.gov/apps/freec.html)

### fusioncatcher

- **Description:** FusionCatcher searches for novel/known somatic fusion genes, translocations, and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS platforms like Solexa/HiSeq/NextSeq/MiSeq) from diseased samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fusioncatcher.html](https://hpc.nih.gov/apps/fusioncatcher.html)

### GangSTR

- **Description:** GangSTR is a tool for genome-wide profiling tandem repeats from short reads
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GangSTR.html](https://hpc.nih.gov/apps/GangSTR.html)

### GATK

- **Description:** GATK, from the Broad Institute, is a structured software library that makes writing efficient analysis tools using next-generation sequencing data very easy, and second it's a suite of tools for working with human medical resequencing projects such as 1000 Genomes and The Cancer Genome Atlas. These tools include things like a depth of coverage analyzers, a quality score recalibrator, a SNP/indel caller and a local realigner.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GATK.html](https://hpc.nih.gov/apps/GATK.html)

### gem

- **Description:** High resolution peak calling and motif discovery for ChIP-seq and ChIP-exo data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gem.html](https://hpc.nih.gov/apps/gem.html)

### Gemini

- **Description:** GEMINI (GEnome MINIng) is designed to be a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample genotypes, and useful genome annotations into an integrated database framework, GEMINI provides a simple, flexible, yet very powerful system for exploring genetic variation for for disease and population genetics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gemini.html](https://hpc.nih.gov/apps/gemini.html)

### GEMMA

- **Description:** GEMMA is the software implementing the Genome-wide Efficient Mixed Model Association algorithm for a standard linear mixed model and some of its close relatives for genome-wide association studies (GWAS).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GEMMA.html](https://hpc.nih.gov/apps/GEMMA.html)

### Genome Browser

- **Description:** The Genome Browser Mirror Fragments is a mirror of the UCSC Genome Browser. The URL is https://hpcnihapps.cit.nih.gov/genome. Users can also access the MySQL databases, supporting files directly, and a huge number of associated executables.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Genome_Browser.html](https://hpc.nih.gov/apps/Genome_Browser.html)

### genomestrip

- **Description:** Genome STRiP (Genome STRucture In Populations) is a suite of tools for discovering and genotyping structural variations using sequencing data. The methods are designed to detect shared variation using data from multiple individuals.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/genomestrip.html](https://hpc.nih.gov/apps/genomestrip.html)

### genrich

- **Description:** Genrich is a peak-caller for genomic enrichment assays (e.g. ChIP-seq, ATAC-seq). It analyzes alignment files generated following the assay and produces a file detailing peaks of significant enrichment.

### GeoMX NGS Pipeline

- **Description:** The GeoMx NGS Pipeline, processes RNA-sequencing files (FASTQ files) from Illumina sequencers according to parameters defined in the Configuration File (which is generated from the GeoMx DSP run). The Pipeline processes information from these files and outputs .dcc files, which can then be uploaded to the GeoMx DSP system for data analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/geomx_ngs_pipeline.html](https://hpc.nih.gov/apps/geomx_ngs_pipeline.html)

### Gistic

- **Description:** Facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gistic.html](https://hpc.nih.gov/apps/gistic.html)

### gmap-gsnap

- **Description:** A Genomic Mapping and Alignment Programs
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gmap-gsnap.html](https://hpc.nih.gov/apps/gmap-gsnap.html)

### gridss

- **Description:** GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements. GRIDSS includes a genome-wide break-end assembler, as well as a structural variation caller for Illumina sequencing data. GRIDSS calls variants based on alignment-guided positional de Bruijn graph genome-wide break-end assembly, split read, and read pair evidence.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gridss.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gridss.html)

### gtex_rnaseq

- **Description:** This module makes available the tools used in the GTEX RNA-Seq pipeline.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gtex_rnaseq.html](https://hpc.nih.gov/apps/gtex_rnaseq.html)

### gtool

- **Description:** GTOOL is a program for transforming sets of genotype data for use with the programs SNPTEST and IMPUTE.

GTOOL can be used to (a) generate subsets of genotype data,
(b) to convert genotype data between the PED file format and the FILE FORMAT used by SNPTEST and IMPUTE.

### guppy

- **Description:** Local accelerated basecalling for Nanopore data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/guppy.html](https://hpc.nih.gov/apps/guppy.html)

### Hail

- **Description:** Hail is an open-source, scalable framework for exploring and analyzing genomic data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Hail.html](https://hpc.nih.gov/apps/Hail.html)

### hap-ibd

- **Description:** detects identity-by-descent (IBD) segments and homozygosity-by-descent (HBD) segments
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hap-ibd.html](https://hpc.nih.gov/apps/hap-ibd.html)

### hap.py

- **Description:** A set of programs based on htslib to benchmark variant calls against gold standard truth datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hap.py.html](https://hpc.nih.gov/apps/hap.py.html)

### hgvs

- **Description:** The hgvs package provides a Python library to facilitate the use of genome, transcript, and protein variants that are represented using the Human Genome Variation Society (varnomen) recommendations. To use, type module load hgvs prior to calling python.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hgvs.html](https://hpc.nih.gov/apps/hgvs.html)

### hicexplorer

- **Description:** Tools to process, normalize and visualize Hi-C data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/hicexplorer.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/hicexplorer.html)

### hichipper

- **Description:** hichipper is a preprocessing and QC pipeline for HiChIP data. This package takes output from a HiC-Pro run and a sample manifest file (.yaml) that coordinates optional high-quality peaks (identified through ChIP-Seq) and restriction fragment locations (see folder here) as input and produces output that can be used to 1) determine library quality, 2) identify and characterize DNA loops and 3) interactively visualize loops.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hichipper.html](https://hpc.nih.gov/apps/hichipper.html)

### hicpro

- **Description:** HiC-Pro: An optimized and flexible pipeline for Hi-C data processing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hicpro.html](https://hpc.nih.gov/apps/hicpro.html)

### hint

- **Description:** a computational method to detect CNVs and Translocations from Hi-C data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hint.html](https://hpc.nih.gov/apps/hint.html)

### hipstr

- **Description:** Tool for genotyping short tandom repeats from Illumina sequencing data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hipstr.html](https://hpc.nih.gov/apps/hipstr.html)

### hisat

- **Description:** HISAT is a fast and sensitive spliced alignment program which uses Hierarchical Indexing for Spliced Alignment of Transcripts.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hisat.html](https://hpc.nih.gov/apps/hisat.html)

### HLA-LA

- **Description:** Fast HLA type inference from whole-genome data. Previously known as HLA-PRG-LA.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/HLA-LA.html](https://hpc.nih.gov/apps/HLA-LA.html)

### HMMRATAC

- **Description:** HMMRATAC peak caller for ATAC-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/HMMRATAC.html](https://hpc.nih.gov/apps/HMMRATAC.html)

### homer

- **Description:** HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and ChIP-Seq analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/homer.html](https://hpc.nih.gov/apps/homer.html)

### hrdetect

- **Description:** Hrdetect is a tool for detection and analysis of homologous recombination deficiency signatures.
  It was developed to accurately detect BRCA1/BRCA2-deficient samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hrdetect.html](https://hpc.nih.gov/apps/hrdetect.html)

### htgts

- **Description:** High-Throughput Genome-Wide Translocation Sequencing pipeline
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/htgts.html](https://hpc.nih.gov/apps/htgts.html)

### htseq

- **Description:** HTSeq is a Python package that provides infrastructure to process data from high-throughput sequencing assays.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/htseq.html](https://hpc.nih.gov/apps/htseq.html)

### humann

- **Description:** HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/humann.html](https://hpc.nih.gov/apps/humann.html)

### icgc-get

- **Description:** icgc-get provides a unified interface to the many sources of data from the International Cancer Genome Consortium.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/icgc-get.html](https://hpc.nih.gov/apps/icgc-get.html)

### IDBA

- **Description:** IDBA is a practical iterative De Bruijn Graph De Novo Assembler for sequence assembly in bioinfomatics. Most assemblers based on de Bruijn graph build a de Bruijn graph with a specific k to perform the assembling task. For all of them, it is very crucial to find a specific value of k. If k is too large, there will be a lot of gap problems in the graph. If k is too small, there will a lot of branch problems. IDBA uses not only one specific k but a range of k values to build the iterative de Bruijn graph. It can keep all the information in graphs with different k values.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/idba.html](https://hpc.nih.gov/apps/idba.html)

### idep

- **Description:** iDEP (integrated Differential Expression and Pathway analysis) is shiny application for analyzing RNA-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/idep.html](https://hpc.nih.gov/apps/idep.html)

### iDiffIR

- **Description:** iDiffIR is a tool for identifying differential IR from RNA-seq data.
  It accepts any sorted, indexed BAM file for single- or paired-end reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iDiffIR.html](https://hpc.nih.gov/apps/iDiffIR.html)

### IDR

- **Description:** The IDR (Irreproducible Discovery Rate) framework is a uniﬁed approach to measure the reproducibility of ﬁndings identiﬁed from replicate experiments and provide highly stable thresholds based on reproducibility. The IDR method compares a pair of ranked lists of identifications (such as ChIP-seq peaks).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/idr.html](https://hpc.nih.gov/apps/idr.html)

### IGV

- **Description:** The Integrative Genomics Viewer is a high-performance visualization tool for interactive exploration of large, integrated genomic datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/IGV.html](https://hpc.nih.gov/apps/IGV.html)

### IGVTools

- **Description:** IGVTools provides utilities for working with ascii file formats used by the Integrated Genome Viewer. The files can be sorted, tiled, indexed, and counted.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/IGV.html](https://hpc.nih.gov/apps/IGV.html)

### IMPUTE

- **Description:** Impute is a program for estimating ("imputing") unobserved genotypes in SNP association studies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/IMPUTE.html](https://hpc.nih.gov/apps/IMPUTE.html)

### InterVar

- **Description:** In 2015, the American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology (AMP) published
  updated standards and guidelines for the clinical interpretation of sequence variants with respect to human diseases on the basis
  of 28 criteria. However, variability between individual interpreters can be extensive because of reasons such as the different understandings of these guidelines and the lack of standard algorithms for implementing them, yet computational tools for semi-automated variant interpretation are not available. To address these problems, InterVar implements these criteria to help human reviewers interpret the clinical significance of variants. InterVar can take a pre-annotated or VCF file as input and generate automated interpretation on 18 criteria.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/InterVar.html](https://hpc.nih.gov/apps/InterVar.html)

### iSAAC

- **Description:** iSAAC is an ultrafast DNA sequence aligner (Isaac Genome Alignment Software) that takes advantage of high-memory hardware (>48 GB) and variant caller (Isaac Variant Caller)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iSAAC.html](https://hpc.nih.gov/apps/iSAAC.html)

### iva

- **Description:** IVA is a de novo assembler designed to assemble virus genomes that have no repeat sequences, using Illumina read pairs sequenced from mixed populations at extremely high and variable depth.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iva.html](https://hpc.nih.gov/apps/iva.html)

### iVar

- **Description:** iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. Additional tools for metagenomic sequencing are actively being incorporated into iVar. While each of these functions can be accomplished using existing tools, iVar contains an intersection of functionality from multiple tools that are required to call iSNVs and consensus sequences from viral sequencing data across multiple replicates.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iVar.html](https://hpc.nih.gov/apps/iVar.html)

### Jacquard

- **Description:** Jacquard makes it easier to analyze multi-patient tumor-normal datasets especially when using multiple vairant callers. It transforms the dialects of different variant callers into a controlled vocabulary of tags with a consistent representation of values. Furthermore, it intelligently merges VCFs from different patients and callers to create a single, unified VCF across your dataset.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/jacquard.html](https://hpc.nih.gov/apps/jacquard.html)

### JAMM

- **Description:** JAMM is a peak finder for NGS datasets (ChIP-Seq, ATAC-Seq, DNase-Seq..etc.) that can integrate replicates and assign peak boundaries accurately. JAMM is applicable to both broad and narrow datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/JAMM.html](https://hpc.nih.gov/apps/JAMM.html)

### Juicer

- **Description:** A One-Click System for Analyzing Loop-Resolution Hi-C Experiments
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/juicer.html](https://hpc.nih.gov/apps/juicer.html)

### jvarkit

- **Description:** Java tools for bioinformatics
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/jvarkit.html](https://hpc.nih.gov/apps/jvarkit.html)

### kaiju

- **Description:** Kaiju is a program for the taxonomic classification of high-throughput sequencing reads, e.g., Illumina or Roche/454, from whole-genome sequencing of metagenomic DNA.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kaiju.html](https://hpc.nih.gov/apps/kaiju.html)

### Katuali

- **Description:** Katuali is a flexible consensus pipeline implemented in Snakemake to basecall, assemble, and polish Oxford Nanopore Technologies' sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Katuali.html](https://hpc.nih.gov/apps/Katuali.html)

### KMC

- **Description:** KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kmc.html](https://hpc.nih.gov/apps/kmc.html)

### KmerGenie

- **Description:** KmerGenie estimates the best k-mer length for genome de novo assembly.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kmergenie.html](https://hpc.nih.gov/apps/kmergenie.html)

### kneaddata

- **Description:** KneadData is a tool designed to perform quality control on metagenomic and metatranscriptomic sequencing data, especially data from microbiome experiments.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kneaddata.html](https://hpc.nih.gov/apps/kneaddata.html)

### kraken

- **Description:** Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kraken.html](https://hpc.nih.gov/apps/kraken.html)

### KrakenUniq

- **Description:** KrakenUniq is a taxonomic sequence classifier that assigns taxonomic
  labels to short DNA reads. It combines the fast k-mer-based classification
  of Kraken
  (Wood and Salzberg, Genome Biology 2014) with an efficient algorithm for
  assessing the coverage of unique k-mers found in each species in a dataset.
  On various test datasets, KrakenUniq gives
  better recall and precision than other methods and effectively classifies and distinguishes pathogens with low
  abundance from false positives in infectious disease samples. By using the probabilistic cardinality estimator
  HyperLogLog, KrakenUniq runs as fast as Kraken and requires little additional memory.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/KrakenUniq.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/KrakenUniq.html)

### laser

- **Description:** LASER is a program to estimate individual ancestry by directly analyzing shotgun sequence reads without calling genotypes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/laser.html](https://hpc.nih.gov/apps/laser.html)

### leafcutter

- **Description:** Leafcutter quantifies RNA splicing variation using short-read RNA-seq data. The core idea is to leverage spliced reads (reads that span an intron) to quantify (differential) intron usage across samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/leafcutter.html](https://hpc.nih.gov/apps/leafcutter.html)

### lefse

- **Description:** LEfSe (Linear discriminant analysis Effect Size) determines the features (organisms, clades, operational taxonomic units, genes, or functions) most likely to explain differences between classes by coupling standard tests for statistical significance with additional tests encoding biological consistency and effect relevance.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lefse.html](https://hpc.nih.gov/apps/lefse.html)

### locuszoom

- **Description:** LocusZoom is designed to facilitate viewing of local association results together with useful information about a locus, such as the location and orientation of the genes it includes, linkage disequilibrium coefficients and local estimates of recombination rates
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/locuszoom.html](https://hpc.nih.gov/apps/locuszoom.html)

### lofreq

- **Description:** LoFreq is a fast and sensitive variant-caller for inferring SNVs and indels from next-generation sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lofreq.html](https://hpc.nih.gov/apps/lofreq.html)

### lumpy

- **Description:** A probabilistic framework for structural variant discovery.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lumpy.html](https://hpc.nih.gov/apps/lumpy.html)

### mach2qtl

- **Description:** mach2qtl uses dosages/posterior probabilities inferred with MACH as predictors in a linear regression to test association with a quantitative trait
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mach2qtl.html](https://hpc.nih.gov/apps/mach2qtl.html)

### macs

- **Description:** Model-based Analysis of ChIP-Seq (MACS) on short reads sequencers such as Genome Analyzer (Illumina / Solexa). MACS empirically models the length of the sequenced ChIP fragments, which tends to be shorter than sonication or library construction size estimates, and uses it to improve the spatial resolution of predicted binding sites. MACS also uses a dynamic Poisson distribution to effectively capture local biases in the genome sequence, allowing for more sensitive and robust prediction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/macs.html](https://hpc.nih.gov/apps/macs.html)

### mafft

- **Description:** Multiple alignment program for amino acid or nucleotide sequences

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mafft.html](https://hpc.nih.gov/apps/mafft.html)

### MAGeCK

- **Description:** MAGeCK is Model-based Analysis of Genome-wide CRISPR/Cas9 Knockout (MAGeCK) method for prioritizing
  single-guide RNAs, genes and pathways in genome-scale CRISPR/Cas9 knockout screens. It demonstrates better performance compared with other methods, identifies both positively and negatively selected genes
  simultaneously, and reports robust results across different experimental conditions.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MAGeCK.html](https://hpc.nih.gov/apps/MAGeCK.html)

### mageck-vispr

- **Description:** MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9 screens.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mageck-vispr.html](https://hpc.nih.gov/apps/mageck-vispr.html)

### MAGGIE

- **Description:** MAGGIE provides a framework for identifying DNA sequence motifs mediating transcription factor binding and function.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MAGGIE.html](https://hpc.nih.gov/apps/MAGGIE.html)

### magicblast

- **Description:** Magic-BLAST is a tool for mapping large next-generation RNA or DNA sequencing runs against a whole genome or transcriptome. Each alignment optimizes a composite score, taking into account simultaneously the two reads of a pair, and in case of RNA-seq, locating the candidate introns and adding up the score of all exons. This is very different from other versions of BLAST, where each exon is scored as a separate hit and read-pairing is ignored.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/magicblast.html](https://hpc.nih.gov/apps/magicblast.html)

### MAJIQ

- **Description:** Modeling Alternative Junction Inclusion Quantification. MAJIQ and Voila are two software packages that together define, quantify, and visualize local splicing variations (LSV) from RNA-Seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/majiq.html](https://hpc.nih.gov/apps/majiq.html)

### manorm

- **Description:** MAnorm is for quantitative comparison of ChIP-Seq data sets describing transcription factor binding sites and epigenetic modifications. The quantitative binding differences inferred by MAnorm showed strong correlation with both the changes in expression of target genes and the binding of cell type-specific regulators.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/manorm.html](https://hpc.nih.gov/apps/manorm.html)

### manta

- **Description:** Structural variant and indel caller for mapped sequencing data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/manta.html](https://hpc.nih.gov/apps/manta.html)

### maps

- **Description:** a set of multiple scripts used to analyze PLAC-Seq and HiChIP data.

### mapsplice

- **Description:** Accurate mapping of RNA-seq reads for splice junction discovery
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mapsplice.html](https://hpc.nih.gov/apps/mapsplice.html)

### mash

- **Description:** mash is a command line tool and library to provide fast genome and metagenome distance estimation using MinHash. Only command line tool is installed
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mash.html](https://hpc.nih.gov/apps/mash.html)

### medaka

- **Description:** medaka is a tool to create a consensus sequence from nanopore sequencing data. This task is performed using neural networks applied from a pileup of individual sequencing reads against a draft assembly.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/medaka.html](https://hpc.nih.gov/apps/medaka.html)

### medusa

- **Description:** A draft genome scaffolder that uses multiple reference genomes in a graph-based approach.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/medusa.html](https://hpc.nih.gov/apps/medusa.html)

### MEGAHIT

- **Description:** MEGAHIT is a single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph (SdBG) to achieve low memory assembly. MEGAHIT can optionally utilize a CUDA-enabled GPU to accelerate its SdBG construction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/megahit.html](https://hpc.nih.gov/apps/megahit.html)

### megalodon

- **Description:** Megalodon is a research command line tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads by anchoring the information rich basecalling neural network output to a reference genome/transcriptome.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/megalodon.html](https://hpc.nih.gov/apps/megalodon.html)

### MEGAN

- **Description:** MEtaGenome ANalyzer that takes a file of reads and a Blast output from comparison against a reference genome, and automatically calculate a taxonomic classification of the reads and if desired, a functional classification.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MEGAN.html](https://hpc.nih.gov/apps/MEGAN.html)

### Meryl

- **Description:** Meryl: a genomic k-mer counter (and sequence utility) with nice features. It is built into the Celera Assembler
  and is also available as a stand-alone application.
  Meryl uses a sorting-based approach that sorts the k-mers in lexicographical order.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Meryl.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Meryl.html)

### metabat

- **Description:** MetaBAT: A robust statistical framework for reconstructing genomes from metagenomic data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/metabat.html](https://hpc.nih.gov/apps/metabat.html)

### metal

- **Description:** The METAL software is designed to facilitate meta-analysis of large datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/metal.html](https://hpc.nih.gov/apps/metal.html)

### metaphlan

- **Description:** MetaPhlAn is a computational tool for profiling the composition of microbial communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now possible to perform accurate strain-level microbial profiling.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/metaphlan.html](https://hpc.nih.gov/apps/metaphlan.html)

### minialign

- **Description:** fast and accurate alignment tool for PacBio and Nanopore long reads
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/minialign.html](https://hpc.nih.gov/apps/minialign.html)

### miniasm

- **Description:** Ultrafast de novo assembly for long noisy reads (though having no consensus step)

### minimac

- **Description:** minimac is a low memory, computationally efficient implementation of the MaCH algorithm for genotype imputation. It is designed to work on phased genotypes and can handle very large reference panels with hundreds or thousands of haplotypes. 'mini' refers to the low amount of computational resources it needs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/minimac.html](https://hpc.nih.gov/apps/minimac.html)

### minimap2

- **Description:** Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/minimap2.html](https://hpc.nih.gov/apps/minimap2.html)

### mirdeep2

- **Description:** miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mirdeep.html](https://hpc.nih.gov/apps/mirdeep.html)

### misopy

- **Description:** MISO (Mixture-of-Isoforms) is a probabilistic framework that quantitates the expression level of alternatively spliced genes from RNA-Seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/misopy.html](https://hpc.nih.gov/apps/misopy.html)

### mitosuite

- **Description:** mitosuite is a graphical tool for human mitochondrial genome profiling in massively parallel sequencing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mitosuite.html](https://hpc.nih.gov/apps/mitosuite.html)

### mixcr

- **Description:** MiXCR is a universal software for fast and accurate analysis of T- and B- cell receptor repertoire sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mixcr.html](https://hpc.nih.gov/apps/mixcr.html)

### mixer

- **Description:** MiXeR is Causal Mixture Model for GWAS summary statistics. The version(1.3) contains a Python port of MiXeR, wrapping the C/C++ core. Also data preprocessing code sumstats.py is included too.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mixer.html](https://hpc.nih.gov/apps/mixer.html)

### MonoVar

- **Description:** Monovar is a statistical method for detecting and genotyping single-nucleotide variants in single-cell data. It takes onto account an allelic dropout, false-positive errors and a nonuniformity of coverage.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MonoVar.html](https://hpc.nih.gov/apps/MonoVar.html)

### MonSTR

- **Description:** Toolkit for calling and analyzing de novo STR
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MonSTR.html](https://hpc.nih.gov/apps/MonSTR.html)

### morgan

- **Description:** MORGAN (Monte Carlo Genetic Analysis)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/morgan.html](https://hpc.nih.gov/apps/morgan.html)

### mosaicforecast

- **Description:** a machine learning method that leverages read-based phasing and read-level features to accurately detect mosaic SNVs (SNPs, small indels) from NGS data.It builds on existing algorithms to achieve a multifold increase in specificity.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mosaicforecast.html](https://hpc.nih.gov/apps/mosaicforecast.html)

### mosdepth

- **Description:** Fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mosdepth.html](https://hpc.nih.gov/apps/mosdepth.html)

### mscentipede

- **Description:** msCentipede is an algorithm for accurately inferring transcription factor binding sites using chromatin accessibility data (Dnase-seq, ATAC-seq)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mscentipede.html](https://hpc.nih.gov/apps/mscentipede.html)

### mtoolbox

- **Description:** A bioinformatics pipeline aimed at the analysis of mitochondrial DNA (mtDNA) in high throughput sequencing studies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mtoolbox.html](https://hpc.nih.gov/apps/mtoolbox.html)

### multiqc

- **Description:** aggregates results for various frequently used bioinformatics tools across multiple samples into a nice visual report
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/multiqc.html](https://hpc.nih.gov/apps/multiqc.html)

### MuSE

- **Description:** MuSE is an approach to somatic variant calling based on the F81 Markov substitution model for molecular evolution, which models the evolution of the reference allele to the allelic composition of the matched tumor and normal tissue at each genomic locus.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://bioinformatics.mdanderson.org/main/MuSE](https://hpc.nih.gov/apps/http://bioinformatics.mdanderson.org/main/MuSE)

### muTect

- **Description:** MuTect is a method developed at the Broad Institute for the reliable and accurate identification of somatic point mutations in next generation sequencing data of cancer genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/muTect.html](https://hpc.nih.gov/apps/muTect.html)

### MutSig

- **Description:** MutSig analyzes lists of mutations discovered in DNA sequencing, to identify genes that were mutated more often than expected by chance given background mutation processes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MutSig.html](https://hpc.nih.gov/apps/MutSig.html)

### nanopolish

- **Description:** nanopolish is a software package for signal-level analysis of Oxford Nanopore sequencing data. Nanopolish can calculate an improved consensus sequence for a draft genome assembly, detect base modifications, call SNPs and indels with respect to a reference genome and more (see Nanopolish modules, below).

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/nanopolish.html](https://hpc.nih.gov/apps/nanopolish.html)

### neusomatic

- **Description:** NeuSomatic is based on deep convolutional neural networks for accurate somatic mutation detection. With properly trained models, it can robustly perform across sequencing platforms, strategies, and conditions. NeuSomatic summarizes and augments sequence alignments in a novel way and incorporates multi-dimensional features to capture variant signals effectively. It is not only a universal but also accurate somatic mutation detection method.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/neusomatic.html](https://hpc.nih.gov/apps/neusomatic.html)

### NGMLR

- **Description:** NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore (standard and ultra-long) to a reference genome with a focus on reads that span structural variations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ngmlr.html](https://hpc.nih.gov/apps/ngmlr.html)

### ngscheckmate

- **Description:** NGSCheckMate validates nucleotide sequencing sample identity from fastq, BAM, or VCF files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ngscheckmate.html](https://hpc.nih.gov/apps/ngscheckmate.html)

### ngsplot

- **Description:** ngsplot is an easy-to-use global visualization tool for next-generation sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ngsplot/](https://hpc.nih.gov/apps/ngsplot/)

### novocraft

- **Description:** Package includes aligner for single-ended and paired-end reads from the Illumina Genome Analyser. Novoalign finds global optimum alignments using full Needleman-Wunsch algorithm with affine gap penalties.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/novocraft.html](https://hpc.nih.gov/apps/novocraft.html)

### ont-fast5-api

- **Description:** Tools to manipulate HDF5 files of the Oxford Nanopore .fast5 file format
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ont-fast5-api.html](https://hpc.nih.gov/apps/ont-fast5-api.html)

### opera-lg

- **Description:** OPERA (Optimal Paired-End Read Assembler) is a sequence assembly program. It uses information from paired-end/mate-pair/long reads to order and orient the intermediate contigs/scaffolds assembled in a genome assembly project, in a process known as Scaffolding.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/opera-lg.html](https://hpc.nih.gov/apps/opera-lg.html)

### parpipe

- **Description:** Complete analysis pipeline for PAR-CLIP data

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/parpipe.html](https://hpc.nih.gov/apps/parpipe.html)

### PartekFlow

- **Description:** Web interface designed specifically for the analysis needs of next generation sequencing applications including RNA, small RNA, and DNA sequencing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://partekflow.cit.nih.gov](https://hpc.nih.gov/apps/https://partekflow.cit.nih.gov)

### pbipa

- **Description:** Improved Phased Assembler (IPA) is the official PacBio software for HiFi genome assembly. IPA was designed to utilize the accuracy of PacBio HiFi reads to produce high-quality phased genome assemblies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pbipa.html](https://hpc.nih.gov/apps/pbipa.html)

### pbsuite

- **Description:** The PBSuite contains two projects created for analysis of Pacific Biosciences long-read sequencing data: PBHoney and PBJelly. PBHoney is an implementation of two variant-identification approaches designed to exploit the high mappability of long reads (i.e., greater than 10,000 bp). PBHoney considers both intra-read discordance and soft-clipped tails of long reads to identify structural variants. PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or long 454 reads in fasta format) to high-confidence draft assembles. PBJelly fills or reduces as many captured gaps as possible to produce upgraded draft genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pbsuite.html](https://hpc.nih.gov/apps/pbsuite.html)

### peakranger

- **Description:** A ChIP-Seq peak caller for narrow and broad peaks
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/peakranger.html](https://hpc.nih.gov/apps/peakranger.html)

### pear

- **Description:** PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. It is fully parallelized and can run with as low as just a few kilobytes of memory.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pear.html](https://hpc.nih.gov/apps/pear.html)

### peddy

- **Description:** peddy is used to compare sex and familial relationships given in a PED file with those inferred from a VCF file
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/peddy.html](https://hpc.nih.gov/apps/peddy.html)

### PennCNV

- **Description:** PennCNV is a free software tool for Copy Number Variation (CNV) detection
  from SNP genotyping arrays. Currently it can handle signal intensity data
  from Illumina and Affymetrix arrays. With appropriate preparation of file format,
  it can also handle other types of SNP arrays and oligonucleotide arrays.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PennCNV.html](https://hpc.nih.gov/apps/PennCNV.html)

### PEPATAC

- **Description:** PEPATAC is a robust pipeline for Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq) built on a loosely coupled modular framework.
  It may be easily applied to ATAC-seq projects of any size,
  from one-off experiments to large-scale sequencing projects.
  It is optimized on unique features of ATAC-seq data to be fast and accurate
  and provides several unique analytical approaches.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PEPATAC.html](https://hpc.nih.gov/apps/PEPATAC.html)

### PePr

- **Description:** PePr is a ChIP-Seq Peak-calling and Prioritization pipeline that uses a sliding window approach and models read counts across replicates and between groups with a negative binomial distribution.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PePr.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PePr.html)

### picard

- **Description:** Picard comprises Java-based command-line utilities that manipulate SAM files, and a Java API (SAM-JDK) for creating new programs that read and write SAM files. Both SAM text format and SAM binary (BAM) format are supported.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/picard.html](https://hpc.nih.gov/apps/picard.html)

### picrust

- **Description:** PICRUSt is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/picrust.html](https://hpc.nih.gov/apps/picrust.html)

### pindel

- **Description:** Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data. It uses a pattern growth approach to identify the breakpoints of these variants from paired-end short reads.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pindel.html](https://hpc.nih.gov/apps/pindel.html)

### pirs

- **Description:** Profile-based Illumina paired-end Read Simulator
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pirs.html](https://hpc.nih.gov/apps/pirs.html)

### plastid

- **Description:** Position-wise analysis of sequencing and genomics data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/plastid.html](https://hpc.nih.gov/apps/plastid.html)

### Platypus

- **Description:** tool for variant-detection in high-throughput sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/platypus.html](https://hpc.nih.gov/apps/platypus.html)

### plink

- **Description:** PLINK is whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/plink.html](https://hpc.nih.gov/apps/plink.html)

### plinkseq

- **Description:** library for the analysis of genetic variation data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/plinkseq.html](https://hpc.nih.gov/apps/plinkseq.html)

### pomoxis

- **Description:** Pomoxis comprises a set of basic bioinformatic tools tailored to nanopore sequencing. Notably tools are included for generating and analysing draft assemblies. Many of these tools are used by the research data analysis group at Oxford Nanopore Technologies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pomoxis.html](https://hpc.nih.gov/apps/pomoxis.html)

### porechop

- **Description:** Trim/demultiplex Oxford Nanopore reads
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/porechop.html](https://hpc.nih.gov/apps/porechop.html)

### poretools

- **Description:** Poretools is a toolkit for manipulating and exploring nanopore sequencing data sets. Poretools operates on individual FAST5 files, directory of FAST5 files, and tar archives of FAST5 files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/poretools.html](https://hpc.nih.gov/apps/poretools.html)

### preseq

- **Description:** predicting library complexity and genome coverage in high-throughput sequencing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/preseq.html](https://hpc.nih.gov/apps/preseq.html)

### PRINSEQ

- **Description:** PRINSEQ is a tool that generates summary statistics of sequence and quality data and that is used to filter, reformat and trim next-generation sequence data. It is particular designed for 454/Roche data, but can also be used for other types of sequence data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/prinseq.html](https://hpc.nih.gov/apps/prinseq.html)

### proseq

- **Description:** proseq-2.0 is a pipeline for preprocesses and alignment
  of run-on sequencing (PRO/GRO/ChRO-seq) data
  from Single-Read or Paired-End Illumina Sequencing

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/proseq.html](https://hpc.nih.gov/apps/proseq.html)

### PROVEAN

- **Description:** PROVEAN (Protein Variation Effect Analyzer) is a software tool which predicts whether an amino acid substitution or indel has an impact on the biological function of a protein.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PROVEAN.html](https://hpc.nih.gov/apps/PROVEAN.html)

### purge_haplotigs

- **Description:** purge_haplotigs is a pipeline to help with curating heterozygous diploid genome assemblies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/purge_haplotigs.html](https://hpc.nih.gov/apps/purge_haplotigs.html)

### pvactools

- **Description:** pVACtools is a cancer immunotherapy suite consisting of pVACseq, pVACfuse, pVACvector
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pvactools.html](https://hpc.nih.gov/apps/pvactools.html)

### pychopper

- **Description:** Pychopper v2 is a tool to identify, orient and trim full-length Nanopore cDNA reads. The tool is also able to rescue fused reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pychopper.html](https://hpc.nih.gov/apps/pychopper.html)

### pyclone

- **Description:** PyClone is statistical model and software tool designed to infer the prevalence of point mutations in heterogeneous cancer samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pyclone.html](https://hpc.nih.gov/apps/pyclone.html)

### pycoQC

- **Description:** pycoQC is a new tool to generate interactive quality control metrics and plots
  from basecalled nanopore reads
  or summary files generated by the basecallers Albacore, Guppy or MinKNOW.
  pycoQC has several novel features, including:

1. python support for creation of dynamic D3.js visualizations and interactive data exploration in Jupyter Notebooks;
2. simple command line interface to generate customizable interactive HTML reports; and
3. multiprocessing FAST5 feature extraction program to generate a summary file directly
   from FAST5 files.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pycoQC.html](https://hpc.nih.gov/apps/pycoQC.html)

### pyega3

- **Description:** A download client for the European Genome-phenome Archive (EGA). The EGA is designed to be a repository for all types of sequence and genotype experiments, including case-control, population, and family studies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pyega3.html](https://hpc.nih.gov/apps/pyega3.html)

### PyLOH

- **Description:** Deconvolving tumor purity and ploidy by integrating copy number alterations and loss of heterozygosity
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PyLOH.html](https://hpc.nih.gov/apps/PyLOH.html)

### qcat

- **Description:** qcat is Python command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/qcat.html](https://hpc.nih.gov/apps/qcat.html)

### QoRTs

- **Description:** QCTOOL is a command-line utility program for basic quality control of gwas datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/QoRTs/](https://hpc.nih.gov/apps/QoRTs/)

### QoRTs

- **Description:** The QoRTs software package is a fast, efficient, and portable multifunction toolkit designed to assist in the analysis, quality control, and data management of RNA-Seq datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/QoRTs/](https://hpc.nih.gov/apps/QoRTs/)

### qualimap

- **Description:** A platform-independednt application written in Java and R that provides both a GUI and a co
  mmand-line interface to facilitate the quality control of alignment sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/qualimap.html](https://hpc.nih.gov/apps/qualimap.html)

### quast

- **Description:** QUAST stands for QUality ASsessment Tool. The tool evaluates genome assemblies by computing various metrics. The package includes the general QUAST tool for genome assemblies, MetaQUAST, the extension for metagenomic datasets, and Icarus, interactive visualizer for these tools.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/quast.html](https://hpc.nih.gov/apps/quast.html)

### racon

- **Description:** Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.

### raremetal

- **Description:** RAREMETAL is a computationally efficient tool for meta-analysis of rare variants using sequencing or genotyping array data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/raremetal.html](https://hpc.nih.gov/apps/raremetal.html)

### Rcorrector

- **Description:** Rcorrector implements a k-mer based method to correct random sequencing errors
  in Illumina RNA-seq reads. Rcorrector uses a De Bruijn graph to compactly represent
  all trusted k-mers in the input reads. Unlike WGS read correctors,
  which use a global threshold to determine trusted k-mers, Rcorrector computes a local
  threshold at every position in a read.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Rcorrector.html](https://hpc.nih.gov/apps/Rcorrector.html)

### READemption

- **Description:** RNA-Seq pipeline including alignment, coverage tracks, quantitation, and differential expression analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/reademption.html](https://hpc.nih.gov/apps/reademption.html)

### RepeatMasker

- **Description:** RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns). On average, almost 50% of a human genomic DNA sequence currently will be masked by the program.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/repeatmasker.html](https://hpc.nih.gov/apps/repeatmasker.html)

### repeatmodeler

- **Description:** RepeatModeler is a de novo transposable element (TE) family identification and modeling package. RepeatModeler assists in automating the runs of the various algorithms given a genomic database, clustering redundant results, refining and classifying the families and producing a high quality library of TE families suitable for use with RepeatMasker and ultimately for submission to the Dfam database (http://dfam.org).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/repeatmodeler.html](https://hpc.nih.gov/apps/repeatmodeler.html)

### rgt

- **Description:** Regulatory Genomics Toolbox: Python library and set of tools for the integrative analysis of high throughput regulatory genomics data. http://www.regulatory-genomics.org
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rgt.html](https://hpc.nih.gov/apps/rgt.html)

### ricopili

- **Description:** RICOPILI stands for Rapid Imputation and COmputational PIpeLIne for GWAS.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ricopili.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ricopili.html)

### rilseq

- **Description:** RILseq computational protocol
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rilseq.html](https://hpc.nih.gov/apps/rilseq.html)

### rnaseqc

- **Description:** RNA-SeQC is a java program which computes a series of quality control metrics for RNA-seq data.

### rockhopper

- **Description:** Rockhopper is a comprehensive and user-friendly system for computational analysis of bacterial RNA-seq data. As input, Rockhopper takes RNA sequencing reads output by high-throughput sequencing technology (FASTQ, QSEQ, FASTA, SAM, or BAM files)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rockhopper.html](https://hpc.nih.gov/apps/rockhopper.html)

### ROSE

- **Description:** ROSE (Rank Ordering of Super-Enhancers) is tool for
  (1) creating stitched enhancers, and  
  (2) separating super-enhancers from typical enhancers.
  given sequencing data (.bam) and a file of previously identified constituent enhancers (.gff)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ROSE.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ROSE.html)

### RSD

- **Description:** Reciprocal Smallest Distance (RSD) is a pairwise orthology algorithm that uses global sequence alignment and maximum likelihood evolutionary distance between sequences to accurately detects orthologs between genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/RSD.html](https://hpc.nih.gov/apps/RSD.html)

### rsem

- **Description:** RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rsem.html](https://hpc.nih.gov/apps/rsem.html)

### rseqc

- **Description:** Rseqc comprehensively evaluate RNA-seq datasets generated from clinical tissues or other well annotated organisms such as mouse, fly and yeast.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rseqc.html](https://hpc.nih.gov/apps/rseqc.html)

### rtg-tools

- **Description:** variant detection for singletons, families, large pedigrees and populations, cancer, structural variant and CNV analysis, and microbial and metagenomic analysis
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rtg-tools.html](https://hpc.nih.gov/apps/rtg-tools.html)

### rvtests

- **Description:** Rare Variant tests is a flexible software package for genetic association studies. It is designed to support unrelated individual or related (family-based) individuals
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rvtests.html](https://hpc.nih.gov/apps/rvtests.html)

### sailfish

- **Description:** Sailfish is a tool for transcript quantification from RNA-seq data. It requires a set of target transcripts (either from a reference or de-novo assembly) to quantify. All that is needed to run sailfish is a fasta file containing your reference transcripts and a (set of) fasta/fastq file(s) containing your RNA-Seq reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sailfish.html](https://hpc.nih.gov/apps/sailfish.html)

### salmon

- **Description:** a tool for quantifying the expression of transcripts using RNA-seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/salmon.html](https://hpc.nih.gov/apps/salmon.html)

### SalmonTE

- **Description:** SalmonTE is an ultra-Fast and Scalable Quantification Pipeline of Transpose Element (TE) Abundances.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/salmonte.html](https://hpc.nih.gov/apps/salmonte.html)

### sambamba

- **Description:** Sambamba is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Current parallelised functionality is an important subset of samtools functionality, including view, index, sort, markdup, and depth.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sambamba.html](https://hpc.nih.gov/apps/sambamba.html)

### samblaster

- **Description:** samblaster is a program for marking duplicates and finding discordant/split read pairs in read-id grouped paired-end SAM files. When marking duplicates, samblaster will use about 20MB per 1M read pairs. In a read-id grouped SAM file all alignments for a read-id (QNAME) are continuous. Aligners naturally produce such files. They can also be created by sorting a SAM file by read-id.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/samblaster.html](https://hpc.nih.gov/apps/samblaster.html)

### samtools

- **Description:** The samtools package now provides samtools, bcftools, tabix, and the underlying htslib library.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/samtools.html](https://hpc.nih.gov/apps/samtools.html)

### scallop

- **Description:** Scallop is a reference-based transcript assembler.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scallop.html](https://hpc.nih.gov/apps/scallop.html)

### scalpel

- **Description:** Bioinformatics pipeline for discovery of genetic variants from NGS reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scalpel.html](https://hpc.nih.gov/apps/scalpel.html)

### ScanIndel

- **Description:** ScanIndel is a python program to detect indels (insertions and deletions) from NGS data by re-align and de novo assemble soft clipped reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scanindel.html](https://hpc.nih.gov/apps/scanindel.html)

### scanpy

- **Description:** Scanpy is a scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scanpy.html](https://hpc.nih.gov/apps/scanpy.html)

### Scramble

- **Description:** Scramble is a mobile element insertion (MEI) detection tool. It identifies clusters of soft clipped reads
  in a BAM file, builds consensus sequences, aligns to representative L1Ta, AluYa5, and SVA-E sequences, and
  outputs MEI calls.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Scramble.html](https://hpc.nih.gov/apps/Scramble.html)

### segemehl

- **Description:** Segemehl is a short read aligner that allows local alignment and can align reads obtained after bisulfite treatment.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/segemehl.html](https://hpc.nih.gov/apps/segemehl.html)

### seqan

- **Description:** SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences with the focus on biological data. It applies a unique generic design that guarantees high performance, generality, extensibility, and integration with other libraries. This package also contains a suite of apps, including Fiona, Gustaf, Mason, RazerS 3, Yara, SeqAn T-Coffee, Stellar, and searchjoin.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/seqan.html](https://hpc.nih.gov/apps/seqan.html)

### seqkit

- **Description:** A cross-platform toolkit for FASTA/Q file manipulation
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/seqkit.html](https://hpc.nih.gov/apps/seqkit.html)

### seqlinkage

- **Description:** SEQLinkage implements a collapsed haplotype pattern (CHP) method to generate markers from sequence data for linkage analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/seqlinkage.html](https://hpc.nih.gov/apps/seqlinkage.html)

### seqtk

- **Description:** seqtk is a toolkit for processing sequences in FASTA/Q formats
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/seqtk.html](https://hpc.nih.gov/apps/seqtk.html)

### sequenza-utils

- **Description:** Sequenza-utils is The supporting python library for the sequenza R package.

Sequenza is a project the estimate purity/ploidy and copy number alteration from tumor sequencing experiments. Sequenza-utils provide command lines programs to transform common NGS file type, such as BAM, pileup and VCF, to input files for the R package

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sequenza-utils.html](https://hpc.nih.gov/apps/sequenza-utils.html)

### sga

- **Description:** SGA (String Graph Assembler) is a de novo genome assembler based on the concept of string graphs. The major goal of SGA is to be very memory efficient, which is achieved by using a compressed representation of DNA sequence reads. It is based on Gene Myers' string graph formulation of assembly and uses the FM-index/Burrows-Wheeler transform to efficiently find overlaps between sequence reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sga.html](https://hpc.nih.gov/apps/sga.html)

### SGA-ICE

- **Description:** The script SGA-ICE (SGA-Iteratively Correcting Errors) implements iterative error correction by using modules from the String Graph Assembler (SGA).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SGA-ICE.html](https://hpc.nih.gov/apps/SGA-ICE.html)

### shapeit

- **Description:** SHAPEIT is a fast and accurate haplotype inference software
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/shapeit.html](https://hpc.nih.gov/apps/shapeit.html)

### shmlast

- **Description:** shmlast is a reimplementation of the Conditional Reciprocal Best Hits algorithm for finding potential orthologs between a transcriptome and a species-specific protein database. It uses the LAST aligner and the pydata stack to achieve much better performance while staying in the Python ecosystem.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/shmlast.html](https://hpc.nih.gov/apps/shmlast.html)

### shrimp

- **Description:** SHRiMP is a software package for aligning genomic reads against a target genome. It was primarily developed with the multitudinous short reads of next generation sequencing machines in mind, as well as Applied Biosystem's colourspace genomic representation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/shrimp.html](https://hpc.nih.gov/apps/shrimp.html)

### sicer

- **Description:** A clustering approach for identification of enriched domains from histone modification ChIP-Seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sicer.html](https://hpc.nih.gov/apps/sicer.html)

### sickle

- **Description:** A windowed adaptive trimming tool for FASTQ files using quality
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sickle.html](https://hpc.nih.gov/apps/sickle.html)

### SIFT

- **Description:** SIFT predicts whether an amino acid substitution affects protein function based on sequence homology and the physical properties of amino acids.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SIFT.html](https://hpc.nih.gov/apps/SIFT.html)

### slamdunk

- **Description:** SlamDunk is a novel, fully automated software tool for automated, robust, scalable and reproducible SLAMseq data analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/slamdunk.html](https://hpc.nih.gov/apps/slamdunk.html)

### smalt

- **Description:** SMALT efficiently aligns DNA sequencing reads with a reference genome.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/smalt.html](https://hpc.nih.gov/apps/smalt.html)

### smart

- **Description:** Specific Methylation Analysis and Report Tool (SMART) uses the signal from bisulfite sequencing experiments across multiple samples to identify genome segments with similar methylation secificities.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/smart.html](https://hpc.nih.gov/apps/smart.html)

### smrtanalysis

- **Description:** SMRT® Analysis is a bioinformatics software suite available for analysis of DNA sequencing data from Pacific Biosciences’ SMRT technology. Users can choose from a variety of analysis protocols that utilize PacBio® and third-party tools. Analysis protocols include de novo genome assembly, cDNA mapping, DNA base-modification detection, and long-amplicon analysis to determine phased consensus sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/smrtanalysis.html](https://hpc.nih.gov/apps/smrtanalysis.html)

### snaptools

- **Description:** A module for working with snap files in Python.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/snaptools.html](https://hpc.nih.gov/apps/snaptools.html)

### sniffles

- **Description:** Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+) using evidence from split-read alignments, high-mismatch regions, and coverage analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sniffles.html](https://hpc.nih.gov/apps/sniffles.html)

### snippy

- **Description:** snippy is a tool for rapid haploid variant calling and core genome alignment
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/snippy.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/snippy.html)

### snp2hla

- **Description:** SNP2HLA is a tool to impute amino acid polymorphisms and single nucleotide polymorphisms in human luekocyte antigenes (HLA) within the major histocompatibility complex (MHC) region in chromosome 6.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/snp2hla.html](https://hpc.nih.gov/apps/snp2hla.html)

### snpEff

- **Description:** snpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes (such as amino acid changes).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/snpEff.html](https://hpc.nih.gov/apps/snpEff.html)

### snptest

- **Description:** SNPTEST is a program for the analysis of single SNP association in genome-wide studies. The tests implemented include

  - Binary (case-control) phenotypes, single and multiple quantitative phenotypes
  - Bayesian and Frequentist tests
  - Ability to condition upon an arbitrary set of covariates
  - Various different methods for the dealing with imputed SNPs.

The program is designed to work seamlessly with the output of both the genotype calling program CHIAMO, the genotype imputation program IMPUTE and the program GTOOL.

### SomaticSeq

- **Description:** SomaticSeq is an ensemble approach to accurately detect somatic mutations. It incorporates multiple somatic mutation caller(s) to obtain a combined call set, and then uses machine learning to distinguish true mutations from false positives from that call set.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SomaticSeq.html](https://hpc.nih.gov/apps/SomaticSeq.html)

### somaticsniper

- **Description:** The purpose of this program is to identify single nucleotide positions that are different between tumor and normal (or, in theory, any two bam files). It takes a tumor bam and a normal bam and compares the two to determine the differences. It outputs a file in a format very similar to Samtools consensus format.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/somaticsniper.html](https://hpc.nih.gov/apps/somaticsniper.html)

### sortmeRNA

- **Description:** SortMeRNA is a local sequence alignment tool for filtering, mapping and clustering.
  The core algorithm is based on approximate seeds and allows for sensitive analysis
  of NGS reads. The main application of SortMeRNA is filtering rRNA
  from metatranscriptomic data. SortMeRNA takes as input a file of reads
  (fasta or fastq format) and one or multiple rRNA database file(s),
  and sorts apart aligned and rejected reads into two files specified by the user.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sortmeRNA.html](https://hpc.nih.gov/apps/sortmeRNA.html)

### spaceranger

- **Description:** 10x pipeline for processing Visium spatial RNA-seq data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/spaceranger.html](https://hpc.nih.gov/apps/spaceranger.html)

### spades

- **Description:** SPAdes – St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/spades.html](https://hpc.nih.gov/apps/spades.html)

### speedseq

- **Description:** SpeedSeq is a genome analysis platform designed for rapid whole-genome variant detection and interpretation
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/speedseq.html](https://hpc.nih.gov/apps/speedseq.html)

### spruce

- **Description:** SPRUCE (Somatic Phylogeny Reconstruction using Combinatorial Enumeration) is an algorithm for inferring the clonal evolution of single-nucleotide and copy-number variants given multi-sample bulk tumor sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/spruce.html](https://hpc.nih.gov/apps/spruce.html)

### sratoolkit

- **Description:** The NCBI SRA Toolkit enables reading ("dumping") of sequencing files from the SRA database and writing ("loading") files into the .sra format.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sratoolkit.html](https://hpc.nih.gov/apps/sratoolkit.html)

### SRST2

- **Description:** SRST2 is a a read mapping-based tool for rapid molecular typing of bacterial pathogens.
  It allows fast and accurate detection of genes, alleles and multi-locus
  sequence types (MLST) from WGS data. SRST2 is highly accurate and outperforms assembly-based methods in terms of both gene detection and allele assignment.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/srst2.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/srst2.html)

### stampy

- **Description:** Short read aligner
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/stampy.html](https://hpc.nih.gov/apps/stampy.html)

### STAR

- **Description:** Spliced Transcripts Alignment to a Reference
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/STAR.html](https://hpc.nih.gov/apps/STAR.html)

### STAR-Fusion

- **Description:** Transcript fusion detection
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/STAR.html](https://hpc.nih.gov/apps/STAR.html)

### STREAM

- **Description:** STREAM stands for Single-cell Trajectories Reconstruction, Exploration And Mapping ofomics data. It is an interactive pipeline capable of disentangling and visualizing
  complex branching trajectories from both single-cell transcriptomic and epigenomic data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/STREAM.html](https://hpc.nih.gov/apps/STREAM.html)

### strelka

- **Description:** Strelka is an analysis package designed to detect somatic SNVs and small indels from the aligned sequencing reads of matched tumor-normal samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/strelka.html](https://hpc.nih.gov/apps/strelka.html)

### stringtie

- **Description:** StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It is primarily a genome-guided transcriptome assembler, although it can borrow algorithmic techniques from de novo genome assembly to help with transcript assembly.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/stringtie.html](https://hpc.nih.gov/apps/stringtie.html)

### ST_pipeline

- **Description:** The Spatial Transcriptomics (ST) pipeline contains the tools and scripts
  needed to process and analyze the raw files
  generated with the Spatial Transcriptomics technology in FASTQ format
  to generate datasets for down-stream analysis. The ST pipeline can also be used
  to process single cell RNASeq data as long as a file with barcodes identifying each cell is provi
  ded.
  The ST Pipeline has been optimized for speed, robustness and it is very easy to use
  with many parameters to adjust all the settings.
  The ST Pipeline is fully parallel and has constant memory use.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/st_pipeline.html](https://hpc.nih.gov/apps/st_pipeline.html)

### subread

- **Description:** High-performance read alignment, quantification and mutation discovery
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/subread.html](https://hpc.nih.gov/apps/subread.html)

### supernova

- **Description:** Supernova generates highly-contiguous, phased, whole-genome de novo assemblies from a Chromium-prepared library.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/supernova.html](https://hpc.nih.gov/apps/supernova.html)

### suppa

- **Description:** Fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions

### SURVIVOR

- **Description:** SURVIVOR is a tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples, and includes various methods to reformat or summarize SVs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/survivor.html](https://hpc.nih.gov/apps/survivor.html)

### svclone

- **Description:** A computational method for inferring the cancer cell fraction of tumour structural variation from whole-genome sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/svclone.html](https://hpc.nih.gov/apps/svclone.html)

### sve

- **Description:** SVE is a python script based execution engine for Structural Variation (SV) detection and can be used for any levels of data inputs, raw FASTQs, aligned BAMs, or variant call format (VCFs), and generates a unified VCF as its output.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sve.html](https://hpc.nih.gov/apps/sve.html)

### svtk

- **Description:** Utilities for consolidating, filtering, resolving, and annotating structural variants.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/svtk.html](https://hpc.nih.gov/apps/svtk.html)

### svtyper

- **Description:** Svtyper is a Bayesian genotyper for structural variants.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/svtyper.html](https://hpc.nih.gov/apps/svtyper.html)

### tagit

- **Description:** Tag(ging) It(erative) of SNVs in multiple populations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tagit.html](https://hpc.nih.gov/apps/tagit.html)

### tailseeker

- **Description:** Tailseeker is the official pipeline for TAIL-seq, which measures poly(A) tail lengths and 3′-end modifications with Illumina SBS sequencers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tailseeker.html](https://hpc.nih.gov/apps/tailseeker.html)

### talon

- **Description:** TALON is a Python package for identifying and quantifying known and novel genes/isoforms in long-read transcriptome data sets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/talon.html](https://hpc.nih.gov/apps/talon.html)

### tappas

- **Description:** A comprehensive computational framework for the analysis of the functional impact of differential splicing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tappas.html](https://hpc.nih.gov/apps/tappas.html)

### taxonkit

- **Description:** A Cross-platform and Efficient NCBI Taxonomy Toolkit

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/taxonkit.html](https://hpc.nih.gov/apps/taxonkit.html)

### telescope

- **Description:** Single locus resolution of Transposable ELEment expression. Telescope estimates transposable element expression (retrotranscriptome) resolved to specific genomic locations. It directly addresses uncertainty in fragment assignment by reassigning ambiguously mapped fragments to the most probable source transcript as determined within a Bayesian statistical model.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/telescope.html](https://hpc.nih.gov/apps/telescope.html)

### telseq

- **Description:** TelSeq is a software that estimates telomere length from whole genome sequencing data (BAMs).

### tetoolkit

- **Description:** A package for including transposable elements in differential enrichment analysis of sequencing datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tetoolkit.html](https://hpc.nih.gov/apps/tetoolkit.html)

### THetA

- **Description:** Tumor Heterogeneity Analysis (THetA) is an algorithm used to estimate tumor purity and clonal/subclonal copy number aberrations simultaneously from high-throughput DNA sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/theta.html](https://hpc.nih.gov/apps/theta.html)

### TMAP

- **Description:** TMAP is a fast and accurate alignment software for short and long nucleotide sequences produced by Ion Torrent sequencing technologies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TMAP.html](https://hpc.nih.gov/apps/TMAP.html)

### tombo

- **Description:** a suite of tools primarily for the identification of modified nucleotides from nanopore sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tombo.html](https://hpc.nih.gov/apps/tombo.html)

### tophat

- **Description:** TopHat is a fast splice junction mapper for RNA-Seq reads. It aligns RNA-Seq reads to mammalian-sized genomes using the ultra high-throughput short read aligner Bowtie, and then analyzes the mapping results to identify splice junctions between exons.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tophat.html](https://hpc.nih.gov/apps/tophat.html)

### TPMCalculator

- **Description:** TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TPMCalculator.html](https://hpc.nih.gov/apps/TPMCalculator.html)

### TransDecoder

- **Description:** TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TransDecoder.html](https://hpc.nih.gov/apps/TransDecoder.html)

### transrate

- **Description:** Transrate is software for de-novo transcriptome assembly quality analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/transrate.html](https://hpc.nih.gov/apps/transrate.html)

### transvar

- **Description:** TransVar is a versatile annotator for 3-way conversion and annotation among genomic characterization(s) of mutations and transcript-dependent annotation(s).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/transvar.html](https://hpc.nih.gov/apps/transvar.html)

### trimgalore

- **Description:** Consistent quality and adapter trimming for RRBS or standard FastQ files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/trimgalore.html](https://hpc.nih.gov/apps/trimgalore.html)

### trimmomatic

- **Description:** Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/trimmomatic.html](https://hpc.nih.gov/apps/trimmomatic.html)

### trinity

- **Description:** Trinity, developed at the Broad Institute and the Hebrew University of Jerusalem, represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/trinity.html](https://hpc.nih.gov/apps/trinity.html)

### Trinotate

- **Description:** Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/trinotate.html](https://hpc.nih.gov/apps/trinotate.html)

### tRNAscan-SE

- **Description:** tRNAscan-SE 2.0 has advanced the state-of-the-art methodology in tRNA gene detection and functional prediction, captured by rich new content of the companion Genomic tRNA Database
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tRNAscan-SE.html](https://hpc.nih.gov/apps/tRNAscan-SE.html)

### truffle

- **Description:** "Fast and accurate shared segment detection and relatedness estimation in un- phased genetic data using TRUFFLE"

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/truffle.html](https://hpc.nih.gov/apps/truffle.html)

### TVC

- **Description:** TVC is the standalone Torrent Variant Caller, part of the Ion Torrent Suite.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TVC.html](https://hpc.nih.gov/apps/TVC.html)

### umitools

- **Description:** tools for dealing with Unique Molecular Identifiers (UMIs)/Random Molecular Tags (RMTs) and single cell RNA-Seq cell barcodes
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/umitools.html](https://hpc.nih.gov/apps/umitools.html)

### vasttools

- **Description:** A toolset for profiling alternative splicing events in RNA-Seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vasttools.html](https://hpc.nih.gov/apps/vasttools.html)

### vcf2maf

- **Description:** A smarter, more reproducible, and more configurable tool for converting a VCF to a MAF.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vcf2maf.html](https://hpc.nih.gov/apps/vcf2maf.html)

### vcfanno

- **Description:** annotate a VCF with other VCFs/BEDs/tabixed files
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vcfanno.html](https://hpc.nih.gov/apps/vcfanno.html)

### vcflib

- **Description:** a simple C++ library for parsing and manipulating VCF files, + many command-line utilities
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vcflib.html](https://hpc.nih.gov/apps/vcflib.html)

### vcftools

- **Description:** VCFtools contains a Perl API (Vcf.pm) and a number of Perl scripts that can be used to perform common tasks with VCF files such as file validation, file merging, intersecting, complements, etc.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vcftools.html](https://hpc.nih.gov/apps/vcftools.html)

### VEP

- **Description:** VEP (Variant Effect Predictor) determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VEP.html](https://hpc.nih.gov/apps/VEP.html)

### verifybamid

- **Description:** verifyBamID is a software that verifies whether the reads in particular file match previously known genotypes for an individual (or group of individuals), and checks whether the reads are contaminated as a mixture of two samples. verifyBamID can detect sample contamination and swaps when external genotypes are available. When external genotypes are not available, verifyBamID still robustly detects sample swaps.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/verifybamid.html](https://hpc.nih.gov/apps/verifybamid.html)

### vg

- **Description:** Tools for working with genome variation graphs
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vg.html](https://hpc.nih.gov/apps/vg.html)

### viper

- **Description:** VIPER combines the use of several dozen RNA-seq tools, suites, and packages to create a complete pipeline that takes RNA-seq analysis from raw sequencing data all the way through alignment, quality control, unsupervised analyses, differential expression, and downstream pathway analysis
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/viper.html](https://hpc.nih.gov/apps/viper.html)

### viral-ngs

- **Description:** viral-ngs comprises a set of viral genomics analysis pipelines developed by the Broad Institute
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/viral-ngs.html](https://hpc.nih.gov/apps/viral-ngs.html)

### vireosnp

- **Description:** Demultiplexing pooled scRNA-seq data without genotype reference
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vireosnp.html](https://hpc.nih.gov/apps/vireosnp.html)

### VirSorter2

- **Description:** VirSorter2 is a DNA and RNA virus identification tool. It leverages genome-informed
  database advances across a collection of customized automatic classifiers
  to improve the accuracy and range of virus sequence detection.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VirSorter2.html](https://hpc.nih.gov/apps/VirSorter2.html)

### VIRTUS

- **Description:** Bioinformatics pipeline for viral transcriptome detection.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VIRTUS.html](https://hpc.nih.gov/apps/VIRTUS.html)

### vsearch

- **Description:** VSEARCH supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, rereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering, conversion and merging of paired-end reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vsearch.html](https://hpc.nih.gov/apps/vsearch.html)

### vt

- **Description:** vt is a variant tool set that discovers short variants from Next Generation Sequencing data.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vt.html](https://hpc.nih.gov/apps/vt.html)

### winnowmap

- **Description:** winnowmap is used for mapping ONT and PacBio reads to repetitive reference sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/winnowmap.html](https://hpc.nih.gov/apps/winnowmap.html)

### xHLA

- **Description:** The HLA gene complex on human chromosome 6 is one of the
  most polymorphic regions in the human genome and contributes
  in large part to the diversity of the immune system. Accurate typing of HLA genes with short-read sequencing data has historically been difficult due to the sequence similarity between the polymorphic alleles. xHLA iteratively refines the mapping results at the amino acid level to
  achieve high typing accuracy for both class I and II HLA genes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/xHLA.html](https://hpc.nih.gov/apps/xHLA.html)

### XHMM

- **Description:** XHMM uses principal component analysis (PCA) normalization and a hidden Markov model (HMM) to detect and genotype copy number variation (CNV) from normalized read-depth data from targeted sequencing experiments.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/XHMM.html](https://hpc.nih.gov/apps/XHMM.html)

### xwas

- **Description:** XWAS (chromosome X-Wide Analysis toolSet) is a software suite for the analysis of the X chromosome in association analyses and similar studies.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/xwas.html](https://hpc.nih.gov/apps/xwas.html)

## Image Analysis

---

### 3DSlicer

- **Description:** A software platform for the analysis (including registration and interactive segmentation) and visualization (including volume rendering) of medical images and for research in image guided therapy.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/3DSlicer.html](https://hpc.nih.gov/apps/3DSlicer.html)

### AFNI

- **Description:** AFNI (Analysis of Functional NeuroImages) is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data - a technique for mapping human brain activity.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/afni.html](https://hpc.nih.gov/apps/afni.html)

### ANTs

- **Description:** Advanced Normalization Tools (ANTs) extracts information from complex datasets that include imaging. Paired with ANTsR (answer), ANTs is useful for managing, interpreting and visualizing multidimensional data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ANTs.html](https://hpc.nih.gov/apps/ANTs.html)

### aslprep

- **Description:** Aslprep is an application for preprocessing of ASL (arterial spin labeling) data and computation of CBF (cerebral blood flow). Aslprep is a pipeline that uses AFNI, FSL, ANTs, and freesurfer.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/aslprep.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/aslprep.html)

### B-SOiD

- **Description:** B-SOiD (Behavioral Segmentation in Deeplabcut) is an
  unsupervised learning algorithm that serves to discover and classify behaviors that are not pre-defined by users.
  It segregates statistically different, sub-second rodent behaviors with a single bottom-up perspective video cameraR by performing a novel expectation maximization
  fitting of Gaussian mixture models on t-Distributed Stochastic Neighbor Embedding (t-SNE).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/b-soid.html](https://hpc.nih.gov/apps/b-soid.html)

### baracus

- **Description:** Baracus predicts brain age, based on data from Freesurfer. It combines data from cortical thickness, cortical surface area, and subcortical information
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/baracus.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/baracus.html)

### BrkRaw

- **Description:** The ‘BrkRaw’ is a python module designed to provide a comprehensive tool to access raw data acquired from Bruker Biospin preclinical MRI scanner. This module is also compatible with the zip compressed data to enable use of the archived data directly.
  The module is comprised of four components, including graphical user interface (GUI), command-line tools, high-level and low-level python APIs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BrkRaw.html](https://hpc.nih.gov/apps/BrkRaw.html)

### broccoli

- **Description:** Pipeline for fast analysis of fMRI data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/broccoli.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/broccoli.html)

### Bsoft

- **Description:** Bsoft is a collection of programs and a platform for development of software for image and molecular processing in structural biology. Problems in structural biology are approached with a highly modular design, allowing fast development of new algorithms without the burden of issues such as file I/O. It provides an easily accessible interface, a resource that can be and has been used in other packages.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Bsoft.html](https://hpc.nih.gov/apps/Bsoft.html)

### c3d

- **Description:** C3D is a command-line tool for converting 3D images between common file formats. The tool also includes a growing list of commands for image manipulation, such as thresholding and resampling.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/c3d.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/c3d.html)

### cisTEM

- **Description:** cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes and obtain high-resolution 3D reconstructions from them.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cisTEM.html](https://hpc.nih.gov/apps/cisTEM.html)

### civet

- **Description:** civet is a brain-imaging pipeline for
  analysis of large MR data sets. civet
  extracts and analyses cortical surfaces
  from MR images, as well as many other
  volumetric and corticometric functions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/civet.html](https://hpc.nih.gov/apps/civet.html)

### cmtk

- **Description:** CMTK is a Software toolkit for computational morphometry of biomedical images. CMTK provides a set of command line tools for processing and I/O.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cmtk.html](https://hpc.nih.gov/apps/cmtk.html)

### connectome-workbench

- **Description:** Tools to browse, download, explore, and analyze data from the Human Connectome Project (HCP). Allows users to compare their own data to that of the HCP.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/connectome-workbench.html](https://hpc.nih.gov/apps/connectome-workbench.html)

### CTF

- **Description:** The CTF MEG software has two main roles:
- Provide a human-machine interface to the CTF MEG elec- tronics to collect MEG and/or EEG data.
- Provide a tool for reviewing and (to a limited extent) ana- lyzing the MEG and/or EEG data acquired by the CTF MEG system.

### ctffind

- **Description:** Programs for finding CTFs of electron micrographs
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ctffind.html](https://hpc.nih.gov/apps/ctffind.html)

### dcm2niix

- **Description:** DICOM to NIfTI converter
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/dcm2niix.html](https://hpc.nih.gov/apps/dcm2niix.html)

### DeepCAD

- **Description:** DeepCAD is a self-supervised deep-learning method
  for spatiotemporal enhancement of calcium imaging data
  that does not require any high signal-to-noise ratio (SNR) observations.
  DeepCAD suppresses detection noise and improves the SNR more than tenfold,
  which reinforces the accuracy of neuron extraction and spike inference
  and facilitates the functional analysis of neural circuits.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepCAD.html](https://hpc.nih.gov/apps/DeepCAD.html)

### DeepLabCut

- **Description:** DeepLabCut is an open source toolbox
  that builds on a state-of-the-art human pose estimation algorithm.
  It allows training of a deep neural network by using limited training data
  to precisely track user-defined features, so that the human labeling accuracy will be matched.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/DeepLabCut.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/DeepLabCut.html)

### dynamo

- **Description:** Dynamo is a software environment for subtomogram averaging of cryo-EM data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/dynamo.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/dynamo.html)

### elastix

- **Description:** a toolbox for rigid and nonrigid registration of images.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/elastix.html](https://hpc.nih.gov/apps/elastix.html)

### EMAN2

- **Description:** EMAN2 is a broadly based greyscale scientific image processing suite with a primary focus on processing data from transmission electron microscopes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/EMAN2.html](https://hpc.nih.gov/apps/EMAN2.html)

### fastsurfer

- **Description:** Fastsurfer is a neuroimaging pipeline based on deep learning.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fastsurfer.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fastsurfer.html)

### Fiji

- **Description:** Fiji is a distribution of the popular open-source software ImageJ focused on biological-image analysis. Fiji uses modern software engineering practices to combine powerful software libraries with a broad range of scripting languages to enable rapid prototyping of image-processing algorithms. Fiji facilitates the transformation of new algorithms into ImageJ plugins that can be shared with end users through an integrated update system.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Fiji.html](https://hpc.nih.gov/apps/Fiji.html)

### fitlins

- **Description:** Fitlins fits linear models to BIDS neuroimaging datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fitlins.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fitlins.html)

### fmriprep

- **Description:** A Robust Preprocessing Pipeline for fMRI Data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fmriprep.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/fmriprep.html)

### Frealign

- **Description:** Frealign is a program for high-resolution refinement of 3D reconstructions from cryo-EM images of single particles.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Frealign.html](https://hpc.nih.gov/apps/Frealign.html)

### Freesurfer

- **Description:** Freesurfer is a set of automated tools for reconstruction of the brain's cortical surface from structural MRI data, and overlay of functional MRI data onto the reconstructed surface.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/freesurfer.html](https://hpc.nih.gov/apps/freesurfer.html)

### FSL

- **Description:** FSL is a comprehensive library of image analysis and statistical tools for FMRI, MRI and DTI brain imaging data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fsl.html](https://hpc.nih.gov/apps/fsl.html)

### Gctf

- **Description:** Gctf provides accurate estimation of the contrast transfer function (CTF) for near-atomic resolution cryo electron microscopy (cryoEM) reconstruction using GPUs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Gctf.html](https://hpc.nih.gov/apps/Gctf.html)

### Huygens

- **Description:** Huygens is an image restoration, deconvolution, resolution and noise reduction. It can process images from all current optical microscopes, including wide-field, confocal, Nipkow (scanning disk confocal), multiple-photon, and 4Pi microscopes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Huygens.html](https://hpc.nih.gov/apps/Huygens.html)

### IMOD

- **Description:** IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/IMOD.html](https://hpc.nih.gov/apps/IMOD.html)

### ITK-SNAP

- **Description:** ITK-SNAP is a tool for segmentation of 3D biomedical images. It requires a graphical connection to run on the cluster.

### magetbrain

- **Description:** Given a set of labelled MR images (atlases) and unlabelled images (subjects), MAGeT produces a segmentation for each subject using a multi-atlas voting procedure based on a template library made up of images from the subject set.

### mango

- **Description:** Mango (Multi-image Analysis GUI) is a viewer for medical research images. It provides analysis tools and a user interface to navigate image volumes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mango.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mango.html)

### minc-toolkit

- **Description:** This metaproject bundles multiple MINC-based packages that historically have been developed somewhat independently
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/BIC-MNI/minc-toolkit-v2](https://hpc.nih.gov/apps/https://github.com/BIC-MNI/minc-toolkit-v2)

### MIPAV

- **Description:** The MIPAV (Medical Image Processing, Analysis, and Visualization) application enables quantitative analysis and visualization of medical images of numerous modalities such as PET, MRI, CT, or microscopy.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MIPAV.html](https://hpc.nih.gov/apps/MIPAV.html)

### MotionCor2

- **Description:** MotionCor2 is a multi-GPU accelerated program that provides iterative, patch-based motion detection combining spatial and temporal constraints and dose weighting for both single particle and tomographic cryo-electon microscopy images.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MotionCor2.html](https://hpc.nih.gov/apps/MotionCor2.html)

### mriqc

- **Description:** MRIQC is an MRI quality control tool
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mriqc.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mriqc.html)

### mrtrix

- **Description:** MRtrix provides a large suite of tools for image processing, analysis and visualisation, with a focus on the analysis of white matter using diffusion-weighted MRI.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mrtrix.html](https://hpc.nih.gov/apps/mrtrix.html)

### mrtrix3tissue

- **Description:** MRtrix3Tissue is a fork of the MRtrix3 project. It aims to add capabilities for 3-Tissue CSD modelling and analysis to a complete version of the MRtrix3 software.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mrtrix3tissue.html](https://hpc.nih.gov/apps/mrtrix3tissue.html)

### nibabies

- **Description:** Preprocessing pipeline for neonate and infant MRI.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/nibabies.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/nibabies.html)

### OpenSlide

- **Description:** OpenSlide is a C library for reading and manipulating digital slides of diverse
  vendor formats. It provides a simple interface to read whole-slide images
  (also known as virtual slides). OpenSlide has been used in the digital pathology projects.

### PEET

- **Description:** PEET (Particle Estimation for Electron Tomography) is an open-source package for aligning and averaging particles in 3-D subvolumes extracted from tomograms. It seeks the optimal alignment of each particle against a reference volume through several iterations. If PEET and IMOD are both installed, most PEET operations are available from the eTomo graphical user interface in IMOD.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PEET.html](https://hpc.nih.gov/apps/PEET.html)

### qsiprep

- **Description:** qsiprep configures pipelines for processing diffusion-weighted MRI (dMRI) data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/qsiprep.html](https://hpc.nih.gov/apps/qsiprep.html)

### QuPath

- **Description:** QuPath is open source software for bioimage analysis. It is often used for digital pathology applications because it offers a powerful set of tools for working with whole slide images - but it can be applied to lots of other kinds of image as well.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/QuPath.html](https://hpc.nih.gov/apps/QuPath.html)

### RELION

- **Description:** RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer program for Maximum A Posteriori refinement of (multiple) 3D reconstructions or 2D class averages in cryo-electron microscopy.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/RELION/index.html](https://hpc.nih.gov/apps/RELION/index.html)

### ResMap

- **Description:** ResMap (Resolution Map) is a Python (NumPy/SciPy) application with a Tkinter GUI and a command-line interface. It is a software package for computing the local resolution of 3D density maps studied in structural biology, primarily electron cryo-microscopy (cryo-EM).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ResMap.html](https://hpc.nih.gov/apps/ResMap.html)

### SAMsrcV3

- **Description:** Synthetic Aperture Magnetometry - The SANsrcV3 suite implements the latest advances in MEG source localization.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SAMsrcV3.html](https://hpc.nih.gov/apps/SAMsrcV3.html)

### SimNIBS

- **Description:** SimNIBS is a free software package for the Simulation of Non-invasive Brain Stimulation. It allows for realistic calculations of the electric field induced by transcranial magnetic stimulation (TMS) and transcranial direct current stimulation (tDCS).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/simnibs.html](https://hpc.nih.gov/apps/simnibs.html)

### SPHIRE

- **Description:** SPHIRE (SPARX for High-Resolution Electron Microscopy) is an open-source,
  user-friendly software suite for the semi-automated
  processing of single particle electron cryo-microscopy (cryo-EM) data.
  It allows fast and reproducible structure determination from cryo-EM images.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SPHIRE.html](https://hpc.nih.gov/apps/SPHIRE.html)

### Spinal Cord Toolbox (SCT)

- **Description:** Spinal Cord Toolbox (SCT), a comprehensive software dedicated to the processing
  of spinal cord MRI data. SCT builds on previously-validated methods and includes
  state-ofthe-art MRI templates and atlases of the spinal cord, algorithms to segment
  and register new data to the templates, and motion correction methods
  for diffusion and functional time series.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SCT.html](https://hpc.nih.gov/apps/SCT.html)

### spm12

- **Description:** The (S)tatistical (P)ara(M)etric application analyzes brain imaging data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/spm12.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/spm12.html)

### summovie

- **Description:** Summovie calculates movie frame sums, using the alignment results from a prior run of Unblur.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://grigoriefflab.janelia.org/unblur](https://hpc.nih.gov/apps/http://grigoriefflab.janelia.org/unblur)

### TeraStitcher

- **Description:** TeraStitcher is a free tool that enables the stitching of Teravoxel-sized tiled microscopy images even on workstations with relatively limited resources of memory (<8 GB) and processing power.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TeraStitcher.html](https://hpc.nih.gov/apps/TeraStitcher.html)

### tesseract

- **Description:** tesseract is a open source commercial quality OCR engine developed at Google.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tesseract.html](https://hpc.nih.gov/apps/tesseract.html)

### topaz

- **Description:** topaz is a pipeline for particle detection in
  cryo-electron microscopy images using convolutional
  neural networks trained from positive and unlabeled
  examples. Topaz also includes methods for micrograph
  and tomogram denoising using deep denoising models.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/topaz.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/topaz.html)

### TORTOISE

- **Description:** (Tolerably Obsessive Registration and Tensor Optimization
  Indolent Software Ensemble)
  The TORTOISE software package is for processing diffusion MRI data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/TORTOISE.html](https://hpc.nih.gov/apps/TORTOISE.html)

### tvb

- **Description:** The Virtual Brain (TVB) scientific library has the purpose of offering modern tools to the Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/tvb.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/tvb.html)

### unblur

- **Description:** Unblur is used to align the frames of movies recorded on an electron microscope to reduce image blurring due to beam-induced motion.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://grigoriefflab.janelia.org/unblur](https://hpc.nih.gov/apps/http://grigoriefflab.janelia.org/unblur)

### xcpengine

- **Description:** xpcEngine performs denoising and estimation of Functional Connectivity on fMRI datasets
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/xcpengine.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/xcpengine.html)

## Linkage/Phylogenetics

---

### AdmixTools

- **Description:** ADMIXTOOLS is a software package that supports formal tests of whether admixture occurred, and makes it possible to infer admixture proportions and dates.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AdmixTools.html](https://hpc.nih.gov/apps/AdmixTools.html)

### admixture

- **Description:** ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/admixture.html](https://hpc.nih.gov/apps/admixture.html)

### AncestryMap

- **Description:** AncestryMap is a software package that allows finding skews in ancestry that are potentially associated with disease genes in recently mixed populations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AncestryMap.html](https://hpc.nih.gov/apps/AncestryMap.html)

### Beagle

- **Description:** Beagle is a package for imputing genotypes, inferring haplotype phase, and performing genetic association analysis. BEAGLE is designed to analyze large-scale data sets with hundreds of thousands of markers genotyped on thousands of samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Beagle.html](https://hpc.nih.gov/apps/Beagle.html)

### beagle-lib

- **Description:** BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. It can make use of highly-parallel processors such as those in graphics cards (GPUs) found in many PCs.

module name: beagle-lib

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://beagle-dev.github.io/html/beagle_8h.html](https://hpc.nih.gov/apps/https://beagle-dev.github.io/html/beagle_8h.html)

### BEAST

- **Description:** BEAST (Bayesian Evolutionary Analysis Sampling Trees) is a cross-platform program for Bayesian MCMC analysis of molecular sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BEAST.html](https://hpc.nih.gov/apps/BEAST.html)

### bpipe

- **Description:** Bpipe provides a platform for running big bioinformatics jobs

### CD-HIT

- **Description:** CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cd-hit.html](https://hpc.nih.gov/apps/cd-hit.html)

### eigensoft

- **Description:** The EIGENSOFT package combines functionality from population genetics methods and EIGENSTRAT stratification correction method.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/eigensoft.html](https://hpc.nih.gov/apps/eigensoft.html)

### FastQTL

- **Description:** In order to discover quantitative trait loci (QTLs),
  multi-dimensional genomic datasets combining
  DNA-seq and ChiP-/RNA-seq require methods that rapidly correlate tens of thousands of
  molecular phenotypes with millions of genetic variants while appropriately controlling for multiple testing. FastQTL implements a popular cis-QTL mapping strategy
  in a user- and cluster-friendly tool. FastQTL also proposes an efficient
  permutation procedure to control for multiple testing.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/FastQTL.html](https://hpc.nih.gov/apps/FastQTL.html)

### FastTree

- **Description:** FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. For large alignments, FastTree is 100-1,000 times faster than PhyML 3.0 or RAxML 7.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/FastTree.html](https://hpc.nih.gov/apps/FastTree.html)

### GCTA

- **Description:** GCTA (Genome-wide Complex Trait Analysis) is designed to estimate the proportion of phenotypic variance explained by genome- or chromosome-wide SNPs for complex traits.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GCTA.html](https://hpc.nih.gov/apps/GCTA.html)

### GIGI

- **Description:** GIGI (Genotype Imputation Given Inheritance) implements an approach that enables computationally efficient imputation in large pedigrees.
  It samples inheritance vectors (IVs) from a Markov Chain Monte Carlo sampler
  by conditioning on genotypes from a sparse set of framework markers.
  Missing genotypes are probabilistically inferred from these IVs
  along with observed dense genotypes that are available on a subset of subjects.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GIGI.html](https://hpc.nih.gov/apps/GIGI.html)

### gubbins

- **Description:** Gubbins is an algorithm that iteratively identifies loci containing elevated densities of base substitions while concurrently constructing a phylogeny based on the putative point mutations outside of these regions.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gubbins.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gubbins.html)

### hyphy

- **Description:** HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package for the analysis of genetic sequences (in particular the inference of natural selection) using techniques in phylogenetics, molecular evolution, and machine learning.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hyphy.html](https://hpc.nih.gov/apps/hyphy.html)

### iqtree

- **Description:** Efficient phylogenomic software by maximum likelihood
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iqtree.html](https://hpc.nih.gov/apps/iqtree.html)

### king

- **Description:** KING is a toolset to explore genotype data from a genome-wide association study (GWAS) or a sequencing project. KING can be used to check family relationship and flag pedigree errors by estimating kinship coefficients and inferring IBD segments for all pairwise relationships.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/king.html](https://hpc.nih.gov/apps/king.html)

### loki

- **Description:** Loki is a linkage analysis package, primarily for large and complex pedigrees, which uses Markov chain Monte Carlo (MCMC) techniques to avoid many of the computational problems that prevent exact computational methods being used for large pedigrees.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/loki.html](https://hpc.nih.gov/apps/loki.html)

### Madeline2

- **Description:** The Madeline 2.0 Pedigree Drawing Engine (PDE) is a
  pedigree drawing program for use in linkage and family-based
  association studies. The program is designed to handle large and
  complex pedigrees with an emphasis on readability and aesthetics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Madeline.html](https://hpc.nih.gov/apps/Madeline.html)

### MALDER

- **Description:** MALDER is a Modified version of ALDER that has been modified to allow multiple admixture events. ALDER computes the weighted linkage disequilibrium (LD) statistic for making inference about population admixture described in: Loh P-R, Lipson M, Patterson N, Moorjani P, Pickrell JK, Reich D, and Berger B. Inferring Admixture Histories of Human Populations Using Linkage Disequilibrium. Genetics, 2013.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/joepickrell/malder/tree/master/MALDER](https://hpc.nih.gov/apps/https://github.com/joepickrell/malder/tree/master/MALDER)

### mantra

- **Description:** Transethnic meta-analysis of genomewide association studies
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mantra.html](https://hpc.nih.gov/apps/mantra.html)

### MEGA

- **Description:** MEGA, Molecular Evolutionary Genetics Analysis, is a software suite for analyzing DNA and protein sequence data from species and populations
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MEGA.html](https://hpc.nih.gov/apps/MEGA.html)

### mega2

- **Description:** Mega2 is a data-handling program for facilitating genetic linkage and association analyses.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mega2.html](https://hpc.nih.gov/apps/mega2.html)

### merlin

- **Description:** MERLIN uses sparse trees to represent gene flow in pedigrees and is one of the fastest pedigree analysis packages around.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/merlin.html](https://hpc.nih.gov/apps/merlin.html)

### mothur

- **Description:** mothur is a tool for analyzing 16S rRNA gene sequences generated on multiple platforms as part of microbial ecology projects.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mothur.html](https://hpc.nih.gov/apps/mothur.html)

### MrBayes

- **Description:** MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mrbayes.html](https://hpc.nih.gov/apps/mrbayes.html)

### pesca

- **Description:** Pesca is a program for estimating the proportion of population-specific and shared causal variants from GWAS study results
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/huwenboshi/pesca/tree/master/docs](https://hpc.nih.gov/apps/https://github.com/huwenboshi/pesca/tree/master/docs)

### phase

- **Description:** infers haplotypes from population genotype data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/phase.html](https://hpc.nih.gov/apps/phase.html)

### Phylip

- **Description:** Phylip is a package of programs for inferring phylogenies (evolutionary trees). Includes methods for parsimony, distance matrix and likelihood methods.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Phylip.html](https://hpc.nih.gov/apps/Phylip.html)

### pplacer

- **Description:** Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give useful information about uncertainty, and to offer advanced visualization and downstream analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pplacer.html](https://hpc.nih.gov/apps/pplacer.html)

### PRIMUS

- **Description:** PRIMUS, Pedigree Reconstruction and Identification of a Maximum Unrelated Set, is used to read genome-wide IBD estimates and identify a maximum unrelated set and pedigree reconstruction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PRIMUS.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PRIMUS.html)

### PRSice

- **Description:** PRSice is a Polygenic Risk Score software for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS) analyses.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/prsice.html](https://hpc.nih.gov/apps/prsice.html)

### QIIME

- **Description:** QIIME is an open source software package for comparison and analysis of microbial communities, primarily based on high-throughput amplicon sequencing data (such as SSU rRNA) generated on a variety of platforms, but also supporting analysis of other types of data (such as shotgun metagenomic data).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/QIIME.html](https://hpc.nih.gov/apps/QIIME.html)

### RAxML

- **Description:** RAxML (randomized axelerated maximum likelihood) is a sequential and parallel program for inference of large phylogenies with maximum likelihood (ML).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/raxml.html](https://hpc.nih.gov/apps/raxml.html)

### raxml-ng

- **Description:** RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree. Successor to raxml.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/raxml-ng.html](https://hpc.nih.gov/apps/raxml-ng.html)

### RefinedIBD

- **Description:** Refined IBD is a software package that detects identity-by-descent segments
  in phased genotype data. It achieves both computational efficiency and highly accurate
  IBD segment reporting by searching for IBD in two steps.
  The first step (identification) uses the GERMLINE algorithm to find shared
  haplotypes exceeding a length threshold.
  The second step (refinement) evaluates candidate segments with a probabilistic approach to
  assess the evidence for IBD.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/RefinedIBD.html](https://hpc.nih.gov/apps/RefinedIBD.html)

### RevBayes

- **Description:** Bayesian phylogenetic inference using probabilistic graphical models and an interpreted language
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/revbayes.html](https://hpc.nih.gov/apps/revbayes.html)

### seqmix

- **Description:** SEQMIX is a program that takes advantage of off-targeted sequence reads from exome/targeted sequencing experiments for accurate local ancestry inference.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/seqmix.html](https://hpc.nih.gov/apps/seqmix.html)

### shapeit

- **Description:** SHAPEIT is a fast and accurate haplotype inference software
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/shapeit.html](https://hpc.nih.gov/apps/shapeit.html)

### SMR

- **Description:** SMR integrates summary-level data from GWAS with data from expression quantitative trait locus (eQTL) studies to identify genes whose expression levels are associated with a complex trait because of pleiotropy. It implements methods to test for pleiotropic association between the expression level of a gene and a complex trait of interest using summary-level data from GWAS and expression quantitative trait loci (eQTL) studies (Zhu et al. 2016 Nat Genet).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SMR.html](https://hpc.nih.gov/apps/SMR.html)

### Solar

- **Description:** SOLAR-Eclipse is an extensive, flexible software package for genetic variance components analysis, including linkage analysis, quantitative genetic analysis, SNP association analysis (QTN and QTLD), and covariate screening.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/solar.html](https://hpc.nih.gov/apps/solar.html)

### sumtrees

- **Description:** Summarize non-parameteric bootstrap or Bayesian posterior probability support for splits or clades on phylogenetic trees.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sumtrees.html](https://hpc.nih.gov/apps/sumtrees.html)

### treemix

- **Description:** TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/treemix.html](https://hpc.nih.gov/apps/treemix.html)

## Mass Spectrometry

---

### fragpipe

- **Description:** FragPipe is a Java Graphical User Interface (GUI) for a suite of computational tools enabling comprehensive analysis of mass spectrometry-based proteomics data. It is powered by MSFragger.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fragpipe.html](https://hpc.nih.gov/apps/fragpipe.html)

### Mascot

- **Description:** The Mascot search engine uses mass spectrometry data to identify proteins from primary sequence databases. Mascot searches can be run directly on the NIH Mascot server at https://biospec.nih.gov, or by using the Mascot daemon on your own desktop PC.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://biospec.nih.gov/](https://hpc.nih.gov/apps/https://biospec.nih.gov/)

### maxquant

- **Description:** MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling techniques as well as label-free quantification are supported.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/maxquant.html](https://hpc.nih.gov/apps/maxquant.html)

### metamorpheus

- **Description:** MetaMorpheus is a bottom-up proteomics database search software with integrated post-translational modification (PTM) discovery capability.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/metamorpheus.html](https://hpc.nih.gov/apps/metamorpheus.html)

### msfragger

- **Description:** An ultrafast database search tool for peptide identification in mass spectrometry-based proteomics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/msfragger.html](https://hpc.nih.gov/apps/msfragger.html)

### philosopher

- **Description:** Philosopher is fast, easy-to-use, scalable, and versatile data analysis software for mass spectrometry-based proteomics. Philosopher is dependency-free and can analyze both traditional database searches and open searches for post-translational modification (PTM) discovery.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/philosopher.html](https://hpc.nih.gov/apps/philosopher.html)

## Mathematical/Statistics

---

### ANTs

- **Description:** Advanced Normalization Tools (ANTs) extracts information from complex datasets that include imaging. Paired with ANTsR (answer), ANTs is useful for managing, interpreting and visualizing multidimensional data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ANTs.html](https://hpc.nih.gov/apps/ANTs.html)

### CLONET

- **Description:** CLONET is a collection of R scripts that allows:

  computing global DNA admixture (1-purity) and ploidy of tumor DNA samples (each with matched normal sample) from sequencing data (WGS, WES, targeted)
  computing clonality of each somatic aberration, including somatic copy number aberrations, point mutations, and structural rearrangements
  nominatig the temporal relation among somatic aberrations and building evolution maps

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CLONET.html](https://hpc.nih.gov/apps/CLONET.html)

### cluster3

- **Description:** cluster3 is a multipurpose open-source library of C routines, callable from other C and C++programs. It implements k-means clustering, hierarchical clustering and self-organizing maps and provides several unique analytical approaches. In addition, it includes a Python and a Perl interface to the C Clustering Library, thereby combining the flexibility of a scripting language with the speed of C.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cluster3.html](https://hpc.nih.gov/apps/cluster3.html)

### cmdstan

- **Description:** Command line interface to stan
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cmdstan.html](https://hpc.nih.gov/apps/cmdstan.html)

### Comsol

- **Description:** The COMSOL Multiphysics engineering simulation software environment facilitates all steps in the modeling process − defining your geometry, meshing, specifying your physics, solving, and then visualizing your results.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Comsol.html](https://hpc.nih.gov/apps/Comsol.html)

### GAUSS

- **Description:** The GAUSS Mathematical and Statistical System is an easy-to-use data analysis environment based on the fast and powerful GAUSS Matrix Programming Language designed for computationally intensive tasks.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GAUSS.html](https://hpc.nih.gov/apps/GAUSS.html)

### gephi

- **Description:** gephi is a scientific application for computation of a number of metrics in graph/network analysis.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gephi.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/gephi.html)

### gurobi

- **Description:** Gurobi is a mathematical optimization solver. It is a commercial product developed by gurobi.com. On Biowulf, Gurobi is licensed for use by the members of the CDSL_Gurobi_users group only. It is installed in /data/CDSL_Gurobi_users and is not accessible by any other users. A token license server, running on Biowulf, manages the Gurobi license.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gurobi.html](https://hpc.nih.gov/apps/gurobi.html)

### HPhi

- **Description:** HPhi is a numerical solver package for a wide range of quantum lattice models including Hubbard-type itinerant electron hamiltonians, quantum spin models, and Kondo-type hamiltonians for itinerant electrons coupled with quantum spins. The Lanczos algorithm for finding ground states and newly developed Lanczos-based algorithm for finite-temperature properties of these models are implemented for parallel computing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://issp-center-dev.github.io/HPhi/manual/master/en/html/index.html](https://hpc.nih.gov/apps/http://issp-center-dev.github.io/HPhi/manual/master/en/html/index.html)

### IDL/ENVI

- **Description:** IDL and ENVI are a complete computing environment for the interactive analysis and visualization of data. IDL integrates an array-oriented language with mathematical analysis and graphical display techniques. ENVI is designed for extracting information from geospatial and medical imagery.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/idl.html](https://hpc.nih.gov/apps/idl.html)

### m2clust

- **Description:** m2clust provides an elegant clustering approach to find clusters in data sets with different density and resolution.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/m2clust.html](https://hpc.nih.gov/apps/m2clust.html)

### Mathematica

- **Description:** Mathematica is an interactive system for doing mathematical computation. It performs numerical, symbolic and graphical computations, and incorporates a high-level programming language.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/../apps/mathematica.html](https://hpc.nih.gov/apps/../apps/mathematica.html)

### Matlab

- **Description:** MATLAB is an interactive software package for scientific and engineering numeric computation. MATLAB integrates numerical analysis, matrix computation, signal processing, and graphics in an environment where problems and solutions are expressed just as they are written mathematically.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Matlab.html](https://hpc.nih.gov/apps/Matlab.html)

### MCL

- **Description:** MCL implements Markov cluster algorithm. Among its applications
  is the assignment of proteins into families based on precomputed sequence
  similarity information. This approach
  does not suffer from the problems that normally hinder other protein sequence
  clustering algorithms, such as the presence of multi-domain proteins,
  promiscuous domains and fragmented proteins.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mcl.html](https://hpc.nih.gov/apps/mcl.html)

### Meep

- **Description:** Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package developed at MIT to model electromagnetic systems, along with the MPB eigenmode package.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Meep.html](https://hpc.nih.gov/apps/Meep.html)

### Octave

- **Description:** GNU Octave is a high-level language, primarily intended for numerical computations. It provides a convenient command line interface for solving linear and nonlinear problems numerically, and for performing other numerical experiments using a language that is mostly compatible with Matlab.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/octave.html](https://hpc.nih.gov/apps/octave.html)

### omeClust

- **Description:** omeClust (previously m2clust) is a clustering method that detects clusters of features using omics data and scores metadata (resolution score) based on their influences in clustering.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/omeClust.html](https://hpc.nih.gov/apps/omeClust.html)

### PEER

- **Description:** PEER stands for probabilistic estimation of expression residuals. It
  is a collection of Bayesian approaches to infer hidden determinants and
  their effects from gene expression profiles using factor analysis methods.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PEER.html](https://hpc.nih.gov/apps/PEER.html)

### R

- **Description:** R (the R Project) is a language and environment for statistical computing and graphics. R is similar to S, and provides a wide variety of statistical and graphical techniques (linear and nonlinear modelling, statistical tests, time series analysis, classification, clustering, ...).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/../apps/R.html](https://hpc.nih.gov/apps/../apps/R.html)

### Rstudio

- **Description:** RStudio is a set of integrated tools designed to help you be more productive with R. It includes a console, syntax-highlighting editor that supports direct code execution, as well as tools for plotting, history, debugging and workspace management.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/RStudio.html](https://hpc.nih.gov/apps/RStudio.html)

### SAS

- **Description:** Base SAS provides a scalable, integrated software environment specially designed for data access, transformation and reporting.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SAS.html](https://hpc.nih.gov/apps/SAS.html)

### team

- **Description:** TEAM is an efficient program for epistasis detection in genome-wide data. With user-provided genotype and phenotype data, as well as a few parameters, the TEAM is able to return the significant SNP-pairs from data sets with hundreds of thousands of samples. It is one of several programs that are implementations of efficient epistasis detection algorithms, which greatly speed up the computationally intensive process.

## Molecular Modeling/Graphics

---

### bio3d-web

- **Description:** Bio3D-web is an online application for analyzing the sequence, structure and conformational heterogeneity
  of protein families.
  This application facilitates the identification of protein structure sets for analysis,
  their alignment and refined structure superposition, sequence and structure conservation analysis,
  mapping and clustering of conformations, and the quantitative comparison of their predicted structural dynamics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bio3d-web.html](https://hpc.nih.gov/apps/bio3d-web.html)

### blender

- **Description:** Blender is the free and open source 3D creation suite. Blender on Biowulf is meant for command-line rendering.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/blender.html](https://hpc.nih.gov/apps/blender.html)

### Chimera

- **Description:** Chimera is a highly extensible program for interactive visualization and analysis of molecular structures and related data, including density maps, supramolecular assemblies, sequence alignments, docking results, trajectories, and conformational ensembles.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Chimera.html](https://hpc.nih.gov/apps/Chimera.html)

### Coot

- **Description:** Coot is for macromolecular model building, model completion and validation, particularly suitable for protein modelling using X-ray data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Coot.html](https://hpc.nih.gov/apps/Coot.html)

### Cytoscape

- **Description:** Cytoscape is an open source software platform for visualizing molecular interaction networks and biological pathways and integrating these networks with annotations, gene expression profiles and other state data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Cytoscape.html](https://hpc.nih.gov/apps/Cytoscape.html)

### Glycosylator

- **Description:** Glycosylator is a Python framework for the identification, modeling and modification of glycans
  in protein structure that can be used directly in a Python script through its API or GUI.
  The GUI allows for a quick visual inspection of the glycosylation state of all the sequons
  on a protein structure. Modeled glycans can be further refined by a genetic algorithm
  for removing clashes and sampling alternative conformations. Glycosylator can also identify
  specific three-dimensional (3D) glycans on a protein structure using a library
  of predefined templates.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Glycosylator.html](https://hpc.nih.gov/apps/Glycosylator.html)

### infinisee

- **Description:** A chemical space (GUI) navigation platform. Based on similarity, infiniSee finds molecules of interest in screening libraries or Chemical Spaces of almost infinite size.

### lammps

- **Description:** LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. It runs on a variety of different computer systems, including single processor systems, distributed-memory machines with MPI, and GPU and Xeon Phi systems. LAMMPS is open source software, released under the GNU General Public License.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lammps.html](https://hpc.nih.gov/apps/lammps.html)

### MD-TASK

- **Description:** MD-TASK is a software suite for molecular dynamics (MD). It employs graph theory techniques, perturbation response scanning, and dynamic cross-correlation to provide unique ways for analyzing MD trajectories.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MD-TASK.html](https://hpc.nih.gov/apps/MD-TASK.html)

### neuroconstruct

- **Description:** Biophysical Neural Network Modeling Software.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/neuroconstruct.html](https://hpc.nih.gov/apps/neuroconstruct.html)

### OpenBabel

- **Description:** Open Babel is a chemical toolbox designed to speak the many languages of chemical data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/OpenBabel.html](https://hpc.nih.gov/apps/OpenBabel.html)

### posefilter

- **Description:** PoseFilter is a PyMOL plugin and assists in the analysis of docked ligands through identification of unique oligomeric poses by utilizing RMSD and interaction fingerprint analysis methods.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PyMOL.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/PyMOL.html)

### Psi4

- **Description:** Psi4 is an ab-initio electronic structure code that supports various methods for calculating energies and gradients of molecular systems.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/psi4.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/psi4.html)

### Rosetta

- **Description:** The Rosetta++ software suite can perform de novo protein structure predictions, identify low free energy sequences for target protein backbones, predict the structure of a protein-protein complex from the individual structures of the monomer components, incorporate NMR data into the basic Rosetta protocol to accelerate the process of NMR structure prediction, and more...
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rosetta.html](https://hpc.nih.gov/apps/rosetta.html)

### Schrodinger

- **Description:** A limited number of Schrödinger applications are available on the Biowulf cluster through the Molecular Modeling Interest Group. Most are available through the Maestro GUI.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/schrodinger/index.html](https://hpc.nih.gov/apps/schrodinger/index.html)

### seesar

- **Description:** an interactive and visual (GUI) molecular modeling and drug design platform for compound evolution and prioritization.

### VMD

- **Description:** VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. To use, type vmd at the prompt.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VMD.html](https://hpc.nih.gov/apps/VMD.html)

## Sequence Analysis

---

### Accurity

- **Description:** Accurity is a tool for inference of tumor purity, tumor cell
  ploidy and absolute allelic copy numbers from tumor-normal WGS data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Accurity.html](https://hpc.nih.gov/apps/Accurity.html)

### advntr

- **Description:** a tool for genotyping Variable Number Tandem Repeats (VNTR) from sequence data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/advntr.html](https://hpc.nih.gov/apps/advntr.html)

### alleleCount

- **Description:** Calculates genotype frequencies of a SNPMatrix. This component tests each SNP for its Hardy-Weinberg equilibrium. If there are NA values, the frequencies of missing value per sample in the input file are calculated.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/alleleCount.html](https://hpc.nih.gov/apps/alleleCount.html)

### ANNOgesic

- **Description:** Processing and integrating RNA-Seq data in order to
  generate high-resolution annotations is challenging, time consuming and requires
  numerous different steps. ANNOgesic is a powerful and modular
  pipeline that provides the required analyses and simplifies
  RNA-Seq-based bacterial and archaeal genome annotation. It predicts and
  annotates numerous features, including small non-coding RNAs, with high
  precision.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ANNOgesic.html](https://hpc.nih.gov/apps/ANNOgesic.html)

### anvio

- **Description:** Anvi’o is an open-source, community-driven analysis and visualization platform for microbial ‘omics. It brings together many aspects of today’s cutting-edge strategies including genomics, metagenomics, metatranscriptomics, pangenomics, metapangenomics, phylogenomics, and microbial population genetics in an integrated and easy-to-use fashion through extensive interactive visualization capabilities.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/anvio.html](https://hpc.nih.gov/apps/anvio.html)

### ARDISS

- **Description:** ARDISS is a method to impute missing summary statistics in mixed-ethnicity cohorts
  through Gaussian Process Regression and automatic relevance determination.
  ARDISS is trained on an external reference panel and does not require information
  about allele frequencies of genotypes from the original study.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ARDISS.html](https://hpc.nih.gov/apps/ARDISS.html)

### arriba

- **Description:** Arriba identifies gene fusions in RNA-Seq data. It also can detect other structural variants in genomic data, such as intron duplications and gene truncations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/arriba.html](https://hpc.nih.gov/apps/arriba.html)

### augustus

- **Description:** AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/augustus.html](https://hpc.nih.gov/apps/augustus.html)

### bamcmp

- **Description:** bamcmp is a tool for deconvolving host and graft reads. It allows an accurate identification of the contaminating host reads when analyzing DNA-Seq and RNA-Seq data from patient-derived xenograft and circulating tumor cell–derived explant models.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamcmp.html](https://hpc.nih.gov/apps/bamcmp.html)

### bamgineer

- **Description:** Bamgineer introduces simulated allele-specific copy number variants into exome and targeted sequence data sets
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bamgineer.html](https://hpc.nih.gov/apps/bamgineer.html)

### Bartender

- **Description:** Bartender is an accurate clustering algorithm to detect barcodes
  and their abundances from raw next-generation sequencing data.
  In contrast with existing methods that cluster based on sequence similarity alone,
  Bartender uses a modified two-sample proportion test that also considers cluster size. This modification
  results in higher accuracy and lower rates of under- and over-clustering artifacts.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bartender.html](https://hpc.nih.gov/apps/bartender.html)

### Blast

- **Description:** NCBI's well-known sequence database searching program which compares a nucleotide or protein query sequence against all sequences in a database.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Blast.html](https://hpc.nih.gov/apps/Blast.html)

### blat

- **Description:** BLAT is a DNA/Protein Sequence Analysis program that is designed to quickly find sequences of 95% and greater similarity of length 40 bases or more.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/blat.html](https://hpc.nih.gov/apps/blat.html)

### BOLT-LMM

- **Description:** The BOLT-LMM algorithm computes statistics for testing association between phenotype and
  genotypes using a linear mixed model (LMM)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/BOLT-LMM.html](https://hpc.nih.gov/apps/BOLT-LMM.html)

### bpipe

- **Description:** Bpipe provides a platform for running big bioinformatics jobs

### bwa-mem2

- **Description:** The next version of the bwa-mem algorithm in bwa.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/bwa-mem2.html](https://hpc.nih.gov/apps/bwa-mem2.html)

### cactus

- **Description:** Cactus is a reference-free whole-genome multiple alignment program.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/cactus.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/cactus.html)

### CADD

- **Description:** CADD (Combined Annotation Dependent Depletion) is a tool for scoring the deleteriousness of single nucleotide variants as well as insertion/deletions variants in the human genome. Currently, it supports the builds: GRCh37/hg19 and GRCh38/hg38.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CADD.html](https://hpc.nih.gov/apps/CADD.html)

### CAVIAR

- **Description:** CAVIAR (CAusal Variants Identication in Associated Regions) is a statistical framework that quantifies the probability of each variant to be causal while allowing with arbitrary number of causal variants
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/caviar.html](https://hpc.nih.gov/apps/caviar.html)

### cellsnp-lite

- **Description:** Efficient genotyping bi-allelic SNPs on single cells
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellsnp-lite.html](https://hpc.nih.gov/apps/cellsnp-lite.html)

### CHESS

- **Description:** The CHESS (Comparison of Hi-C Experiments using Structural Similarity) application
  implements an algorithm for the comparison of chromatin contact maps and automatic
  differential feature extraction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/chess.html](https://hpc.nih.gov/apps/chess.html)

### chipseq_pipeline

- **Description:** AQUAS Transcription Factor and Histone ChIP-Seq processing pipeline. The AQUAS pipeline is based off the ENCODE (phase-3) transcription factor and histone ChIP-seq pipeline specifications (by Anshul Kundaje)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/chipseq_pipeline.html](https://hpc.nih.gov/apps/chipseq_pipeline.html)

### cicero

- **Description:** an assembly-based algorithm to detect diverse classes of driver gene fusions from RNA-seq.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cicero.html](https://hpc.nih.gov/apps/cicero.html)

### circos

- **Description:** Circos is a program for the generation of publication-quality, circularly composited renditions of genomic data and related annotations. Circos is particularly suited for visualizing alignments, conservation and intra and inter-chromosomal relationships. Also, Circos is useful to visualize any type of information that benefits from a circular layout. Thus, although it has been designed for the field of genomics, it is sufficiently flexible to be used in other data domains.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circos.html](https://hpc.nih.gov/apps/circos.html)

### CITE-seq-Count

- **Description:** CITE-seq-Count is a python package that counts antibody TAGS. It handles the data
  from both the Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seqi)
  and/or Cell Hashing experiment.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CITE-seq-Count.html](https://hpc.nih.gov/apps/CITE-seq-Count.html)

### Clair3

- **Description:** Clair3 is a small variant caller for Illumina, PacBio and ONT long reads. Compare to PEPPER (r0.4), Clair3 (v0.1) shows a better SNP F1-score with ≤30-fold of ONT data (precisionFDA Truth Challenge V2), and a better Indel F1-score, while runs generally four times faster.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Clair3.html](https://hpc.nih.gov/apps/Clair3.html)

### Clinker

- **Description:** Clinker is a bioinformatics pipeline that generates a superTranscriptome from popular fusion finder outputs
  (JAFFA, tophatFusion, SOAP, deFUSE, Pizzly, etc),
  that can be then be either viewed in genome viewers such as IGV or through the included plotting feature developed with GViz.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Clinker.html](https://hpc.nih.gov/apps/Clinker.html)

### clustalo

- **Description:** Clustal-Omega is a general purpose multiple sequence alignment (MSA)
  program for proteins and DNA/RNA. It produces high quality MSAs and is
  capable of handling data-sets of hundreds of thousands of sequences in
  reasonable time.

### ClustalW

- **Description:** ClustalW is a general-purpose multiple alignment program for DNA or protein sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ClustalW.html](https://hpc.nih.gov/apps/ClustalW.html)

### CutRunTools

- **Description:** CutRunTools is a flexible, general pipeline for facilitating the identification of
  chromatin-associated protein binding and genomic footprinting analysis from antibody-targeted
  CutRun primary cleavage data. CutRunTools extracts endonuclease cut site information from
  sequences of short read fragments and produces single-locus binding estimates, aggregate motif
  footprints, and informative visualizations to support the high-resolution mapping capability of
  CutRun.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CutRunTools.html](https://hpc.nih.gov/apps/CutRunTools.html)

### DeconSeq

- **Description:** The DeconSeq tool can be used to automatically detect and efficiently remove sequence contamination from genomic and metagenomic datasets. It is easily configurable and provides a user-friendly interface.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeconSeq.html](https://hpc.nih.gov/apps/DeconSeq.html)

### DeepM6A

- **Description:** DeepM6A application implements a deep-learning-based algorithm
  for predicting potential DNA 6mA sites de novo from sequence at single-nucleotide resolution.
  Application of this tool to three representative model organisms, Arabidopsis thaliana, Drosophila melanogaster and Escherichia coli,
  demonstrated that it supercedes the conventional k-mer-based approaches
  in both accuracy and performance.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepM6A.html](https://hpc.nih.gov/apps/DeepM6A.html)

### deepsea

- **Description:** DeepSEA is a deep learning-based algorithmic framework for predicting the chromatin effects of sequence alterations with single nucleotide sensitivity. DeepSEA can accurately predict the epigenetic state of a sequence, including transcription factors binding, DNase I sensitivities and histone marks in multiple cell types, and further utilize this capability to predict the chromatin effects of sequence variants and prioritize regulatory variants.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/deepsea.html](https://hpc.nih.gov/apps/deepsea.html)

### diamond

- **Description:** DIAMOND is a new high-throughput program for aligning DNA reads or protein sequences against a protein reference database such as NR, at up to 20,000 times the speed of BLAST, with high sensitivity.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/diamond.html](https://hpc.nih.gov/apps/diamond.html)

### DNAWorks

- **Description:** DNAWorks is a computer program that automates the design of oligonucleotides for gene synthesis by PCR-based gene assembly. The program requires simple input information: an amino acid sequence of the target protein or a DNA sequence, and a desired annealing temperature. It is a web-based tool available at https://hpcwebapps.cit.nih.gov/dnaworks/.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpcwebapps.cit.nih.gov/dnaworks/](https://hpc.nih.gov/apps/https://hpcwebapps.cit.nih.gov/dnaworks/)

### drep

- **Description:** dRep is a python program for rapidly comparing large numbers of genomes. dRep can also "de-replicate" a genome set by identifying groups of highly similar genomes and choosing the best representative genome for each genome set.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/drep.html](https://hpc.nih.gov/apps/drep.html)

### dyno

- **Description:** dyno is a meta package that installs several other packages
  from the dynvers (https://github.com/dynverse). It comprises a set of R packages
  to construct and interpret single-cell trajectories.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/dyno.html](https://hpc.nih.gov/apps/dyno.html)

### Eagle

- **Description:** Eagle performs a reference-based haplotype phasing. It attains high accuracy across a broad range of cohort sizes by efficiently leveraging information from large external reference panels (such as the Haplotype Reference onsortium; HRC)
  using a new data structure based on the positional Burrows-Wheeler transform.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Eagle.html](https://hpc.nih.gov/apps/Eagle.html)

### emim

- **Description:** Estimation of Maternal, Imprinting and interaction effects using Multinomial modeling

### EukRep

- **Description:** Microbial eukaryotes are integral components of natural microbial communities and their
  inclusion is critical for many ecosystem studies, yet the majority of published metagenome
  analyses ignore eukaryotes. In order to include eukaryotes in environmental studies,
  eukaryotic genomes shoould be recovered from complex metagenomic samples. A key step for genome recovery is separation of eukaryotic and prokaryotic fragments. EukRep is a kmer- and SVM-based strategy for eukaryotic sequence identification from environmental samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/EukRep.html](https://hpc.nih.gov/apps/EukRep.html)

### exonerate

- **Description:** Exonerate is a generic tool for pairwise sequence comparison. It allows you to align sequences using a many alignment models, either exhaustive dynamic programming or a variety of heuristics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/exonerate.html](https://hpc.nih.gov/apps/exonerate.html)

### ExpansionHunter

- **Description:** Expansion Hunter: a tool for estimating repeat sizes. There are a number of regions in the human genome consisting of repetitions of short unit sequence (commonly a trimer). Such repeat regions can expand to a size much larger than the read length and thereby cause a disease. Expansion Hunter aims to estimate sizes of such repeats by performing a targeted search through a BAM/CRAM file for reads that span, flank, and are fully contained in each repeat.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/Illumina/ExpansionHunter/blob/master/docs/01_Introduction.md](https://hpc.nih.gov/apps/https://github.com/Illumina/ExpansionHunter/blob/master/docs/01_Introduction.md)

### ExpansionHunterDenovo

- **Description:** ExpansionHunter Denovo (EHdn) is a suite of tools for detecting novel expansions of short tandem repeats (STRs). EHdn is intended for analysis of a collection of BAM/CRAM files containing alignments of short (100-200bp) reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/Illumina/ExpansionHunterDenovo/blob/master/documentation/00_Introduction.md](https://hpc.nih.gov/apps/https://github.com/Illumina/ExpansionHunterDenovo/blob/master/documentation/00_Introduction.md)

### fastp

- **Description:** A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/fastp.html](https://hpc.nih.gov/apps/fastp.html)

### FitHiChIP

- **Description:** FitHiChIP is a computational method for identifying chromatin contacts
  among regulatory regions such as enhancers and promoters from HiChIP/PLAC-seq data.
  FitHiChIP jointly models the non-uniform coverage and genomic distance scaling of HiChIP data,
  captures previously validated enhancer interactions for several genes
  including MYC and TP53, and recovers contacts genome-wide
  that are supported by ChIA-PET, promoter capture Hi-C and Hi-C data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/FitHiChIP.html](https://hpc.nih.gov/apps/FitHiChIP.html)

### FLAIR

- **Description:** FLAIR (Full-Length Alternative Isoform analysis of RNA) is a workflow
  leveraging the full-length transcript sequencing data that nanopore affords.
  It uses multiple alignment steps and splice site filters
  to increase confidence in the set of isoforms defined from noisy data.

### focus

- **Description:** FOCUS (Fine-mapping Of CaUsal gene Sets) is software to fine-map transcriptome-wide association study statistics at genomic risk regions. The software takes as input summary GWAS data along with eQTL weights and outputs a credible set of genes to explain observed genomic risk.

### FuSeq

- **Description:** FuSeq is a software for discovering fusion genes from paired-end RNA sequencing data. It implements a fast and accurate method to discover fusion genes based on quasi-mapping to quickly map the reads, extract initial candidates from split reads and fusion equivalence classes of mapped reads, and finally apply multiple filters and statistical tests to get the final candidates.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/FuSeq.html](https://hpc.nih.gov/apps/FuSeq.html)

### genometools

- **Description:** collection of bioinformatic tools
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/genometools.html](https://hpc.nih.gov/apps/genometools.html)

### gffcompare

- **Description:** gffcompare can be used to compare and evaluate the accuracy of RNA-Seq transcript assemblers (Cufflinks, Stringtie). It can collapse (merge) duplicate transcripts from multiple GTF/GFF3 files (e.g. resulted from assembly of different samples) and classify transcripts from one or multiple GTF/GFF3 files as they relate to reference transcripts provided in a annotation file (also in GTF/GFF3 format).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gffcompare.html](https://hpc.nih.gov/apps/gffcompare.html)

### GimmeMotifs

- **Description:** GimmeMotifs is a pipeline for transcription factor motif analysis written in Python.
  It incorporates an ensemble of computational tools to predict
  motifs de novo from ChIP-sequencing data. Similar
  redundant motifs are compared using the weighted information
  content similarity score and clustered using an iterative
  procedure. A comprehensive output report is generated with several
  different evaluation metrics to compare and evaluate the results.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GimmeMotifs.html](https://hpc.nih.gov/apps/GimmeMotifs.html)

### glnexus

- **Description:** Joint variant calling for large cohort sequencing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/glnexus.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/glnexus.html)

### guidance

- **Description:** GUIDANCE is meant to be used for weighting, filtering or masking unreliably aligned positions in sequence alignments before subsequent analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/guidance.html](https://hpc.nih.gov/apps/guidance.html)

### hhsuite

- **Description:** The HH-suite is an open-source software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hhsuite.html](https://hpc.nih.gov/apps/hhsuite.html)

### hipstr

- **Description:** Tool for genotyping short tandom repeats from Illumina sequencing data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hipstr.html](https://hpc.nih.gov/apps/hipstr.html)

### HMMER

- **Description:** HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called "profile hidden Markov models" (profile HMMs).

Compared to BLAST, FASTA, and other sequence alignment and database search tools based on older scoring methodology, HMMER aims to be significantly more accurate and more able to detect remote homologs because of the strength of its underlying mathematical models.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/hmmer.html](https://hpc.nih.gov/apps/hmmer.html)

### HTGTSrep

- **Description:** A pipeline for comprehensive analysis of HTGTS-Rep-seq.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/HTGTSrep.html](https://hpc.nih.gov/apps/HTGTSrep.html)

### iDiffIR

- **Description:** iDiffIR is a tool for identifying differential IR from RNA-seq data.
  It accepts any sorted, indexed BAM file for single- or paired-end reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iDiffIR.html](https://hpc.nih.gov/apps/iDiffIR.html)

### igblast

- **Description:** IgBlast is a sequence analysis tool for immunoglobulin variable domains.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/igblast.html](https://hpc.nih.gov/apps/igblast.html)

### ImReP

- **Description:** ImReP is a novel computational method
  for rapid and accurate profiling of the adaptive immune repertoire from regular RNA-Seq data.
  It is able to efficiently extract TCR- and BCR-derived reads from RNA-Seq data.
  ImReP can also accurately assemble the complementary determining regions 3 (CDR3s),
  the most variable regions of B and T cell receptors, and determine their antigen specificity.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ImReP.html](https://hpc.nih.gov/apps/ImReP.html)

### infernal

- **Description:** Package for searching DNA sequence databases for RNA structure and sequence similarities
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/infernal.html](https://hpc.nih.gov/apps/infernal.html)

### interproscan

- **Description:** InterProScan is the software package that allows sequences (protein and nucleic) to be scanned against InterPro's signatures. Signatures are predictive models, provided by several different databases, that make up the InterPro consortium.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/interproscan.html](https://hpc.nih.gov/apps/interproscan.html)

### InterVar

- **Description:** In 2015, the American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology (AMP) published
  updated standards and guidelines for the clinical interpretation of sequence variants with respect to human diseases on the basis
  of 28 criteria. However, variability between individual interpreters can be extensive because of reasons such as the different understandings of these guidelines and the lack of standard algorithms for implementing them, yet computational tools for semi-automated variant interpretation are not available. To address these problems, InterVar implements these criteria to help human reviewers interpret the clinical significance of variants. InterVar can take a pre-annotated or VCF file as input and generate automated interpretation on 18 criteria.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/InterVar.html](https://hpc.nih.gov/apps/InterVar.html)

### iVar

- **Description:** iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. Additional tools for metagenomic sequencing are actively being incorporated into iVar. While each of these functions can be accomplished using existing tools, iVar contains an intersection of functionality from multiple tools that are required to call iSNVs and consensus sequences from viral sequencing data across multiple replicates.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/iVar.html](https://hpc.nih.gov/apps/iVar.html)

### jellyfish

- **Description:** Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/jellyfish.html](https://hpc.nih.gov/apps/jellyfish.html)

### kallisto

- **Description:** kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kallisto.html](https://hpc.nih.gov/apps/kallisto.html)

### LASTZ

- **Description:** LASTZ is a tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically. LASTZ is a drop-in replacement for BLASTZ, and is backward compatible with BLASTZ's command-line syntax.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/LASTZ.html](https://hpc.nih.gov/apps/LASTZ.html)

### ldsc

- **Description:** ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ldsc.html](https://hpc.nih.gov/apps/ldsc.html)

### m-tools

- **Description:** A selection of software developed at the Australian Centre for Ecogenomics to aid in the analysis of metagenomic datasets: unitem, refinem, checkm, graftm, groopm, bamm, finishm, singlem, orfm, and coverm
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://ecogenomic.org/m-tools](https://hpc.nih.gov/apps/https://ecogenomic.org/m-tools)

### magma

- **Description:** MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous GWAS or meta-analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/magma.html](https://hpc.nih.gov/apps/magma.html)

### mantis

- **Description:** Microsatellite Analysis for Normal-Tumor InStability is a program developed for detecting microsatellite instability from paired-end BAM files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mantis.html](https://hpc.nih.gov/apps/mantis.html)

### maps

- **Description:** a set of multiple scripts used to analyze PLAC-Seq and HiChIP data.

### MashMap

- **Description:** MashMap is an approximate algorithm for computing local alignment boundaries between long
  DNA sequences. Given a minimum alignment length and an identity threshold, it computes the
  desired alignment boundaries and identity estimates using kmer-based statistics,
  and maintains sufficient probabilistic guarantees on the output sensitivity.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MashMap.html](https://hpc.nih.gov/apps/MashMap.html)

### MELT

- **Description:** MELT is an application for identifying mobile elements in genomic data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://melt.igs.umaryland.edu/manual.php](https://hpc.nih.gov/apps/https://melt.igs.umaryland.edu/manual.php)

### meme

- **Description:** MEME is used to discover motifs in groups of DNA/protein sequences or databases.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Meme.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Meme.html)

### mfold

- **Description:** MFOLD predicts DNA and RNA secondary structure.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mfold.html](https://hpc.nih.gov/apps/mfold.html)

### MMARGE

- **Description:** MMARGE: Motif Mutation Analysis for Regulatory Genomic Elements
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/vlink/marge](https://hpc.nih.gov/apps/https://github.com/vlink/marge)

### MoChA

- **Description:** MoChA is a bcftools extension to call mosaic chromosomal alterations
  starting from phased VCF files
  with either B Allele Frequency (BAF) and Log R Ratio (LRR) or allelic depth (AD)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MoChA.html](https://hpc.nih.gov/apps/MoChA.html)

### MQLS

- **Description:** MQLS ("More Poweful" or "Modified" Quasi-likelihood Score Test) is a program for case-control association testing of a binary trait in samples that contain related individuals.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/msisensor-pro.html](https://hpc.nih.gov/apps/msisensor-pro.html)

### msisensor-pro

- **Description:** evaluates Microsatellite Instability (MSI) for cancer patients with next generation sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/msisensor-pro.html](https://hpc.nih.gov/apps/msisensor-pro.html)

### MUMmer

- **Description:** Mummer is a system for aligning entire genomes extremely rapidly.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mummer.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mummer.html)

### MUSCLE

- **Description:** Fast Multiple Sequence Alignment program.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/muscle.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/muscle.html)

### mutsig2cv

- **Description:** mutsig2cv analyzes somatic point mutations discovered in DNA sequencing, identifying genes mutated more often than expected by chance.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mutsig2cv.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/mutsig2cv.html)

### ncbi-toolkit

- **Description:** The NCBI C++ Toolkit is a set of executables and libraries for a multitude of sequence analysis functions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ncbi-toolkit.html](https://hpc.nih.gov/apps/ncbi-toolkit.html)

### netOglyc

- **Description:** NetOglyc produces neural network predictions of mucin type GalNAc O-glycosylation sites in mammalian proteins.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/netOglyc.html](https://hpc.nih.gov/apps/netOglyc.html)

### Nirvana

- **Description:** Nirvana provides clinical-grade annotation of genomic variants (SNVs, MNVs, insertions, deletions, indels, and SVs (including CNVs). It can be run as a stand-alone package or integrated into larger software tools that require variant annotation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Nirvana.html](https://hpc.nih.gov/apps/Nirvana.html)

### OligoArray

- **Description:** OligoArray computes gene specific oligonucleotides that are free of secondary structure for genome-scale oligonucleotide microarray construction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/OligoArray.html](https://hpc.nih.gov/apps/OligoArray.html)

### oncotator

- **Description:** Tool for annotating information onto genomic point mutations (SNPs/SNVs) and indels.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/oncotator.html](https://hpc.nih.gov/apps/oncotator.html)

### OpenCRAVAT

- **Description:** OpenCRAVAT is a new open source, scalable decision support system for
  variant and gene prioritization. It includses a modular resource catalog
  to maximize community and developer involvement, and as a result the catalog
  is being actively developed and growing every month. Resources made available via
  the store are well-suited for analysis of cancer, as well as Mendelian and complex
  diseases.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/OpenCRAVAT.html](https://hpc.nih.gov/apps/OpenCRAVAT.html)

### OptiType

- **Description:** OptiType is a HLA genotyping algorithm based on integer linear programming, capable of producing accurate 4-digit HLA genotyping predictions from NGS data by simultaneously selecting all major and minor HLA Class I alleles.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/optitype.html](https://hpc.nih.gov/apps/optitype.html)

### ORFfinder

- **Description:** ORF finder searches for open reading frames (ORFs) in the DNA sequence you enter. The program returns the range of each ORF, along with its protein translation. Use ORF finder to search newly sequenced DNA for potential protein encoding segments, verify predicted protein using newly developed SMART BLAST or regular BLASTP.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ORFfinder.html](https://hpc.nih.gov/apps/ORFfinder.html)

### PAINTOR

- **Description:** PAINTOR (Probabilistic Annotation INtegraTOR) is a probabilistic framework that integrates association strength with genomic functional annotation data to improve accuracy in selecting plausible causal variants for functional validation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PAINTOR.html](https://hpc.nih.gov/apps/PAINTOR.html)

### pangolin

- **Description:** Phylogenetic Assignment of Named Global Outbreak LINeages. PANGOLIN is a system for identifying phylogenetic COVID lineages that contribute most to active spread.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pangolin.html](https://hpc.nih.gov/apps/pangolin.html)

### PEPPER_deepvariant

- **Description:** PEPPER-Margin-DeepVariant is a haplotype-aware variant calling pipeline for processing third-generation
  nanopore sequence data. This pipeline is also applicable to PacBio HiFi data,
  providing an efficient solution with superior performance over the current WhatsHap-DeepVariant standard.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PEPPER_deepvariant.html](https://hpc.nih.gov/apps/PEPPER_deepvariant.html)

### phaser

- **Description:** phasing and Allele Specific Expression from RNA-seq. Performs haplotype phasing using read alignments in BAM format from both DNA and RNA based assays, and provides measures of haplotypic expression for RNA based assays.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/phaser.html](https://hpc.nih.gov/apps/phaser.html)

### PhIP-Stat

- **Description:** PhIP-Stat is a set of analysis tools for tools for PhIP-seq experiments.
  It allows for processing of PhIP-Seq raw data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/phip-stat.html](https://hpc.nih.gov/apps/phip-stat.html)

### phylowgs

- **Description:** This Python/C++ code is the accompanying software for the paper PhyloWGS: Reconstructing subclonal composition and evolution from whole-genome sequencing of tumors, with authors Amit G. Deshwar, Shankar Vembu, Christina K. Yung, Gun Ho Jang, Lincoln Stein, and Quaid Morris.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/phylowgs.html](https://hpc.nih.gov/apps/phylowgs.html)

### popsycle

- **Description:** A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet/Freemuxlet methods and auxiliary tools

### PRANK

- **Description:** PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences. PRANK is based on a novel algorithm that treats insertions correctly and avoids over-estimation of the number of deletion events.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PRANK.html](https://hpc.nih.gov/apps/PRANK.html)

### preseq

- **Description:** predicting library complexity and genome coverage in high-throughput sequencing
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/preseq.html](https://hpc.nih.gov/apps/preseq.html)

### primer3

- **Description:** Primer3 is a program for designing PCR primers, hybridization probes, and sequencing primers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/primer3.html](https://hpc.nih.gov/apps/primer3.html)

### prokka

- **Description:** Prokka is a software tool for the rapid annotation of prokaryotic genomes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/prokka.html](https://hpc.nih.gov/apps/prokka.html)

### PSIPRED

- **Description:** PSIPRED is a simple and accurate secondary structure prediction method, incorporating two feed-forward neural networks which perform an analysis on output obtained from PSI-BLAST (Position Specific Iterated - BLAST).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PSIPRED.html](https://hpc.nih.gov/apps/PSIPRED.html)

### pysamstats

- **Description:** Pysamstats is a fast Python and command-line utility for extracting simple statistics against genome positions based on sequence alignments from a SAM or BAM file.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pysamstats.html](https://hpc.nih.gov/apps/pysamstats.html)

### QTLtools

- **Description:** A tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) in few easy-to-perform steps.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/QTLtools.html](https://hpc.nih.gov/apps/QTLtools.html)

### randfold

- **Description:** RandFold computes the probability that, for a given RNA sequence, the Minimum Free Energy (MFE) of the secondary structure is different from a distribution of MFE computed with random sequences.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/randfold.html](https://hpc.nih.gov/apps/randfold.html)

### regenie

- **Description:** regenie is a C++ program for whole genome regression modelling of large genome-wide association studies. It is developed and supported by a team of scientists at the Regeneron Genetics Center. regenie employs the BGEN library.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/regenie.html](https://hpc.nih.gov/apps/regenie.html)

### rMATS

- **Description:** MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rMATS.html](https://hpc.nih.gov/apps/rMATS.html)

### rmblast

- **Description:** RMBlast is a RepeatMasker-compatible version of the standard NCBI blastn program. RMBlast supports RepeatMasker searches by adding a few necessary features to the stock NCBI blastn program.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rmblast.html](https://hpc.nih.gov/apps/rmblast.html)

### RNAmmer

- **Description:** RNAmmer predicts ribosomal RNA genes in full genome sequences by utilising two levels of Hidden Markov Models: An initial spotter model searches both strands. The spotter model is constructed from highly conserved loci within a structural alignment of known rRNA sequences. Once the spotter model detects an approximate position of a gene, flanking regions are extracted and parsed to the full model which matches the entire gene.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps)

### rnastructure

- **Description:** RNAstructure is a complete package for RNA and DNA secondary structure prediction and analysis. It includes algorithms for secondary structure prediction, including facility to predict base pairing probabilities. It also can be used to predict bimolecular structures and can predict the equilibrium binding affinity of an oligonucleotide to a structured RNA target.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rnastructure.html](https://hpc.nih.gov/apps/rnastructure.html)

### roary

- **Description:** Roary is a high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka) and calculates the pan genome.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/roary.html](https://hpc.nih.gov/apps/roary.html)

### ROSE

- **Description:** ROSE (Rank Ordering of Super-Enhancers) is tool for
  (1) creating stitched enhancers, and  
  (2) separating super-enhancers from typical enhancers.
  given sequencing data (.bam) and a file of previously identified constituent enhancers (.gff)
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ROSE.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ROSE.html)

### SAIGE

- **Description:** R package for large-scale genetic association studies.

### scanpy

- **Description:** Scanpy is a scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scanpy.html](https://hpc.nih.gov/apps/scanpy.html)

### scvelo

- **Description:** scVelo is a method to describe the rate of gene expression change for an individual gene at a given time point based on the ratio of its spliced and unspliced messenger RNA (mRNA).
  It avoids errors in the velocity estimates by solving the full transcriptional dynamics of splicing kinetics using a likelihood-based dynamical model.
  This generalizes RNA velocity to systems with transient cell states, which are common in development and in response to perturbations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scvelo.html](https://hpc.nih.gov/apps/scvelo.html)

### SEACR

- **Description:** SEACR is intended to call peaks and enriched regions from sparse Cleavage Under Targets and Release Using Nuclease (CUT&RUN) or chromatin profiling data in which background is dominated by "zeroes" (i.e. regions with no read coverage).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SEACR.html](https://hpc.nih.gov/apps/SEACR.html)

### selscan

- **Description:** selscan is a tool for haplotype-based scans to detect natural selection, which are useful to identify recent or ongoing positive selection in genomes. It is an efficient multithreaded application that implements Extended Haplotype Homozygosity (EHH), Integrated Haplotype Score (iHS), and Cross-population EHH (XPEHH). selscan accepts phased genotypes in multiple formats, including TPED.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/selscan.html](https://hpc.nih.gov/apps/selscan.html)

### signalp

- **Description:** SignalP predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive bacteria, Gram-negative bacteria, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on
  a combination of several artificial neural networks.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps)

### smcpp

- **Description:** SMC++ is a program for estimating the size history of populations from whole genome sequence data.

### smoove

- **Description:** smoove simplifies and speeds calling and genotyping SVs for short reads.
  It also improves specificity by removing many spurious alignment signals
  that are indicative of low-level noise and often contribute to spurious calls.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/smoove.html](https://hpc.nih.gov/apps/smoove.html)

### SOAPdenovo-Trans

- **Description:** SOAPdenovo-Trans is a de novo transcriptome
  assembler designed specifically for RNA-Seq.
  Its performance on transcriptome datasets from rice and mouse.
  It provides higher contiguity, lower redundancy and faster execution
  than other popular transcriptome assemblers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SOAPdenovo-Trans.html](https://hpc.nih.gov/apps/SOAPdenovo-Trans.html)

### sonicparanoid

- **Description:** A stand-alone software tool for the identification of orthologous relationships among multiple species.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sonicparanoid.html](https://hpc.nih.gov/apps/sonicparanoid.html)

### star-seqr

- **Description:** Star-seqr is a tool for identifying gene fusions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/starseqr.html](https://hpc.nih.gov/apps/starseqr.html)

### steme

- **Description:** An efficient accurate motif finder based on MEME and implemented using suffix arrays.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/steme.html](https://hpc.nih.gov/apps/steme.html)

### stripenn

- **Description:** Stripenn is a command line interface python package developed for detection of atchitectural stripes
  from chromatin conformation capture (3C) data. It implements an algorithm rooted in computer vision
  for demarcation and quantification of the architectural stripes. Stripenn was demonstrated to outperform
  existing methods, be applicable in the context of analysis of B and T lymphocytes, and to allow
  examination of the role of sequence variation on the architectural stripes by studying the conservation
  of these features in inbred strains of mice.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/stripenn.html](https://hpc.nih.gov/apps/stripenn.html)

### SurvivalGWAS_SV

- **Description:** SurvivalGWAS_SV is an easy to use software that is able to handle large scale genome-wide data,
  allowing for imputed genotypes by modelling time to event outcomes under a dosage model.
  The software can adjust for multiple covariates and incorporate SNP-covariate interaction effects.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SurvivalGWAS_SV.html](https://hpc.nih.gov/apps/SurvivalGWAS_SV.html)

### SynthDNM

- **Description:** SynthDNM is a random-forest based classifier that can be readily adapted
  to new sequencing or variant-calling pipelines
  by applying a flexible approach to constructing simulated
  training examples from real data. The optimized SynthDNM classifiers predict de novo SNPs and indels
  with robust accuracy across multiple methods of variant calling.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/SynthDNM.html](https://hpc.nih.gov/apps/SynthDNM.html)

### tantan

- **Description:** A tool to mask low complexity and short period tandem repeats
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tantan.html](https://hpc.nih.gov/apps/tantan.html)

### TelomereHunter

- **Description:** TelomereHunter is a software for the detailed characterization of telomere maintenance mechanism footprints in the genome. The tool is implemented for the analysis of large cancer genome cohorts and provides a variety of diagnostic diagrams as well as machine-readable output for subsequent analysis.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/telomerehunter.html](https://hpc.nih.gov/apps/telomerehunter.html)

### TMHMM

- **Description:** TMHMM predicts transmembrane helices in proteins.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/trinotate.html#deps)

### tombo

- **Description:** a suite of tools primarily for the identification of modified nucleotides from nanopore sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/tombo.html](https://hpc.nih.gov/apps/tombo.html)

### TRF

- **Description:** A tandem repeat in DNA is two or more adjacent, approximate copies of a pattern of nucleotides. Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/trf.html](https://hpc.nih.gov/apps/trf.html)

### trust4

- **Description:** Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. TRUST4 performs de novo assembly on
  V, J, C genes including the hypervariable complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to report the corresponding information. TRUST4 supports both single-end and paired-end sequencing data with any read length.

### unicycler

- **Description:** Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore) where it runs a miniasm+Racon pipeline.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/unicycler.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/unicycler.html)

### uropa

- **Description:** UROPA is a command line based tool for genomic region annotation
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/uropa.html](https://hpc.nih.gov/apps/uropa.html)

### usearch

- **Description:** USEARCH is a unique sequence analysis tool with thousands of users world-wide. USEARCH offers search and clustering algorithms that are often orders of magnitude faster than BLAST.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/usearch.html](https://hpc.nih.gov/apps/usearch.html)

### VADR

- **Description:** VADR stands for Viral Annotation DefineR. It is a suite of tools for classifying and analyzing sequences
  homologous to a set of reference models of viral genomes or gene families. It has been mainly tested for analysis of Norovirus, Dengue, and SARS-CoV-2 virus sequences
  in preparation for submission to the GenBank database.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VADR.html](https://hpc.nih.gov/apps/VADR.html)

### VarScan

- **Description:** A platform-independent, technology-independent software tool for identifying SNPs and indels in massively parallel sequencing of individual and pooled samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VarScan.html](https://hpc.nih.gov/apps/VarScan.html)

### VCF-kit

- **Description:** VCF-kit is a collection of utility tools for processing and analyzing the VCF (variant call format) files, including primer generation for variant validation, dendrogram production,genotype imputation from sequence data in linkage studies, and additional tools to be used by statistical and population geneticists.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VCF-kit.html](https://hpc.nih.gov/apps/VCF-kit.html)

### velocyto

- **Description:** Velocyto is a library for the analysis of RNA velocity. It includes a command line tool and an analysis pipeline.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/velocyto.html](https://hpc.nih.gov/apps/velocyto.html)

### viennarna

- **Description:** RNA Secondary Structure Prediction and Comparison
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/viennarna.html](https://hpc.nih.gov/apps/viennarna.html)

### viper

- **Description:** VIPER combines the use of several dozen RNA-seq tools, suites, and packages to create a complete pipeline that takes RNA-seq analysis from raw sequencing data all the way through alignment, quality control, unsupervised analyses, differential expression, and downstream pathway analysis
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/viper.html](https://hpc.nih.gov/apps/viper.html)

### VirSorter2

- **Description:** VirSorter2 is a DNA and RNA virus identification tool. It leverages genome-informed
  database advances across a collection of customized automatic classifiers
  to improve the accuracy and range of virus sequence detection.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VirSorter2.html](https://hpc.nih.gov/apps/VirSorter2.html)

### weblogo

- **Description:** contains seqlogo utility to create sequence logo summarizing sequence alignments
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/weblogo.html](https://hpc.nih.gov/apps/weblogo.html)

### WISExome

- **Description:** WISExome is the tool that implements a within-sample comparison approach
  to CNV detection. It correctly identifies known
  pathogenic CNVs.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/WISExome.html](https://hpc.nih.gov/apps/WISExome.html)

### xengsort

- **Description:** A fast xenograft read sorter based on space-efficient k-mer hashing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/xengsort.html](https://hpc.nih.gov/apps/xengsort.html)

### xenome

- **Description:** xenome is a tool for classifying reads from xenograft source.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/xenome.html](https://hpc.nih.gov/apps/xenome.html)

### xHLA

- **Description:** The HLA gene complex on human chromosome 6 is one of the
  most polymorphic regions in the human genome and contributes
  in large part to the diversity of the immune system. Accurate typing of HLA genes with short-read sequencing data has historically been difficult due to the sequence similarity between the polymorphic alleles. xHLA iteratively refines the mapping results at the amino acid level to
  achieve high typing accuracy for both class I and II HLA genes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/xHLA.html](https://hpc.nih.gov/apps/xHLA.html)

## Structural Biology

---

### alphafold2

- **Description:** This package provides an implementation of the protein structure inference pipeline of AlphaFold v2.0.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/alphafold2.html](https://hpc.nih.gov/apps/alphafold2.html)

### CCP4

- **Description:** CCP4 is a suite of programs for protein crystallography and structural biology.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/CCP4.html](https://hpc.nih.gov/apps/CCP4.html)

### cryoDRGN

- **Description:** CryoDRGN is an algorithm that leverages the representation power of deep neural networks
  to directly reconstruct continuous distributions of 3D density maps
  and map per-particle heterogeneity of single-particle cryo-EM datasets.
  It contains interactive tools to visualize a dataset’s distribution of per-particle
  variability, generate density maps for exploratory analysis, extract particle
  subsets for use with other tools and generate trajectories to visualize molecular motions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cryoDRGN.html](https://hpc.nih.gov/apps/cryoDRGN.html)

### crystfel

- **Description:** CrystFEL is a suite of programs for processing diffraction data acquired serially in a snapshot manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser source.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/crystfel.html](https://hpc.nih.gov/apps/crystfel.html)

### CSD

- **Description:** The Cambridge Structural Database is the world repository of small molecule crystal structures.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/csd.html](https://hpc.nih.gov/apps/csd.html)

### Dali

- **Description:** The three-dimensional co-ordinates of each protein are used to calculate residue - residue distance matrices.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/dali.html](https://hpc.nih.gov/apps/dali.html)

### DeepHiC

- **Description:** DeepHiC ("deep Hi-C") is a generative adversarial network aimed at predicting the high-resolution Hi-C contact maps from ithe low-coverage sequencing data.
  Hi-C is commonly used to study three-dimensional genome organization.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepHiC.html](https://hpc.nih.gov/apps/DeepHiC.html)

### DeepMM

- **Description:** DeepMM implements fully automated de novo structure modeling method, MAINMAST, which builds three-dimensional models of a protein from a near-atomic resolution EM map. The method directly traces the protein’s main-chain and identifies Cα positions as tree-graph structures in the EM map.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DeepMM.html](https://hpc.nih.gov/apps/DeepMM.html)

### DSSP

- **Description:** The DSSP program was designed by Wolfgang Kabsch and Chris Sander to standardize secondary structure assignment. DSSP is a database of secondary structure assignments (and much more) for all protein entries in the Protein Data Bank (PDB). DSSP is also the program that calculates DSSP entries from PDB entries. DSSP does not predict secondary structure.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DSSP.html](https://hpc.nih.gov/apps/DSSP.html)

### EM-GAN

- **Description:** EM-GAN is a tool for post-processing Electron Microscopy maps. It uses Generative Adversarial Networks to improve the resolution of the maps.

### emspring

- **Description:** SPRING (Single Particle Reconstruction from Images of kN own Geometry) is a single-particle based helical reconstruction package for electron cryo-micrographs and has been used to determine 3D structures of a variety of highly ordered and less ordered specimens.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/emspring.html](https://hpc.nih.gov/apps/emspring.html)

### gautomatch

- **Description:** Fully automatic acccurate, convenient and extremely fast particle picking for EM
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gautomatch.html](https://hpc.nih.gov/apps/gautomatch.html)

### I-TASSER

- **Description:** I-TASSER (Iterative Threading ASSEmbly Refinement) is a hierarchical approach to protein structure and function prediction.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/I-TASSER.html](https://hpc.nih.gov/apps/I-TASSER.html)

### lafter

- **Description:** LAFTER is a local filter for single particle TEM reconstructions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lafter.html](https://hpc.nih.gov/apps/lafter.html)

### lammps

- **Description:** LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. It runs on a variety of different computer systems, including single processor systems, distributed-memory machines with MPI, and GPU and Xeon Phi systems. LAMMPS is open source software, released under the GNU General Public License.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/lammps.html](https://hpc.nih.gov/apps/lammps.html)

### LocScale

- **Description:** LocScale is a reference-based local amplitude scaling tool using prior model information to improve contrast of cryo-EM density maps. It can be helpful in the common case of resolution variation in the 3D reconstruction and it can be used as an alternative to other commonly applied map sharpening methods.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/LocScale.html](https://hpc.nih.gov/apps/LocScale.html)

### mdtraj

- **Description:** MDTraj is a python library that allows users to manipulate molecular dynamics (MD) trajectories and perform a variety of analyses, including fast RMSD, solvent accessible surface area, hydrogen bonding, etc.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/mdtraj.html](https://hpc.nih.gov/apps/mdtraj.html)

### moabs

- **Description:** A comprehensive, accurate and efficient solution for analysis of large scale base-resolution DNA methylation data, bisulfite sequencing or single molecule direct sequencing.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/moabs.html](https://hpc.nih.gov/apps/moabs.html)

### mustache

- **Description:** Mustache is a tool for identifying chromatin loops from HiC and MicroC contact maps

### naccess

- **Description:** The naccess program calculates the atomic accessible surface defined by rolling a probe of given size around a van der Waals surface.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/naccess.html](https://hpc.nih.gov/apps/naccess.html)

### novactf

- **Description:** NovaCTF is a freeware for 3D-CTF correction for electron microscopy.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/novactf.html](https://hpc.nih.gov/apps/novactf.html)

### pf_refinement

- **Description:** Protofilament Refinement is a software package to further refine microtubule structures, by aligning individual protofilaments rather than full microtubule segements.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pf_refinement.html](https://hpc.nih.gov/apps/pf_refinement.html)

### Phenix

- **Description:** PHENIX is a software suite for the automated determination of macromolecular structures using X-ray crystallography and other methods.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Phenix.html](https://hpc.nih.gov/apps/Phenix.html)

### PROCHECK

- **Description:** PROCHECK checks the stereochemical quality of a protein structure, producing a number of PostScript plots analysing its overall and residue-by-residue geometry. It includes PROCHECK-NMR for checking the quality of structures solved by NMR.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PROCHECK](https://hpc.nih.gov/apps/PROCHECK)

### pydockrmsd

- **Description:** DockRMSD is capable of deterministically identifying the minimum symmetry-corrected RMSD and is able to do so without significant loss of computational efficiency compared to other methods.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pydockrmsd.html](https://hpc.nih.gov/apps/pydockrmsd.html)

### pyem

- **Description:** UCSF pyem is a collection of Python modules and command-line utilities for electron microscopy of biological samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/pyem.html](https://hpc.nih.gov/apps/pyem.html)

### PyMOL

- **Description:** A comprehensive molecular visualization product for rendering and animating 3D molecular structures.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PyMOL.html](https://hpc.nih.gov/apps/PyMOL.html)

### PyRosetta

- **Description:** PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling suite. It enables users to design their own custom molecular modeling algorithms using Rosetta sampling methods and energy functions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/PyRosetta.html](https://hpc.nih.gov/apps/PyRosetta.html)

### rDock

- **Description:** The rDock platform is a suite of command-line tools for high-throughput docking and virtual screening.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rDock.html](https://hpc.nih.gov/apps/rDock.html)

### Schrodinger

- **Description:** A limited number of Schrödinger applications are available on the Biowulf cluster through the Molecular Modeling Interest Group. Most are available through the Maestro GUI.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/schrodinger/index.html](https://hpc.nih.gov/apps/schrodinger/index.html)

### Scipion

- **Description:** Scipion is an image processing framework to obtain 3D models of macromolecular complexes using Electron Microscopy (3DEM). It integrates several software packages and presents an unified interface for both biologists and developers. Scipion allows to execute workflows combining different software tools, while taking care of formats and conversions. Additionally, all steps are tracked and can be reproduced later on.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Scipion/index.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/Scipion/index.html)

### scratch1d

- **Description:** SCRATCH-1D is a suite of one-dimensional predictors included in the long-established and widely used SCRATCH suite of predictors developed by the Institute for Genomics and Bioinformatics (IGB) of the University of California, Irvine (UCI).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scratch1d.html](https://hpc.nih.gov/apps/scratch1d.html)

### sidesplitter

- **Description:** Sidesplitter reduces over-fitting in both idealised and experimental settings, while maintaining
  independence between the two sides of a split refinement. It can improve the final resolution in refinements of structures prone to severe over-fitting, such as membrane proteins in detergent micelles.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sidesplitter.html](https://hpc.nih.gov/apps/sidesplitter.html)

### subtom

- **Description:** Subtom is a pipeline for subvolume alignment and averaging of electron cryo-tomography data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/subtom.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/subtom.html)

### symmetry

- **Description:** This project collects tools to detect, analyze, and visualize protein symmetry.

### VMD

- **Description:** VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. To use, type vmd at the prompt.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/VMD.html](https://hpc.nih.gov/apps/VMD.html)

### Xplor-NIH

- **Description:** Xplor-NIH is a structure determination program which builds on the X-PLOR v3.851 program, including additional tools developed at the NIH.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Xplor-NIH/index.html](https://hpc.nih.gov/apps/Xplor-NIH/index.html)

### ZDOCK

- **Description:** ZDOCK predicts protein-docking models, and uses a fast Fourier transform to search all possible binding modes for proteins, evaluating based on shape complementarity, desolvation energy, and electrostatics.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/ZDOCK.html](https://hpc.nih.gov/apps/ZDOCK.html)

## Systems Biology

---

### AMON

- **Description:** AMON (Annotation of Metabolite Origins via Networks) is an open-source bioinformatics application
  that can be used to (1) annotate which compounds in the metabolome could have been produced by bacteria present or the host; (2) evaluate the pathway enrichment of host verses microbial metabolites, and (3) to visualize which compounds may have been produced by host versus microbial enzymes in KEGG pathway maps.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/AMON.html](https://hpc.nih.gov/apps/AMON.html)

### cellphonedb

- **Description:** A publicly available repository of curated receptors, ligands and their interactions. Subunit architecture is included for both ligands and receptors, representing heteromeric complexes accurately.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cellphonedb.html](https://hpc.nih.gov/apps/cellphonedb.html)

### eggNOGmapper

- **Description:** eggNOGmapper is a tool for functional annotation of large sets of sequences based on fast orthology assignments using precomputed clusters and phylogenies from the eggNOG database. Orthology assignment is ideally suited for functional inference. However, predicting orthology is computationally intensive at large scale, and most
  other pipelines are relatively inaccessible (e.g., new assignments only available through database updates), so less precise homology-based functional transfer was previously the default for (meta-)genome annotation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/eggNOGmapper.html](https://hpc.nih.gov/apps/eggNOGmapper.html)

### EukRep

- **Description:** Microbial eukaryotes are integral components of natural microbial communities and their
  inclusion is critical for many ecosystem studies, yet the majority of published metagenome
  analyses ignore eukaryotes. In order to include eukaryotes in environmental studies,
  eukaryotic genomes shoould be recovered from complex metagenomic samples. A key step for genome recovery is separation of eukaryotic and prokaryotic fragments. EukRep is a kmer- and SVM-based strategy for eukaryotic sequence identification from environmental samples.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/EukRep.html](https://hpc.nih.gov/apps/EukRep.html)

### genesis

- **Description:** GENESIS (GEneral NEural SImulation System) is a software platform for the simulation of neural systems ranging from subcellular components and biochemical reactions to complex models of single neurons, large networks, and systems-level processes.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/genesis.html](https://hpc.nih.gov/apps/genesis.html)

### GRNBoost

- **Description:** GRNBoost is a library built on top of Apache Spark that implements a scalable strategy for gene regulatory network (GRN) inference. GRNBoost was inspired by GENIE3, a popular algorithm for GRN inference. GRNBoost adopts GENIE3's algorithmic blueprint and aims at improving its runtime performance and data size capability.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/GRNBoost.html](https://hpc.nih.gov/apps/GRNBoost.html)

### gsea

- **Description:** Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.gsea-msigdb.org/gsea/index.jsp](https://hpc.nih.gov/apps/https://www.gsea-msigdb.org/gsea/index.jsp)

### MAKER

- **Description:** MAKER is a portable and easily configurable genome annotation pipeline.
  Its purpose is to allow smaller eukaryotic and prokaryotic genome projects
  to independently annotate their genomes and to create genome databases.
  MAKER identifies repeats, aligns ESTs and proteins to a genome,
  produces ab-initio gene predictions and automatically synthesizes these data
  into gene annotations having evidence-based quality values.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MAKER.html](https://hpc.nih.gov/apps/MAKER.html)

### Neuron

- **Description:** NEURON is a simulation environment for modeling individual neurons and networks of neurons. It provides tools for conveniently building, managing, and using models in a way that is numerically sound and computationally efficient. It is particularly well-suited to problems that are closely linked to experimental data, especially those that involve cells with complex anatomical and biophysical properties.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Neuron.html](https://hpc.nih.gov/apps/Neuron.html)

### scvelo

- **Description:** scVelo is a method to describe the rate of gene expression change for an individual gene at a given time point based on the ratio of its spliced and unspliced messenger RNA (mRNA).
  It avoids errors in the velocity estimates by solving the full transcriptional dynamics of splicing kinetics using a likelihood-based dynamical model.
  This generalizes RNA velocity to systems with transient cell states, which are common in development and in response to perturbations.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/scvelo.html](https://hpc.nih.gov/apps/scvelo.html)

### vdjtools

- **Description:** A comprehensive analysis framework for T-cell and B-cell repertoire sequencing data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vdjtools.html](https://hpc.nih.gov/apps/vdjtools.html)

## Utilities

---

### aria2

- **Description:** multiprotocol download utility
  Type 'module load aria2'then 'aria2c --help' for more info.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/aria2.html](https://hpc.nih.gov/apps/aria2.html)

### ariba

- **Description:** Antimicrobial resistance identification.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ariba.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ariba.html)

### asciinema

- **Description:** asciinema [as-kee-nuh-muh] is a free and open source solution for recording terminal sessions and sharing them.
  Type 'module load asciinema' then 'asciinema' to run.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://asciinema.org/](https://hpc.nih.gov/apps/https://asciinema.org/)

### Aspera

- **Description:** High-speed fasp-powered file transfers. Mostly used to download data from NCBI, which has an Aspera server. See the data transfer page for details.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps//docs/transfer.html](https://hpc.nih.gov/apps//docs/transfer.html)

### AWS

- **Description:** Command-line tools for Amazon Web Services. Use
  'module load python; aws -help' to see the command-line help, or http://aws.amazon.com/cli/.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://aws.amazon.com/cli/](https://hpc.nih.gov/apps/http://aws.amazon.com/cli/)

### azcopy

- **Description:** a command-line utility to copy blobs or files to or from Azure storage

### bbcp

- **Description:** Secure and fast copy utility

### bgionline

- **Description:** BGI Online is a cloud platform for bioinformatic analysis. The BGI online tools can be used for downloading data and more. Use 'module load bgionline' to access the tools. [Documentation]
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps//docs/bgionlinecli.pdf](https://hpc.nih.gov/apps//docs/bgionlinecli.pdf)

### chance

- **Description:** CHANCE performs CHiP-seq analytics and confidence estimation.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/chance.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/chance.html)

### circos

- **Description:** Circos is a program for the generation of publication-quality, circularly composited renditions of genomic data and related annotations. Circos is particularly suited for visualizing alignments, conservation and intra and inter-chromosomal relationships. Also, Circos is useful to visualize any type of information that benefits from a circular layout. Thus, although it has been designed for the field of genomics, it is sufficiently flexible to be used in other data domains.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/circos.html](https://hpc.nih.gov/apps/circos.html)

### coreutils

- **Description:** The GNU Core Utilities are the basic file, shell and text manipulation utilities of the GNU operating system.

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://www.gnu.org/software/coreutils/manual/](https://hpc.nih.gov/apps/http://www.gnu.org/software/coreutils/manual/)

### cpdf

- **Description:** Coherent PDF tools
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.coherentpdf.com/cpdfmanual.pdf](https://hpc.nih.gov/apps/https://www.coherentpdf.com/cpdfmanual.pdf)

### crumble

- **Description:** Crumble is a tool to explore controlled loss of quality values for compressing CRAM files. Crumble can read a SAM/BAM/CRAM file, compute which confidence values to keep and which to omit, and emit a new file with most qualities removed.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/crumble.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/crumble.html)

### cryolo

- **Description:** Automated particle picker for cryo-EM
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/cryolo.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/cryolo.html)

### datalad

- **Description:** Datalad is a tool for uploading and downloading public up-t-to-date neuroimaging datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/datalad.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/datalad.html)

### datamash

- **Description:** GNU datamash is a command-line program which performs basic numeric,textual and statistical operations on input textual data files.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.gnu.org/software/datamash/](https://hpc.nih.gov/apps/https://www.gnu.org/software/datamash/)

### DNAnexus

- **Description:** DNAnexus is a cloud-based commercial solution for next-generation sequence analysis and visualization. It has a command-line interface (CLI) which can be used to log in to the DNAnexus platform, upload and navigate data, and launch analyses.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/DNAnexus.html](https://hpc.nih.gov/apps/DNAnexus.html)

### EDirect

- **Description:** Entrez Direct (EDirect) is an advanced method for accessing the NCBI's set of interconnected databases (publication, sequence, structure, gene, variation, expression, etc.) from a UNIX terminal window.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/edirect.html](https://hpc.nih.gov/apps/edirect.html)

### Editors

- **Description:** Text Editors available on the systems
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/Editors.html](https://hpc.nih.gov/apps/Editors.html)

### gdc-client

- **Description:** The GDC Data Transfer Tool provides an optimized method of transferring data to and from the GDC, and enables resumption of interrupted transfers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gdc-client.html](https://hpc.nih.gov/apps/gdc-client.html)

### gdrive

- **Description:** gdrive is a command line utility for interacting with Google Drive.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/prasmussen/gdrive](https://hpc.nih.gov/apps/https://github.com/prasmussen/gdrive)

### gen3-client

- **Description:** The gen3-client provides an easy-to-use, command-line interface for uploading and downloading data files to and from a Gen3 data commons from the terminal or command prompt.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/gen3-client.html](https://hpc.nih.gov/apps/gen3-client.html)

### Ghostscript

- **Description:** Ghostscript is an interpreter for the PostScript language and for PDF.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.ghostscript.com/](https://hpc.nih.gov/apps/https://www.ghostscript.com/)

### gnuplot

- **Description:** Gnuplot is a portable command-line driven graphing utility to visualize mathematical functions and data interactively, and can support many non-interactive uses such as web scripting. Type 'gnuplot' to run, or 'module avail gnuplot' to see other available versions.

### google-cloud-sdk

- **Description:** Google Cloud SDK is a set of tools that you can use to manage resources and applications hosted on Google Cloud Platform. These include the gcloud, gsutil, and bq command line tools. See docs at https://cloud.google.com/sdk/docs/how-to.
  Type 'module load google-cloud-sdk' to use on Biowulf.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://cloud.google.com/sdk/docs/how-to](https://hpc.nih.gov/apps/https://cloud.google.com/sdk/docs/how-to)

### Grace

- **Description:** Grace is a WYSIWYG 2D plotting tool for the X-Window system. It is a successor to Xmgr.
  Type 'module load grace', then 'xmgrace' or 'gracebat' to run.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://plasma-gate.weizmann.ac.il/Grace/](https://hpc.nih.gov/apps/http://plasma-gate.weizmann.ac.il/Grace/)

### graphviz

- **Description:** Graphviz is open source graph visualization software. Graph visualization is a way of representing structural information as diagrams of abstract graphs and networks. It has important applications in networking, bioinformatics, software engineering, database and web design, machine learning, and in visual interfaces for other technical domains.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://graphviz.gitlab.io/](https://hpc.nih.gov/apps/https://graphviz.gitlab.io/)

### groff

- **Description:** Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands and produces formatted output. Output may be PostScript or PDF, html, or ASCII/UTF8 for display at the terminal. Formatting commands may be either low-level typesetting requests (“primitives”) or macros from a supplied set. Users may also write their own macros. All three may be combined.

### gzip

- **Description:** the data compression program
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.gnu.org/software/gzip/](https://hpc.nih.gov/apps/https://www.gnu.org/software/gzip/)

### h5util

- **Description:** h5utils is a set of utilities for visualization and conversion of scientific data in the free, portable HDF5 format. Type 'module load h5utils' to access the executables (e.g. h5topng)

### ImageMagick

- **Description:** ImageMagick is a software suite to create, edit, compose, or convert bitmap images. It can read and write images in a variety of formats. The ImageMagick tools are available by default (e.g. type 'convert') or use 'module load ImageMagick' to load the latest versions.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://www.imagemagick.org/](https://hpc.nih.gov/apps/http://www.imagemagick.org/)

### jo

- **Description:** A small utility to create JSON objects from command line arguments.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/jpmens/jo/blob/master/jo.md](https://hpc.nih.gov/apps/https://github.com/jpmens/jo/blob/master/jo.md)

### jq

- **Description:** Command line json processor
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://stedolan.github.io/jq/manual](https://hpc.nih.gov/apps/https://stedolan.github.io/jq/manual)

### kronatools

- **Description:** Krona allows hierarchical data to be explored with zooming, multi-layered pie charts. Krona charts can be created using an Excel template or KronaTools, which includes support for several bioinformatics tools and raw data formats. The interactive charts are self-contained and can be viewed with any modern web browser.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/kronatools.html](https://hpc.nih.gov/apps/kronatools.html)

### lbzip2

- **Description:** lbzip2 is a multi-threaded compression utility with support for bzip2 compressed file format.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://lbzip2.org/](https://hpc.nih.gov/apps/http://lbzip2.org/)

### longshot

- **Description:** Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/longshot.html](https://hpc.nih.gov/apps/longshot.html)

### mc

- **Description:** GNU Midnight Commander is a visual file manager, with a feature rich full-screen text mode application that allows you to copy, move and delete files and whole directory trees, search for files and run commands in the subshell. Type module load mc and then the command mc to get started.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://midnight-commander.org](https://hpc.nih.gov/apps/https://midnight-commander.org)

### mercurial

- **Description:** mercurial is a version control system that runs within Python. To use, type ml python/2.7. For help, type hg --help.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.mercurial-scm.org/wiki/Mercurial](https://hpc.nih.gov/apps/https://www.mercurial-scm.org/wiki/Mercurial)

### MySQL

- **Description:** MySQL is an open-source relational database management system.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/MySQL.html](https://hpc.nih.gov/apps/MySQL.html)

### nda-tools

- **Description:** In order to submit data to the National Institute of Mental Health Data Archives (NDA),
  users must validate their data to ensure it complies with the required format.
  This is done using the NDA validation tool, vtcmd. Additionally, users can package and download data from NDA as well, using the downloadcmd tool.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/nda-tools.html](https://hpc.nih.gov/apps/nda-tools.html)

### neovim

- **Description:** Neovim is a refactor, and sometimes redactor, in the tradition of Vim (which itself derives from Stevie). It is not a rewrite but a continuation and extension of Vim.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://neovim.io/](https://hpc.nih.gov/apps/https://neovim.io/)

### netpbm

- **Description:** Netpbm is a toolkit for manipulation of graphic images, including conversion of images between a variety of different formats. There are over 300 separate tools in the package including converters for about 100 graphics formats. Examples of the sort of image manipulation we're talking about are: Shrinking an image by 10%; Cutting the top half off of an image; Making a mirror image; Creating a sequence of images that fade from one image to another.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://netpbm.sourceforge.net/doc/](https://hpc.nih.gov/apps/http://netpbm.sourceforge.net/doc/)

### nvchecker

- **Description:** nvchecker (short for new version checker) is for checking if a new version of some software has been released.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/lilydjwg/nvchecker](https://hpc.nih.gov/apps/https://github.com/lilydjwg/nvchecker)

### parallel

- **Description:** GNU parallel is a shell tool for executing jobs in parallel using one or more computers.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://www.gnu.org/software/parallel/](https://hpc.nih.gov/apps/http://www.gnu.org/software/parallel/)

### paraview

- **Description:** ParaView is an open-source, multi-platform data analysis and visualization application.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/paraview.html](https://hpc.nih.gov/apps/paraview.html)

### patchelf

- **Description:** patchelf is a small utility to modify the dynamic linker and RPATH of ELF executables.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://nixos.org/patchelf.html](https://hpc.nih.gov/apps/http://nixos.org/patchelf.html)

### pdf2svg

- **Description:** A simple PDF to SVG converter using the Poppler and Cairo libraries.

### pigz

- **Description:** pigz (parallel implementation of gzip) is a fully functional replacement for gzip that exploits multiple processors and multiple cores to the hilt when compressing data.

### pilon

- **Description:** Pilon is a software tool which can be used to:

1. Automatically improve draft assemblies
2. Find variation among strains, including large event detection

- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/pilon.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/pilon.html)

### POVRay

- **Description:** POVRAY (Persistence of Vision RAYtracer) is a high-quality tool for creating three-dimensional graphics. Raytraced images are publication-quality and 'photo-realistic', but are computationally expensive so that large images can take many hours to create.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/POVRay/index.html](https://hpc.nih.gov/apps/POVRay/index.html)

### pysurfer

- **Description:** PySurfer is a Python library for visualizing brain surfaces produced by neuroimaging datasets.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/pysurfer.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/pysurfer.html)

### rclone

- **Description:** Rclone is a utility for synchronizing directories on a file-based storage system (e.g. /home or /data) with an object store such as Amazon S3. It uses the S3 protocol, and it can be used with the HPC object storage system.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/rclone.html](https://hpc.nih.gov/apps/rclone.html)

### rdfind

- **Description:** rdfind is a program that finds duplicate files. It is useful for compressing backup directories or just finding duplicate files. It compares files based on their content, NOT on their file names. After typing module load rdfind, type man rdfind for more information.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://rdfind.pauldreik.se/](https://hpc.nih.gov/apps/https://rdfind.pauldreik.se/)

### rfmix

- **Description:** Local Ancestry and Admixture Inference
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/rfmix.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/rfmix.html)

### sb_cli

- **Description:** Use the Seven Bridges Command Line Interface (SB CLI) to programmatically access and automate your interaction with the Platform via the API. The CLI is called by a simple command: sb.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/sb_cli.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/sb_cli.html)

### screen

- **Description:** Screen is a full-screen window manager that multiplexes a physical terminal between several processes, typically interactive shells.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.gnu.org/software/screen/](https://hpc.nih.gov/apps/https://www.gnu.org/software/screen/)

### singularity

- **Description:** Singularity is a container platform focused on supporting `Mobility of Compute`. It allows users to emulate, and share custom Linux environments allowing for the creation of self-contained development stacks.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/singularity.html](https://hpc.nih.gov/apps/singularity.html)

### snp-sites

- **Description:** Rapidly extracts SNPs from a multi-FASTA alignment
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/snp-sites.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/snp-sites.html)

### spark

- **Description:** Apache Spark is a fast and general engine for large-scale data processing. It is commonly used as an in-memory alternative to Hadoop MapReduce.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/spark.html](https://hpc.nih.gov/apps/spark.html)

### SQLite

- **Description:** SQLite is a software library that implements a self-contained, serverless, zero-configuration, transactional SQL database engine.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/sqlite.html](https://hpc.nih.gov/apps/sqlite.html)

### stow

- **Description:** GNU Stow is a symlink farm manager which takes distinct packages of software and/or data located in separate directories on the filesystem, and makes them appear to be installed in the same place.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/stow.html](https://hpc.nih.gov/apps/stow.html)

### swarm

- **Description:** Swarm is a script designed to simplify submitting a group of commands to the Biowulf cluster.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/swarm.html](https://hpc.nih.gov/apps/swarm.html)

### swig

- **Description:** SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/http://www.swig.org/](https://hpc.nih.gov/apps/http://www.swig.org/)

### synapseclient

- **Description:** The synapseclient package provides an interface to Synapse, a collaborative workspace for reproducible, data intensive research projects
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/synapseclient.html](https://hpc.nih.gov/apps/synapseclient.html)

### sysbench

- **Description:** sysbench is a scriptable multi-threaded benchmark tool based on LuaJIT. It is most frequently used for database benchmarks, but can also be used to create arbitrarily complex workloads that do not involve a database server.

### texinfo

- **Description:** Texinfo is the official documentation format of the GNU project

### tmux

- **Description:** tmux is a terminal multiplexer.
  Type 'module load tmux' to load the module, then 'tmux --help'
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/tmux](https://hpc.nih.gov/apps/https://github.com/tmux)

### UKBB

- **Description:** UK Biobank tools for downloading and processing UKBB data. Type 'module load ukbb' to access them.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ukbb.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/ukbb.html)

### vartrix

- **Description:** VarTrix is a software tool for extracting single cell variant information from 10x Genomics single cell data.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/vartrix.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/vartrix.html)

### vcf2db

- **Description:** vcf2db creates a gemini-compatible database from a VCF.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vcf2db.html](https://hpc.nih.gov/apps/vcf2db.html)

### vim

- **Description:** a text editor that is upwards compatible to Vi. It can be used to edit all kinds of plain text. It is especially useful for editing programs with syntactical coloring.

### visidata

- **Description:** VisiData is an interactive multitool for tabular data
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://visidata.org](https://hpc.nih.gov/apps/https://visidata.org)

### vscode

- **Description:** Free source code editor with many utilities for Python, Julia and others.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/vscode.html](https://hpc.nih.gov/apps/vscode.html)

### whatshap

- **Description:** WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly. It is especially suitable for long reads, but works also well with short reads.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/whatshap.html](https://hpc.nih.gov/apps/https://hpc.nih.gov/apps/whatshap.html)

### wuzz

- **Description:** Interactive cli tool for HTTP inspection
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/asciimoo/wuzz](https://hpc.nih.gov/apps/https://github.com/asciimoo/wuzz)

### xclip

- **Description:** xclip is a command line utility that is designed to run on any system with an X11 implementation. It provides an interface to X selections ("the clipboard") from the command line. It can read data from standard in or a file and place it in an X selection for pasting into other X applications. xclip can also print an X selection to standard out, which can then be redirected to a file or another program.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://github.com/astrand/xclip](https://hpc.nih.gov/apps/https://github.com/astrand/xclip)

### xpdf

- **Description:** Xpdf is a free PDF viewer and toolkit, including a text extractor, image converter, HTML converter, and more. Most of the tools are available as open source.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/https://www.xpdfreader.com/](https://hpc.nih.gov/apps/https://www.xpdfreader.com/)

## Workflow Managers

---

### cromwell

- **Description:** A Workflow Management System geared towards scientific workflows.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/cromwell.html](https://hpc.nih.gov/apps/cromwell.html)

### croo

- **Description:** A Python package for organizing outputs from Cromwell. Croo parses metadata.json which is an output from Cromwell and makes an organized directory with a copy (or a soft link) of each output file as described in an output definition JSON file specified by --out-def-json.

### nextflow

- **Description:** Data-driven computational pipelines
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/nextflow.html](https://hpc.nih.gov/apps/nextflow.html)

### snakemake

- **Description:** Snakemake aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification language (DSL) in python style. It is well suited for bioinformatic workflows.
- **NIH Site Reference:** Documentation and possibly Benchmarks [https://hpc.nih.gov/apps/snakemake.html](https://hpc.nih.gov/apps/snakemake.html)

## Appendex

---

<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->

<!-- code_chunk_output -->

- [lassl](#lassl)
  - [Puget Systems - Long Scientific Software List](#puget-systems-long-scientific-software-list)
- [Scientific Applications](#scientific-applications)
  - [More catagories TO-DO](#more-catagories-to-do)
  - [Computational Chemistry](#computational-chemistry)
    - [Acemd](#acemd)
    - [AMBER](#amber)
    - [AMPL](#ampl)
    - [APBS](#apbs)
    - [Autodock](#autodock)
    - [AutodockVina](#autodockvina)
    - [campari](#campari)
    - [CHARMM](#charmm)
    - [cryoDRGN](#cryodrgn)
    - [GAMESS](#gamess)
    - [Gaussian](#gaussian)
    - [Gromacs](#gromacs)
    - [NAMD](#namd)
    - [ORCA](#orca)
    - [Psi4](#psi4)
    - [Q-Chem](#q-chem)
    - [Schrodinger](#schrodinger)
    - [VMD](#vmd)
  - [Deep Learning](#deep-learning)
    - [ARDISS](#ardiss)
    - [B-SOiD](#b-soid)
    - [BioBERT](#biobert)
    - [BioGANs](#biogans)
    - [CANDLE](#candle)
    - [Clairvoyante](#clairvoyante)
    - [cryoDRGN](#cryodrgn-1)
    - [cuDNN](#cudnn)
    - [DanQ](#danq)
    - [DeepEmhancer](#deepemhancer)
    - [DeepHiC](#deephic)
    - [DeepHit](#deephit)
    - [DeepLab](#deeplab)
    - [DeepLabCut](#deeplabcut)
    - [DeepM6A](#deepm6a)
    - [DeepMM](#deepmm)
    - [digits](#digits)
    - [meka](#meka)
    - [ReLeaSE](#release)
    - [RoseTTAFold](#rosettafold)
    - [SpliceAI](#spliceai)
    - [tensorrt](#tensorrt)
    - [Tybalt](#tybalt)
    - [UNet](#unet)
    - [YOLO](#yolo)
  - [High-Throughput Sequencing](#high-throughput-sequencing)
    - [ABC](#abc)
    - [abruijn](#abruijn)
    - [abyss](#abyss)
    - [adam](#adam)
    - [AfterQC](#afterqc)
    - [albacore](#albacore)
    - [angsd](#angsd)
    - [AnnotSV](#annotsv)
    - [ANNOVAR](#annovar)
    - [Arioc](#arioc)
    - [ascatNgs](#ascatngs)
    - [atac_dnase_pipelines](#atac_dnase_pipelines)
    - [atropos](#atropos)
    - [augustus](#augustus)
    - [bam2fastq](#bam2fastq)
    - [bam2mpg](#bam2mpg)
    - [bamliquidator](#bamliquidator)
    - [bamreadcount](#bamreadcount)
    - [BAMSurgeon](#bamsurgeon)
    - [bamtools](#bamtools)
    - [bamUtil](#bamutil)
    - [Bartender](#bartender)
    - [basespace_cli](#basespace_cli)
    - [bazam](#bazam)
    - [bbtools](#bbtools)
    - [bcbio-nextgen](#bcbio-nextgen)
    - [bcl-convert](#bcl-convert)
    - [bcl2fastq](#bcl2fastq)
    - [bedops](#bedops)
    - [bedtools](#bedtools)
    - [BETA](#beta)
    - [BFC](#bfc)
    - [bigSCale2](#bigscale2)
    - [binlorry](#binlorry)
    - [bioawk](#bioawk)
    - [biobambam2](#biobambam2)
    - [biom-format](#biom-format)
    - [bismark](#bismark)
    - [bison](#bison)
    - [bonito](#bonito)
    - [bowtie](#bowtie)
    - [bowtie2](#bowtie2)
    - [bpipe](#bpipe)
    - [BRASS](#brass)
    - [breakdancer](#breakdancer)
    - [breseq](#breseq)
    - [bsmap](#bsmap)
    - [busco](#busco)
    - [bwa](#bwa)
    - [bwa-mem2](#bwa-mem2)
    - [CADD](#cadd)
    - [Canu](#canu)
    - [Canvas](#canvas)
    - [casper](#casper)
    - [caveman](#caveman)
    - [ccbrpipeliner](#ccbrpipeliner)
    - [ceas](#ceas)
    - [cellprofiler](#cellprofiler)
    - [cellranger](#cellranger)
    - [cellranger-arc](#cellranger-arc)
    - [cellranger-atac](#cellranger-atac)
    - [cellranger-dna](#cellranger-dna)
    - [cellsnp](#cellsnp)
    - [cellsnp-lite](#cellsnp-lite)
    - [cgpBattenberg](#cgpbattenberg)
    - [CHESS](#chess)
    - [ChIPseeqer](#chipseeqer)
    - [ChromHMM](#chromhmm)
    - [cicero](#cicero)
    - [circaidme](#circaidme)
    - [circexplorer2](#circexplorer2)
    - [circlator](#circlator)
    - [circos](#circos)
    - [circtools](#circtools)
    - [Clair3](#clair3)
    - [clark](#clark)
    - [clinEff](#clineff)
    - [cnvkit](#cnvkit)
    - [cnvnator](#cnvnator)
    - [cogentap](#cogentap)
    - [combp](#combp)
    - [conpair](#conpair)
    - [CREST](#crest)
    - [crispresso](#crispresso)
    - [crossmap](#crossmap)
    - [csvkit](#csvkit)
    - [ctk](#ctk)
    - [cufflinks](#cufflinks)
    - [cutadapt](#cutadapt)
    - [CutRunTools2](#cutruntools2)
    - [DANPOS](#danpos)
    - [deepconsensus](#deepconsensus)
    - [DeepM6A](#deepm6a-1)
    - [deepsignal](#deepsignal)
    - [deeptools](#deeptools)
    - [deepvariant](#deepvariant)
    - [defuse](#defuse)
    - [delly](#delly)
    - [DETONATE](#detonate)
    - [distiller-nf](#distiller-nf)
    - [dosageconvertor](#dosageconvertor)
    - [drop](#drop)
    - [dropseq](#dropseq)
    - [drseq](#drseq)
    - [ea-utils](#ea-utils)
    - [encode-atac-seq-pipeline](#encode-atac-seq-pipeline)
    - [EPACTS](#epacts)
    - [epic2](#epic2)
    - [EricScript](#ericscript)
    - [eukdetect](#eukdetect)
    - [exomiser](#exomiser)
    - [express](#express)
    - [fade](#fade)
    - [fanc](#fanc)
    - [fastqc](#fastqc)
    - [fastqtools](#fastqtools)
    - [fastq_screen](#fastq_screen)
    - [fastxtoolkit](#fastxtoolkit)
    - [flappie](#flappie)
    - [flashpca](#flashpca)
    - [Flexbar](#flexbar)
    - [flye](#flye)
    - [fqtools](#fqtools)
    - [freebayes](#freebayes)
    - [freec](#freec)
    - [fusioncatcher](#fusioncatcher)
    - [GangSTR](#gangstr)
    - [GATK](#gatk)
    - [gem](#gem)
    - [Gemini](#gemini)
    - [GEMMA](#gemma)
    - [Genome Browser](#genome-browser)
    - [genomestrip](#genomestrip)
    - [genrich](#genrich)
    - [GeoMX NGS Pipeline](#geomx-ngs-pipeline)
    - [Gistic](#gistic)
    - [gmap-gsnap](#gmap-gsnap)
    - [gridss](#gridss)
    - [gtex_rnaseq](#gtex_rnaseq)
    - [gtool](#gtool)
    - [guppy](#guppy)
    - [Hail](#hail)
    - [hap-ibd](#hap-ibd)
    - [hap.py](#happy)
    - [hgvs](#hgvs)
    - [hicexplorer](#hicexplorer)
    - [hichipper](#hichipper)
    - [hicpro](#hicpro)
    - [hint](#hint)
    - [hipstr](#hipstr)
    - [hisat](#hisat)
    - [HLA-LA](#hla-la)
    - [HMMRATAC](#hmmratac)
    - [homer](#homer)
    - [hrdetect](#hrdetect)
    - [htgts](#htgts)
    - [htseq](#htseq)
    - [humann](#humann)
    - [icgc-get](#icgc-get)
    - [IDBA](#idba)
    - [idep](#idep)
    - [iDiffIR](#idiffir)
    - [IDR](#idr)
    - [IGV](#igv)
    - [IGVTools](#igvtools)
    - [IMPUTE](#impute)
    - [InterVar](#intervar)
    - [iSAAC](#isaac)
    - [iva](#iva)
    - [iVar](#ivar)
    - [Jacquard](#jacquard)
    - [JAMM](#jamm)
    - [Juicer](#juicer)
    - [jvarkit](#jvarkit)
    - [kaiju](#kaiju)
    - [Katuali](#katuali)
    - [KMC](#kmc)
    - [KmerGenie](#kmergenie)
    - [kneaddata](#kneaddata)
    - [kraken](#kraken)
    - [KrakenUniq](#krakenuniq)
    - [laser](#laser)
    - [leafcutter](#leafcutter)
    - [lefse](#lefse)
    - [locuszoom](#locuszoom)
    - [lofreq](#lofreq)
    - [lumpy](#lumpy)
    - [mach2qtl](#mach2qtl)
    - [macs](#macs)
    - [mafft](#mafft)
    - [MAGeCK](#mageck)
    - [mageck-vispr](#mageck-vispr)
    - [MAGGIE](#maggie)
    - [magicblast](#magicblast)
    - [MAJIQ](#majiq)
    - [manorm](#manorm)
    - [manta](#manta)
    - [maps](#maps)
    - [mapsplice](#mapsplice)
    - [mash](#mash)
    - [medaka](#medaka)
    - [medusa](#medusa)
    - [MEGAHIT](#megahit)
    - [megalodon](#megalodon)
    - [MEGAN](#megan)
    - [Meryl](#meryl)
    - [metabat](#metabat)
    - [metal](#metal)
    - [metaphlan](#metaphlan)
    - [minialign](#minialign)
    - [miniasm](#miniasm)
    - [minimac](#minimac)
    - [minimap2](#minimap2)
    - [mirdeep2](#mirdeep2)
    - [misopy](#misopy)
    - [mitosuite](#mitosuite)
    - [mixcr](#mixcr)
    - [mixer](#mixer)
    - [MonoVar](#monovar)
    - [MonSTR](#monstr)
    - [morgan](#morgan)
    - [mosaicforecast](#mosaicforecast)
    - [mosdepth](#mosdepth)
    - [mscentipede](#mscentipede)
    - [mtoolbox](#mtoolbox)
    - [multiqc](#multiqc)
    - [MuSE](#muse)
    - [muTect](#mutect)
    - [MutSig](#mutsig)
    - [nanopolish](#nanopolish)
    - [neusomatic](#neusomatic)
    - [NGMLR](#ngmlr)
    - [ngscheckmate](#ngscheckmate)
    - [ngsplot](#ngsplot)
    - [novocraft](#novocraft)
    - [ont-fast5-api](#ont-fast5-api)
    - [opera-lg](#opera-lg)
    - [parpipe](#parpipe)
    - [PartekFlow](#partekflow)
    - [pbipa](#pbipa)
    - [pbsuite](#pbsuite)
    - [peakranger](#peakranger)
    - [pear](#pear)
    - [peddy](#peddy)
    - [PennCNV](#penncnv)
    - [PEPATAC](#pepatac)
    - [PePr](#pepr)
    - [picard](#picard)
    - [picrust](#picrust)
    - [pindel](#pindel)
    - [pirs](#pirs)
    - [plastid](#plastid)
    - [Platypus](#platypus)
    - [plink](#plink)
    - [plinkseq](#plinkseq)
    - [pomoxis](#pomoxis)
    - [porechop](#porechop)
    - [poretools](#poretools)
    - [preseq](#preseq)
    - [PRINSEQ](#prinseq)
    - [proseq](#proseq)
    - [PROVEAN](#provean)
    - [purge_haplotigs](#purge_haplotigs)
    - [pvactools](#pvactools)
    - [pychopper](#pychopper)
    - [pyclone](#pyclone)
    - [pycoQC](#pycoqc)
    - [pyega3](#pyega3)
    - [PyLOH](#pyloh)
    - [qcat](#qcat)
    - [QoRTs](#qorts)
    - [QoRTs](#qorts-1)
    - [qualimap](#qualimap)
    - [quast](#quast)
    - [racon](#racon)
    - [raremetal](#raremetal)
    - [Rcorrector](#rcorrector)
    - [READemption](#reademption)
    - [RepeatMasker](#repeatmasker)
    - [repeatmodeler](#repeatmodeler)
    - [rgt](#rgt)
    - [ricopili](#ricopili)
    - [rilseq](#rilseq)
    - [rnaseqc](#rnaseqc)
    - [rockhopper](#rockhopper)
    - [ROSE](#rose)
    - [RSD](#rsd)
    - [rsem](#rsem)
    - [rseqc](#rseqc)
    - [rtg-tools](#rtg-tools)
    - [rvtests](#rvtests)
    - [sailfish](#sailfish)
    - [salmon](#salmon)
    - [SalmonTE](#salmonte)
    - [sambamba](#sambamba)
    - [samblaster](#samblaster)
    - [samtools](#samtools)
    - [scallop](#scallop)
    - [scalpel](#scalpel)
    - [ScanIndel](#scanindel)
    - [scanpy](#scanpy)
    - [Scramble](#scramble)
    - [segemehl](#segemehl)
    - [seqan](#seqan)
    - [seqkit](#seqkit)
    - [seqlinkage](#seqlinkage)
    - [seqtk](#seqtk)
    - [sequenza-utils](#sequenza-utils)
    - [sga](#sga)
    - [SGA-ICE](#sga-ice)
    - [shapeit](#shapeit)
    - [shmlast](#shmlast)
    - [shrimp](#shrimp)
    - [sicer](#sicer)
    - [sickle](#sickle)
    - [SIFT](#sift)
    - [slamdunk](#slamdunk)
    - [smalt](#smalt)
    - [smart](#smart)
    - [smrtanalysis](#smrtanalysis)
    - [snaptools](#snaptools)
    - [sniffles](#sniffles)
    - [snippy](#snippy)
    - [snp2hla](#snp2hla)
    - [snpEff](#snpeff)
    - [snptest](#snptest)
    - [SomaticSeq](#somaticseq)
    - [somaticsniper](#somaticsniper)
    - [sortmeRNA](#sortmerna)
    - [spaceranger](#spaceranger)
    - [spades](#spades)
    - [speedseq](#speedseq)
    - [spruce](#spruce)
    - [sratoolkit](#sratoolkit)
    - [SRST2](#srst2)
    - [stampy](#stampy)
    - [STAR](#star)
    - [STAR-Fusion](#star-fusion)
    - [STREAM](#stream)
    - [strelka](#strelka)
    - [stringtie](#stringtie)
    - [ST_pipeline](#st_pipeline)
    - [subread](#subread)
    - [supernova](#supernova)
    - [suppa](#suppa)
    - [SURVIVOR](#survivor)
    - [svclone](#svclone)
    - [sve](#sve)
    - [svtk](#svtk)
    - [svtyper](#svtyper)
    - [tagit](#tagit)
    - [tailseeker](#tailseeker)
    - [talon](#talon)
    - [tappas](#tappas)
    - [taxonkit](#taxonkit)
    - [telescope](#telescope)
    - [telseq](#telseq)
    - [tetoolkit](#tetoolkit)
    - [THetA](#theta)
    - [TMAP](#tmap)
    - [tombo](#tombo)
    - [tophat](#tophat)
    - [TPMCalculator](#tpmcalculator)
    - [TransDecoder](#transdecoder)
    - [transrate](#transrate)
    - [transvar](#transvar)
    - [trimgalore](#trimgalore)
    - [trimmomatic](#trimmomatic)
    - [trinity](#trinity)
    - [Trinotate](#trinotate)
    - [tRNAscan-SE](#trnascan-se)
    - [truffle](#truffle)
    - [TVC](#tvc)
    - [umitools](#umitools)
    - [vasttools](#vasttools)
    - [vcf2maf](#vcf2maf)
    - [vcfanno](#vcfanno)
    - [vcflib](#vcflib)
    - [vcftools](#vcftools)
    - [VEP](#vep)
    - [verifybamid](#verifybamid)
    - [vg](#vg)
    - [viper](#viper)
    - [viral-ngs](#viral-ngs)
    - [vireosnp](#vireosnp)
    - [VirSorter2](#virsorter2)
    - [VIRTUS](#virtus)
    - [vsearch](#vsearch)
    - [vt](#vt)
    - [winnowmap](#winnowmap)
    - [xHLA](#xhla)
    - [XHMM](#xhmm)
    - [xwas](#xwas)
  - [Image Analysis](#image-analysis)
    - [3DSlicer](#3dslicer)
    - [AFNI](#afni)
    - [ANTs](#ants)
    - [aslprep](#aslprep)
    - [B-SOiD](#b-soid-1)
    - [baracus](#baracus)
    - [BrkRaw](#brkraw)
    - [broccoli](#broccoli)
    - [Bsoft](#bsoft)
    - [c3d](#c3d)
    - [cisTEM](#cistem)
    - [civet](#civet)
    - [cmtk](#cmtk)
    - [connectome-workbench](#connectome-workbench)
    - [CTF](#ctf)
    - [ctffind](#ctffind)
    - [dcm2niix](#dcm2niix)
    - [DeepCAD](#deepcad)
    - [DeepLabCut](#deeplabcut-1)
    - [dynamo](#dynamo)
    - [elastix](#elastix)
    - [EMAN2](#eman2)
    - [fastsurfer](#fastsurfer)
    - [Fiji](#fiji)
    - [fitlins](#fitlins)
    - [fmriprep](#fmriprep)
    - [Frealign](#frealign)
    - [Freesurfer](#freesurfer)
    - [FSL](#fsl)
    - [Gctf](#gctf)
    - [Huygens](#huygens)
    - [IMOD](#imod)
    - [ITK-SNAP](#itk-snap)
    - [magetbrain](#magetbrain)
    - [mango](#mango)
    - [minc-toolkit](#minc-toolkit)
    - [MIPAV](#mipav)
    - [MotionCor2](#motioncor2)
    - [mriqc](#mriqc)
    - [mrtrix](#mrtrix)
    - [mrtrix3tissue](#mrtrix3tissue)
    - [nibabies](#nibabies)
    - [OpenSlide](#openslide)
    - [PEET](#peet)
    - [qsiprep](#qsiprep)
    - [QuPath](#qupath)
    - [RELION](#relion)
    - [ResMap](#resmap)
    - [SAMsrcV3](#samsrcv3)
    - [SimNIBS](#simnibs)
    - [SPHIRE](#sphire)
    - [Spinal Cord Toolbox (SCT)](#spinal-cord-toolbox-sct)
    - [spm12](#spm12)
    - [summovie](#summovie)
    - [TeraStitcher](#terastitcher)
    - [tesseract](#tesseract)
    - [topaz](#topaz)
    - [TORTOISE](#tortoise)
    - [tvb](#tvb)
    - [unblur](#unblur)
    - [xcpengine](#xcpengine)
  - [Linkage/Phylogenetics](#linkagephylogenetics)
    - [AdmixTools](#admixtools)
    - [admixture](#admixture)
    - [AncestryMap](#ancestrymap)
    - [Beagle](#beagle)
    - [beagle-lib](#beagle-lib)
    - [BEAST](#beast)
    - [bpipe](#bpipe-1)
    - [CD-HIT](#cd-hit)
    - [eigensoft](#eigensoft)
    - [FastQTL](#fastqtl)
    - [FastTree](#fasttree)
    - [GCTA](#gcta)
    - [GIGI](#gigi)
    - [gubbins](#gubbins)
    - [hyphy](#hyphy)
    - [iqtree](#iqtree)
    - [king](#king)
    - [loki](#loki)
    - [Madeline2](#madeline2)
    - [MALDER](#malder)
    - [mantra](#mantra)
    - [MEGA](#mega)
    - [mega2](#mega2)
    - [merlin](#merlin)
    - [mothur](#mothur)
    - [MrBayes](#mrbayes)
    - [pesca](#pesca)
    - [phase](#phase)
    - [Phylip](#phylip)
    - [pplacer](#pplacer)
    - [PRIMUS](#primus)
    - [PRSice](#prsice)
    - [QIIME](#qiime)
    - [RAxML](#raxml)
    - [raxml-ng](#raxml-ng)
    - [RefinedIBD](#refinedibd)
    - [RevBayes](#revbayes)
    - [seqmix](#seqmix)
    - [shapeit](#shapeit-1)
    - [SMR](#smr)
    - [Solar](#solar)
    - [sumtrees](#sumtrees)
    - [treemix](#treemix)
  - [Mass Spectrometry](#mass-spectrometry)
    - [fragpipe](#fragpipe)
    - [Mascot](#mascot)
    - [maxquant](#maxquant)
    - [metamorpheus](#metamorpheus)
    - [msfragger](#msfragger)
    - [philosopher](#philosopher)
  - [Mathematical/Statistics](#mathematicalstatistics)
    - [ANTs](#ants-1)
    - [CLONET](#clonet)
    - [cluster3](#cluster3)
    - [cmdstan](#cmdstan)
    - [Comsol](#comsol)
    - [GAUSS](#gauss)
    - [gephi](#gephi)
    - [gurobi](#gurobi)
    - [HPhi](#hphi)
    - [IDL/ENVI](#idlenvi)
    - [m2clust](#m2clust)
    - [Mathematica](#mathematica)
    - [Matlab](#matlab)
    - [MCL](#mcl)
    - [Meep](#meep)
    - [Octave](#octave)
    - [omeClust](#omeclust)
    - [PEER](#peer)
    - [R](#r)
    - [Rstudio](#rstudio)
    - [SAS](#sas)
    - [team](#team)
  - [Molecular Modeling/Graphics](#molecular-modelinggraphics)
    - [bio3d-web](#bio3d-web)
    - [blender](#blender)
    - [Chimera](#chimera)
    - [Coot](#coot)
    - [Cytoscape](#cytoscape)
    - [Glycosylator](#glycosylator)
    - [infinisee](#infinisee)
    - [lammps](#lammps)
    - [MD-TASK](#md-task)
    - [neuroconstruct](#neuroconstruct)
    - [OpenBabel](#openbabel)
    - [posefilter](#posefilter)
    - [Psi4](#psi4-1)
    - [Rosetta](#rosetta)
    - [Schrodinger](#schrodinger-1)
    - [seesar](#seesar)
    - [VMD](#vmd-1)
  - [Sequence Analysis](#sequence-analysis)
    - [Accurity](#accurity)
    - [advntr](#advntr)
    - [alleleCount](#allelecount)
    - [ANNOgesic](#annogesic)
    - [anvio](#anvio)
    - [ARDISS](#ardiss-1)
    - [arriba](#arriba)
    - [augustus](#augustus-1)
    - [bamcmp](#bamcmp)
    - [bamgineer](#bamgineer)
    - [Bartender](#bartender-1)
    - [Blast](#blast)
    - [blat](#blat)
    - [BOLT-LMM](#bolt-lmm)
    - [bpipe](#bpipe-2)
    - [bwa-mem2](#bwa-mem2-1)
    - [cactus](#cactus)
    - [CADD](#cadd-1)
    - [CAVIAR](#caviar)
    - [cellsnp-lite](#cellsnp-lite-1)
    - [CHESS](#chess-1)
    - [chipseq_pipeline](#chipseq_pipeline)
    - [cicero](#cicero-1)
    - [circos](#circos-1)
    - [CITE-seq-Count](#cite-seq-count)
    - [Clair3](#clair3-1)
    - [Clinker](#clinker)
    - [clustalo](#clustalo)
    - [ClustalW](#clustalw)
    - [CutRunTools](#cutruntools)
    - [DeconSeq](#deconseq)
    - [DeepM6A](#deepm6a-2)
    - [deepsea](#deepsea)
    - [diamond](#diamond)
    - [DNAWorks](#dnaworks)
    - [drep](#drep)
    - [dyno](#dyno)
    - [Eagle](#eagle)
    - [emim](#emim)
    - [EukRep](#eukrep)
    - [exonerate](#exonerate)
    - [ExpansionHunter](#expansionhunter)
    - [ExpansionHunterDenovo](#expansionhunterdenovo)
    - [fastp](#fastp)
    - [FitHiChIP](#fithichip)
    - [FLAIR](#flair)
    - [focus](#focus)
    - [FuSeq](#fuseq)
    - [genometools](#genometools)
    - [gffcompare](#gffcompare)
    - [GimmeMotifs](#gimmemotifs)
    - [glnexus](#glnexus)
    - [guidance](#guidance)
    - [hhsuite](#hhsuite)
    - [hipstr](#hipstr-1)
    - [HMMER](#hmmer)
    - [HTGTSrep](#htgtsrep)
    - [iDiffIR](#idiffir-1)
    - [igblast](#igblast)
    - [ImReP](#imrep)
    - [infernal](#infernal)
    - [interproscan](#interproscan)
    - [InterVar](#intervar-1)
    - [iVar](#ivar-1)
    - [jellyfish](#jellyfish)
    - [kallisto](#kallisto)
    - [LASTZ](#lastz)
    - [ldsc](#ldsc)
    - [m-tools](#m-tools)
    - [magma](#magma)
    - [mantis](#mantis)
    - [maps](#maps-1)
    - [MashMap](#mashmap)
    - [MELT](#melt)
    - [meme](#meme)
    - [mfold](#mfold)
    - [MMARGE](#mmarge)
    - [MoChA](#mocha)
    - [MQLS](#mqls)
    - [msisensor-pro](#msisensor-pro)
    - [MUMmer](#mummer)
    - [MUSCLE](#muscle)
    - [mutsig2cv](#mutsig2cv)
    - [ncbi-toolkit](#ncbi-toolkit)
    - [netOglyc](#netoglyc)
    - [Nirvana](#nirvana)
    - [OligoArray](#oligoarray)
    - [oncotator](#oncotator)
    - [OpenCRAVAT](#opencravat)
    - [OptiType](#optitype)
    - [ORFfinder](#orffinder)
    - [PAINTOR](#paintor)
    - [pangolin](#pangolin)
    - [PEPPER_deepvariant](#pepper_deepvariant)
    - [phaser](#phaser)
    - [PhIP-Stat](#phip-stat)
    - [phylowgs](#phylowgs)
    - [popsycle](#popsycle)
    - [PRANK](#prank)
    - [preseq](#preseq-1)
    - [primer3](#primer3)
    - [prokka](#prokka)
    - [PSIPRED](#psipred)
    - [pysamstats](#pysamstats)
    - [QTLtools](#qtltools)
    - [randfold](#randfold)
    - [regenie](#regenie)
    - [rMATS](#rmats)
    - [rmblast](#rmblast)
    - [RNAmmer](#rnammer)
    - [rnastructure](#rnastructure)
    - [roary](#roary)
    - [ROSE](#rose-1)
    - [SAIGE](#saige)
    - [scanpy](#scanpy-1)
    - [scvelo](#scvelo)
    - [SEACR](#seacr)
    - [selscan](#selscan)
    - [signalp](#signalp)
    - [smcpp](#smcpp)
    - [smoove](#smoove)
    - [SOAPdenovo-Trans](#soapdenovo-trans)
    - [sonicparanoid](#sonicparanoid)
    - [star-seqr](#star-seqr)
    - [steme](#steme)
    - [stripenn](#stripenn)
    - [SurvivalGWAS_SV](#survivalgwas_sv)
    - [SynthDNM](#synthdnm)
    - [tantan](#tantan)
    - [TelomereHunter](#telomerehunter)
    - [TMHMM](#tmhmm)
    - [tombo](#tombo-1)
    - [TRF](#trf)
    - [trust4](#trust4)
    - [unicycler](#unicycler)
    - [uropa](#uropa)
    - [usearch](#usearch)
    - [VADR](#vadr)
    - [VarScan](#varscan)
    - [VCF-kit](#vcf-kit)
    - [velocyto](#velocyto)
    - [viennarna](#viennarna)
    - [viper](#viper-1)
    - [VirSorter2](#virsorter2-1)
    - [weblogo](#weblogo)
    - [WISExome](#wisexome)
    - [xengsort](#xengsort)
    - [xenome](#xenome)
    - [xHLA](#xhla-1)
  - [Structural Biology](#structural-biology)
    - [alphafold2](#alphafold2)
    - [CCP4](#ccp4)
    - [cryoDRGN](#cryodrgn-2)
    - [crystfel](#crystfel)
    - [CSD](#csd)
    - [Dali](#dali)
    - [DeepHiC](#deephic-1)
    - [DeepMM](#deepmm-1)
    - [DSSP](#dssp)
    - [EM-GAN](#em-gan)
    - [emspring](#emspring)
    - [gautomatch](#gautomatch)
    - [I-TASSER](#i-tasser)
    - [lafter](#lafter)
    - [lammps](#lammps-1)
    - [LocScale](#locscale)
    - [mdtraj](#mdtraj)
    - [moabs](#moabs)
    - [mustache](#mustache)
    - [naccess](#naccess)
    - [novactf](#novactf)
    - [pf_refinement](#pf_refinement)
    - [Phenix](#phenix)
    - [PROCHECK](#procheck)
    - [pydockrmsd](#pydockrmsd)
    - [pyem](#pyem)
    - [PyMOL](#pymol)
    - [PyRosetta](#pyrosetta)
    - [rDock](#rdock)
    - [Schrodinger](#schrodinger-2)
    - [Scipion](#scipion)
    - [scratch1d](#scratch1d)
    - [sidesplitter](#sidesplitter)
    - [subtom](#subtom)
    - [symmetry](#symmetry)
    - [VMD](#vmd-2)
    - [Xplor-NIH](#xplor-nih)
    - [ZDOCK](#zdock)
  - [Systems Biology](#systems-biology)
    - [AMON](#amon)
    - [cellphonedb](#cellphonedb)
    - [eggNOGmapper](#eggnogmapper)
    - [EukRep](#eukrep-1)
    - [genesis](#genesis)
    - [GRNBoost](#grnboost)
    - [gsea](#gsea)
    - [MAKER](#maker)
    - [Neuron](#neuron)
    - [scvelo](#scvelo-1)
    - [vdjtools](#vdjtools)
  - [Utilities](#utilities)
    - [aria2](#aria2)
    - [ariba](#ariba)
    - [asciinema](#asciinema)
    - [Aspera](#aspera)
    - [AWS](#aws)
    - [azcopy](#azcopy)
    - [bbcp](#bbcp)
    - [bgionline](#bgionline)
    - [chance](#chance)
    - [circos](#circos-2)
    - [coreutils](#coreutils)
    - [cpdf](#cpdf)
    - [crumble](#crumble)
    - [cryolo](#cryolo)
    - [datalad](#datalad)
    - [datamash](#datamash)
    - [DNAnexus](#dnanexus)
    - [EDirect](#edirect)
    - [Editors](#editors)
    - [gdc-client](#gdc-client)
    - [gdrive](#gdrive)
    - [gen3-client](#gen3-client)
    - [Ghostscript](#ghostscript)
    - [gnuplot](#gnuplot)
    - [google-cloud-sdk](#google-cloud-sdk)
    - [Grace](#grace)
    - [graphviz](#graphviz)
    - [groff](#groff)
    - [gzip](#gzip)
    - [h5util](#h5util)
    - [ImageMagick](#imagemagick)
    - [jo](#jo)
    - [jq](#jq)
    - [kronatools](#kronatools)
    - [lbzip2](#lbzip2)
    - [longshot](#longshot)
    - [mc](#mc)
    - [mercurial](#mercurial)
    - [MySQL](#mysql)
    - [nda-tools](#nda-tools)
    - [neovim](#neovim)
    - [netpbm](#netpbm)
    - [nvchecker](#nvchecker)
    - [parallel](#parallel)
    - [paraview](#paraview)
    - [patchelf](#patchelf)
    - [pdf2svg](#pdf2svg)
    - [pigz](#pigz)
    - [pilon](#pilon)
    - [POVRay](#povray)
    - [pysurfer](#pysurfer)
    - [rclone](#rclone)
    - [rdfind](#rdfind)
    - [rfmix](#rfmix)
    - [sb_cli](#sb_cli)
    - [screen](#screen)
    - [singularity](#singularity)
    - [snp-sites](#snp-sites)
    - [spark](#spark)
    - [SQLite](#sqlite)
    - [stow](#stow)
    - [swarm](#swarm)
    - [swig](#swig)
    - [synapseclient](#synapseclient)
    - [sysbench](#sysbench)
    - [texinfo](#texinfo)
    - [tmux](#tmux)
    - [UKBB](#ukbb)
    - [vartrix](#vartrix)
    - [vcf2db](#vcf2db)
    - [vim](#vim)
    - [visidata](#visidata)
    - [vscode](#vscode)
    - [whatshap](#whatshap)
    - [wuzz](#wuzz)
    - [xclip](#xclip)
    - [xpdf](#xpdf)
  - [Workflow Managers](#workflow-managers)
    - [cromwell](#cromwell)
    - [croo](#croo)
    - [nextflow](#nextflow)
    - [snakemake](#snakemake)
  - [Appendex](#appendex)

<!-- /code_chunk_output -->
